{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "vcf=pd.read_table('APD002_T2.ALL.consensus.tab', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Set ipython's max row display\n",
    "pd.set_option('display.max_row', 1000)\n",
    "\n",
    "# Set iPython's max column width to 50\n",
    "pd.set_option('display.max_columns', 50)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>width</th>\n",
       "      <th>strand</th>\n",
       "      <th>AD</th>\n",
       "      <th>AD.1</th>\n",
       "      <th>AF</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>SYMBOL</th>\n",
       "      <th>HGVSc</th>\n",
       "      <th>HGVSp</th>\n",
       "      <th>CLIN_SIG</th>\n",
       "      <th>PolyPhen</th>\n",
       "      <th>SIFT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1961657</td>\n",
       "      <td>1961657</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>10</td>\n",
       "      <td>0.164</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GABRD</td>\n",
       "      <td>ENST00000378585.4:c.1295G&gt;A</td>\n",
       "      <td>ENSP00000367848.4:p.Arg432His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>16259538</td>\n",
       "      <td>16259538</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPEN</td>\n",
       "      <td>ENST00000375759.3:c.6803C&gt;A</td>\n",
       "      <td>ENSP00000364912.3:p.Ala2268Asp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>tolerated_low_confidence(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>16461628</td>\n",
       "      <td>16461628</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>5</td>\n",
       "      <td>0.161</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>EPHA2</td>\n",
       "      <td>ENST00000358432.5:c.1485C&gt;T</td>\n",
       "      <td>ENSP00000351209.5:p.Asp495%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>1</td>\n",
       "      <td>25880505</td>\n",
       "      <td>25880505</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>9</td>\n",
       "      <td>0.180</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>LDLRAP1</td>\n",
       "      <td>ENST00000374338.4:c.181C&gt;G</td>\n",
       "      <td>ENSP00000363458.4:p.Pro61Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.914)</td>\n",
       "      <td>tolerated(0.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>1</td>\n",
       "      <td>28816442</td>\n",
       "      <td>28816442</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>4</td>\n",
       "      <td>0.074</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PHACTR4</td>\n",
       "      <td>ENST00000373836.3:c.1846+705T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>1</td>\n",
       "      <td>46501061</td>\n",
       "      <td>46501061</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>9</td>\n",
       "      <td>0.290</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PIK3R3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      <td>1</td>\n",
       "      <td>53683815</td>\n",
       "      <td>53683815</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>4</td>\n",
       "      <td>0.121</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C1orf123</td>\n",
       "      <td>ENST00000294360.4:c.261+29G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>1</td>\n",
       "      <td>55088984</td>\n",
       "      <td>55088984</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>71</td>\n",
       "      <td>10</td>\n",
       "      <td>0.123</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAM151A</td>\n",
       "      <td>ENST00000302250.2:c.85G&gt;A</td>\n",
       "      <td>ENSP00000306888.2:p.Ala29Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.75)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      <td>1</td>\n",
       "      <td>86591188</td>\n",
       "      <td>86591188</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>COL24A1</td>\n",
       "      <td>ENST00000370571.2:c.831A&gt;C</td>\n",
       "      <td>ENSP00000359603.2:p.Lys277Asn</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>1</td>\n",
       "      <td>112846126</td>\n",
       "      <td>112846126</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>5</td>\n",
       "      <td>0.185</td>\n",
       "      <td>intron_variant&amp;non_coding_transcript_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP5-965F6.2</td>\n",
       "      <td>ENST00000427290.1:n.167+56858G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      <td>1</td>\n",
       "      <td>114193692</td>\n",
       "      <td>114193692</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>6</td>\n",
       "      <td>0.122</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MAGI3</td>\n",
       "      <td>ENST00000307546.9:c.2304C&gt;T</td>\n",
       "      <td>ENSP00000304604.9:p.Arg768%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      <td>1</td>\n",
       "      <td>118634278</td>\n",
       "      <td>118634278</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>4</td>\n",
       "      <td>0.070</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPAG17</td>\n",
       "      <td>ENST00000336338.5:c.1310G&gt;A</td>\n",
       "      <td>ENSP00000337804.5:p.Arg437Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>149919163</td>\n",
       "      <td>149919163</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>5</td>\n",
       "      <td>0.074</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OTUD7B</td>\n",
       "      <td>ENST00000369135.4:c.1312T&gt;C</td>\n",
       "      <td>ENSP00000358131.3:p.Ser438Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.367)</td>\n",
       "      <td>tolerated(0.11)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>149939389</td>\n",
       "      <td>149939389</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>3</td>\n",
       "      <td>0.049</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OTUD7B</td>\n",
       "      <td>ENST00000369135.4:c.332G&gt;T</td>\n",
       "      <td>ENSP00000358131.3:p.Arg111Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      <td>1</td>\n",
       "      <td>152850751</td>\n",
       "      <td>152850751</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>87</td>\n",
       "      <td>28</td>\n",
       "      <td>0.243</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SMCP</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      <td>1</td>\n",
       "      <td>152884003</td>\n",
       "      <td>152884003</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>0.075</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IVL</td>\n",
       "      <td>ENST00000368764.3:c.1730A&gt;G</td>\n",
       "      <td>ENSP00000357753.3:p.Glu577Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.942)</td>\n",
       "      <td>tolerated_low_confidence(0.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      <td>1</td>\n",
       "      <td>153791394</td>\n",
       "      <td>153791394</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>4</td>\n",
       "      <td>0.103</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GATAD2B</td>\n",
       "      <td>ENST00000368655.4:c.470A&gt;G</td>\n",
       "      <td>ENSP00000357644.4:p.Lys157Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.956)</td>\n",
       "      <td>tolerated(0.25)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      <td>1</td>\n",
       "      <td>166246456</td>\n",
       "      <td>166246456</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>70</td>\n",
       "      <td>3</td>\n",
       "      <td>0.041</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-9L18.2</td>\n",
       "      <td>ENST00000400979.2:n.379T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      <td>1</td>\n",
       "      <td>172571853</td>\n",
       "      <td>172571853</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SUCO</td>\n",
       "      <td>ENST00000263688.3:c.3191A&gt;G</td>\n",
       "      <td>ENSP00000263688.3:p.Asp1064Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      <td>1</td>\n",
       "      <td>180217567</td>\n",
       "      <td>180217567</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>3</td>\n",
       "      <td>0.061</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>LHX4</td>\n",
       "      <td>ENST00000263726.2:c.224T&gt;C</td>\n",
       "      <td>ENSP00000263726.2:p.Val75Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      <td>1</td>\n",
       "      <td>193219058</td>\n",
       "      <td>193219058</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>4</td>\n",
       "      <td>0.069</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CDC73</td>\n",
       "      <td>ENST00000367435.3:c.1559+57G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>24050735</td>\n",
       "      <td>24050735</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>79</td>\n",
       "      <td>5</td>\n",
       "      <td>0.060</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ATAD2B</td>\n",
       "      <td>ENST00000238789.5:c.1819+984C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>54076289</td>\n",
       "      <td>54076289</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>14</td>\n",
       "      <td>0.241</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GPR75-ASB3</td>\n",
       "      <td>ENST00000352846.3:c.101+10676G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>2</td>\n",
       "      <td>56227804</td>\n",
       "      <td>56227804</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>13</td>\n",
       "      <td>0.241</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MIR216B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      <td>2</td>\n",
       "      <td>64863545</td>\n",
       "      <td>64863545</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SERTAD2</td>\n",
       "      <td>ENST00000313349.3:c.461C&gt;A</td>\n",
       "      <td>ENSP00000326933.3:p.Pro154His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.818)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      <td>2</td>\n",
       "      <td>120850877</td>\n",
       "      <td>120850877</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>EPB41L5</td>\n",
       "      <td>ENST00000263713.5:c.1178+25A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      <td>2</td>\n",
       "      <td>122161436</td>\n",
       "      <td>122161436</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>4</td>\n",
       "      <td>0.105</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CLASP1</td>\n",
       "      <td>ENST00000263710.4:c.2733+511A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      <td>2</td>\n",
       "      <td>189856266</td>\n",
       "      <td>189856266</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.095</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>COL3A1</td>\n",
       "      <td>ENST00000304636.3:c.897+9G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      <td>2</td>\n",
       "      <td>209168996</td>\n",
       "      <td>209168996</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>4</td>\n",
       "      <td>0.062</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PIKFYVE</td>\n",
       "      <td>ENST00000264380.4:c.1422C&gt;T</td>\n",
       "      <td>ENSP00000264380.4:p.Asp474%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      <td>2</td>\n",
       "      <td>220439634</td>\n",
       "      <td>220439634</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>83</td>\n",
       "      <td>4</td>\n",
       "      <td>0.046</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>INHA</td>\n",
       "      <td>ENST00000243786.2:c.487G&gt;A</td>\n",
       "      <td>ENSP00000243786.2:p.Val163Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.46)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>30</td>\n",
       "      <td>2</td>\n",
       "      <td>239010520</td>\n",
       "      <td>239010520</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>0.075</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SCLY</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>31665184</td>\n",
       "      <td>31665184</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>8</td>\n",
       "      <td>0.222</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>STT3B</td>\n",
       "      <td>ENST00000295770.2:c.1558C&gt;A</td>\n",
       "      <td>ENSP00000295770.2:p.His520Asn</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.028)</td>\n",
       "      <td>tolerated(0.44)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>37574951</td>\n",
       "      <td>37574951</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ITGA9</td>\n",
       "      <td>ENST00000264741.5:c.1520G&gt;A</td>\n",
       "      <td>ENSP00000264741.5:p.Gly507Glu</td>\n",
       "      <td>benign</td>\n",
       "      <td>benign(0.02)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>41759288</td>\n",
       "      <td>41759288</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>4</td>\n",
       "      <td>0.074</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ULK4</td>\n",
       "      <td>ENST00000301831.4:c.2387A&gt;G</td>\n",
       "      <td>ENSP00000301831.4:p.Asn796Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>34</td>\n",
       "      <td>3</td>\n",
       "      <td>49137421</td>\n",
       "      <td>49137421</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>4</td>\n",
       "      <td>0.077</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>QARS</td>\n",
       "      <td>ENST00000306125.6:c.1268C&gt;G</td>\n",
       "      <td>ENSP00000307567.6:p.Thr423Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>tolerated(0.06)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35</td>\n",
       "      <td>3</td>\n",
       "      <td>113697105</td>\n",
       "      <td>113697105</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>KIAA1407</td>\n",
       "      <td>ENST00000295878.3:c.2534G&gt;A</td>\n",
       "      <td>ENSP00000295878.3:p.Trp845Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>124692689</td>\n",
       "      <td>124692689</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>4</td>\n",
       "      <td>0.069</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>HEG1</td>\n",
       "      <td>ENST00000311127.4:c.3882G&gt;A</td>\n",
       "      <td>ENSP00000311502.3:p.Pro1294%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>37</td>\n",
       "      <td>3</td>\n",
       "      <td>132242359</td>\n",
       "      <td>132242359</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>65</td>\n",
       "      <td>3</td>\n",
       "      <td>0.044</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DNAJC13</td>\n",
       "      <td>ENST00000260818.6:c.5887-25A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>134644699</td>\n",
       "      <td>134644699</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>5</td>\n",
       "      <td>0.076</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHB1</td>\n",
       "      <td>ENST00000398015.3:c.100T&gt;A</td>\n",
       "      <td>ENSP00000381097.3:p.Trp34Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>39</td>\n",
       "      <td>3</td>\n",
       "      <td>151171301</td>\n",
       "      <td>151171301</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>3</td>\n",
       "      <td>0.065</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IGSF10</td>\n",
       "      <td>ENST00000282466.3:c.586T&gt;C</td>\n",
       "      <td>ENSP00000282466.3:p.Phe196Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.005)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>40</td>\n",
       "      <td>3</td>\n",
       "      <td>180041515</td>\n",
       "      <td>180041515</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>19</td>\n",
       "      <td>0.339</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-420J11.1</td>\n",
       "      <td>ENST00000493579.1:n.621G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>41</td>\n",
       "      <td>3</td>\n",
       "      <td>190167616</td>\n",
       "      <td>190167616</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>7</td>\n",
       "      <td>0.104</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TMEM207</td>\n",
       "      <td>ENST00000354905.2:c.-18G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>42</td>\n",
       "      <td>4</td>\n",
       "      <td>55988729</td>\n",
       "      <td>55988729</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KDR</td>\n",
       "      <td>ENST00000263923.4:c.68-1372G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>56308687</td>\n",
       "      <td>56308687</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>4</td>\n",
       "      <td>0.071</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>CLOCK</td>\n",
       "      <td>ENST00000309964.4:c.2017G&gt;T</td>\n",
       "      <td>ENSP00000308741.4:p.Gly673Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>140599783</td>\n",
       "      <td>140599783</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>4</td>\n",
       "      <td>0.073</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MGST2</td>\n",
       "      <td>ENST00000265498.1:c.145G&gt;A</td>\n",
       "      <td>ENSP00000265498.1:p.Val49Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.96)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>45</td>\n",
       "      <td>4</td>\n",
       "      <td>142154850</td>\n",
       "      <td>142154850</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>4</td>\n",
       "      <td>0.129</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ZNF330</td>\n",
       "      <td>ENST00000262990.4:c.689-19A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>177683539</td>\n",
       "      <td>177683539</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>0.115</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>VEGFC</td>\n",
       "      <td>ENST00000280193.2:c.147+29780T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>185339813</td>\n",
       "      <td>185339813</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>11</td>\n",
       "      <td>0.164</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>IRF2</td>\n",
       "      <td>ENST00000393593.3:c.237G&gt;A</td>\n",
       "      <td>ENSP00000377218.3:p.Ala79%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>48</td>\n",
       "      <td>5</td>\n",
       "      <td>10430082</td>\n",
       "      <td>10430082</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>10</td>\n",
       "      <td>0.167</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MARCH6</td>\n",
       "      <td>ENST00000274140.5:c.2584A&gt;G</td>\n",
       "      <td>ENSP00000274140.4:p.Ile862Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.78)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>49</td>\n",
       "      <td>5</td>\n",
       "      <td>10638114</td>\n",
       "      <td>10638114</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.083</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ANKRD33B</td>\n",
       "      <td>ENST00000296657.5:c.497-26A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>50</td>\n",
       "      <td>5</td>\n",
       "      <td>121297625</td>\n",
       "      <td>121297625</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>3</td>\n",
       "      <td>0.058</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SRFBP1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>51</td>\n",
       "      <td>5</td>\n",
       "      <td>126781439</td>\n",
       "      <td>126781439</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>0.091</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MEGF10</td>\n",
       "      <td>ENST00000274473.6:c.2728+54C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>52</td>\n",
       "      <td>5</td>\n",
       "      <td>131940554</td>\n",
       "      <td>131940554</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>4</td>\n",
       "      <td>0.087</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RAD50</td>\n",
       "      <td>ENST00000265335.6:c.2581C&gt;A</td>\n",
       "      <td>ENSP00000265335.6:p.Gln861Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.103)</td>\n",
       "      <td>tolerated(0.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>53</td>\n",
       "      <td>5</td>\n",
       "      <td>140251674</td>\n",
       "      <td>140251674</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PCDHA10</td>\n",
       "      <td>ENST00000307360.5:c.2388+13653T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>54</td>\n",
       "      <td>5</td>\n",
       "      <td>140594488</td>\n",
       "      <td>140594488</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>6</td>\n",
       "      <td>0.125</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PCDHB12</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>55</td>\n",
       "      <td>5</td>\n",
       "      <td>150924038</td>\n",
       "      <td>150924038</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>57</td>\n",
       "      <td>5</td>\n",
       "      <td>0.081</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAT2</td>\n",
       "      <td>ENST00000261800.5:c.6650C&gt;T</td>\n",
       "      <td>ENSP00000261800.5:p.Thr2217Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.255)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>56</td>\n",
       "      <td>6</td>\n",
       "      <td>3983819</td>\n",
       "      <td>3983819</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>25</td>\n",
       "      <td>4</td>\n",
       "      <td>0.138</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C6ORF50</td>\n",
       "      <td>ENST00000356722.3:c.*109C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>57</td>\n",
       "      <td>6</td>\n",
       "      <td>24873214</td>\n",
       "      <td>24873214</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FAM65B</td>\n",
       "      <td>ENST00000259698.4:c.258-27C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>58</td>\n",
       "      <td>6</td>\n",
       "      <td>29769880</td>\n",
       "      <td>29769880</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RPL7AP7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>59</td>\n",
       "      <td>6</td>\n",
       "      <td>33244788</td>\n",
       "      <td>33244788</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>7</td>\n",
       "      <td>0.184</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>WDR46</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>60</td>\n",
       "      <td>6</td>\n",
       "      <td>34840222</td>\n",
       "      <td>34840222</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>25</td>\n",
       "      <td>4</td>\n",
       "      <td>0.138</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>UHRF1BP1</td>\n",
       "      <td>ENST00000192788.5:c.*7T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>61</td>\n",
       "      <td>6</td>\n",
       "      <td>35436741</td>\n",
       "      <td>35436741</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FANCE</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>62</td>\n",
       "      <td>6</td>\n",
       "      <td>36946386</td>\n",
       "      <td>36946386</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>4</td>\n",
       "      <td>0.093</td>\n",
       "      <td>splice_region_variant&amp;synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MTCH1</td>\n",
       "      <td>ENST00000373616.5:c.408C&gt;A</td>\n",
       "      <td>ENSP00000362718.5:p.Ala136%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>63</td>\n",
       "      <td>6</td>\n",
       "      <td>42985846</td>\n",
       "      <td>42985846</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>4</td>\n",
       "      <td>0.077</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RRP36</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>64</td>\n",
       "      <td>6</td>\n",
       "      <td>43180949</td>\n",
       "      <td>43180949</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>10</td>\n",
       "      <td>0.156</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CUL9</td>\n",
       "      <td>ENST00000252050.4:c.5275C&gt;T</td>\n",
       "      <td>ENSP00000252050.4:p.Arg1759Trp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>65</td>\n",
       "      <td>6</td>\n",
       "      <td>52051109</td>\n",
       "      <td>52051109</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>0.083</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IL17A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>66</td>\n",
       "      <td>6</td>\n",
       "      <td>80383283</td>\n",
       "      <td>80383283</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>4</td>\n",
       "      <td>0.103</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SH3BGRL2</td>\n",
       "      <td>ENST00000369838.4:c.46-48C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>67</td>\n",
       "      <td>6</td>\n",
       "      <td>89974138</td>\n",
       "      <td>89974138</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>6</td>\n",
       "      <td>0.107</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GABRR2</td>\n",
       "      <td>ENST00000402938.3:c.1079G&gt;A</td>\n",
       "      <td>ENSP00000386029.4:p.Arg360Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.034)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>68</td>\n",
       "      <td>6</td>\n",
       "      <td>111587333</td>\n",
       "      <td>111587333</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>4</td>\n",
       "      <td>0.118</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KIAA1919</td>\n",
       "      <td>ENST00000368847.4:c.568G&gt;T</td>\n",
       "      <td>ENSP00000357840.4:p.Gly190Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>69</td>\n",
       "      <td>7</td>\n",
       "      <td>565436</td>\n",
       "      <td>565436</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>4</td>\n",
       "      <td>0.089</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AC147651.3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>70</td>\n",
       "      <td>7</td>\n",
       "      <td>55260474</td>\n",
       "      <td>55260474</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>3</td>\n",
       "      <td>0.045</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EGFR</td>\n",
       "      <td>ENST00000275493.2:c.2641A&gt;G</td>\n",
       "      <td>ENSP00000275493.2:p.Met881Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.583)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>71</td>\n",
       "      <td>7</td>\n",
       "      <td>65553885</td>\n",
       "      <td>65553885</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ASL</td>\n",
       "      <td>ENST00000304874.9:c.810C&gt;A</td>\n",
       "      <td>ENSP00000307188.9:p.Phe270Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.982)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>72</td>\n",
       "      <td>7</td>\n",
       "      <td>93125190</td>\n",
       "      <td>93125190</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>0.085</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CALCR</td>\n",
       "      <td>ENST00000359558.2:c.28+26C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>73</td>\n",
       "      <td>7</td>\n",
       "      <td>100029092</td>\n",
       "      <td>100029092</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>68</td>\n",
       "      <td>9</td>\n",
       "      <td>0.117</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PPP1R35</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74</td>\n",
       "      <td>7</td>\n",
       "      <td>100278679</td>\n",
       "      <td>100278679</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>99</td>\n",
       "      <td>5</td>\n",
       "      <td>0.048</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GIGYF1</td>\n",
       "      <td>ENST00000275732.5:c.*663A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>75</td>\n",
       "      <td>7</td>\n",
       "      <td>100817842</td>\n",
       "      <td>100817842</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>131</td>\n",
       "      <td>21</td>\n",
       "      <td>0.138</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NAT16</td>\n",
       "      <td>ENST00000300303.2:c.247C&gt;T</td>\n",
       "      <td>ENSP00000300303.2:p.Arg83Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.077)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>76</td>\n",
       "      <td>7</td>\n",
       "      <td>113558433</td>\n",
       "      <td>113558433</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>55</td>\n",
       "      <td>3</td>\n",
       "      <td>0.052</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PPP1R3A</td>\n",
       "      <td>ENST00000284601.3:c.619T&gt;C</td>\n",
       "      <td>ENSP00000284601.3:p.Phe207Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.971)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>77</td>\n",
       "      <td>7</td>\n",
       "      <td>135333354</td>\n",
       "      <td>135333354</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.077</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NUP205</td>\n",
       "      <td>ENST00000285968.6:c.*50T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>78</td>\n",
       "      <td>7</td>\n",
       "      <td>136978123</td>\n",
       "      <td>136978123</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>88</td>\n",
       "      <td>8</td>\n",
       "      <td>0.083</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PTN</td>\n",
       "      <td>ENST00000348225.2:c.-1-38402G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>79</td>\n",
       "      <td>7</td>\n",
       "      <td>136982347</td>\n",
       "      <td>136982347</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>66</td>\n",
       "      <td>5</td>\n",
       "      <td>0.070</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PTN</td>\n",
       "      <td>ENST00000348225.2:c.-1-42626C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80</td>\n",
       "      <td>7</td>\n",
       "      <td>139725986</td>\n",
       "      <td>139725986</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>4</td>\n",
       "      <td>0.065</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PARP12</td>\n",
       "      <td>ENST00000263549.3:c.1780+11A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>81</td>\n",
       "      <td>7</td>\n",
       "      <td>140064270</td>\n",
       "      <td>140064270</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>0.091</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SLC37A3</td>\n",
       "      <td>ENST00000326232.9:c.313G&gt;A</td>\n",
       "      <td>ENSP00000321498.9:p.Val105Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.65)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>82</td>\n",
       "      <td>7</td>\n",
       "      <td>143098692</td>\n",
       "      <td>143098692</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHA1</td>\n",
       "      <td>ENST00000275815.3:c.157G&gt;A</td>\n",
       "      <td>ENSP00000275815.3:p.Glu53Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.043)</td>\n",
       "      <td>tolerated(0.44)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>83</td>\n",
       "      <td>8</td>\n",
       "      <td>22464198</td>\n",
       "      <td>22464198</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>0.088</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C8orf58</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>84</td>\n",
       "      <td>8</td>\n",
       "      <td>41543691</td>\n",
       "      <td>41543691</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>18</td>\n",
       "      <td>0.340</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ANK1</td>\n",
       "      <td>ENST00000265709.8:c.4492C&gt;T</td>\n",
       "      <td>ENSP00000265709.8:p.Arg1498Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.902)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>85</td>\n",
       "      <td>8</td>\n",
       "      <td>75765469</td>\n",
       "      <td>75765469</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>3</td>\n",
       "      <td>0.047</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PI15</td>\n",
       "      <td>ENST00000260113.2:c.*3981G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>86</td>\n",
       "      <td>8</td>\n",
       "      <td>86554927</td>\n",
       "      <td>86554927</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>0.107</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>REXO1L8P</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>87</td>\n",
       "      <td>8</td>\n",
       "      <td>144688268</td>\n",
       "      <td>144688268</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>94</td>\n",
       "      <td>4</td>\n",
       "      <td>0.041</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PYCRL</td>\n",
       "      <td>ENST00000220966.6:c.642C&gt;A</td>\n",
       "      <td>ENSP00000220966.6:p.Ser214Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.851)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>88</td>\n",
       "      <td>8</td>\n",
       "      <td>144992103</td>\n",
       "      <td>144992103</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLEC</td>\n",
       "      <td>ENST00000322810.4:c.12297A&gt;G</td>\n",
       "      <td>ENSP00000323856.4:p.Thr4099%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>89</td>\n",
       "      <td>9</td>\n",
       "      <td>85615980</td>\n",
       "      <td>85615980</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>6</td>\n",
       "      <td>0.103</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RASEF</td>\n",
       "      <td>ENST00000376447.3:c.1268G&gt;A</td>\n",
       "      <td>ENSP00000365630.3:p.Arg423Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.018)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>90</td>\n",
       "      <td>9</td>\n",
       "      <td>96030048</td>\n",
       "      <td>96030048</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>5</td>\n",
       "      <td>0.085</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>WNK2</td>\n",
       "      <td>ENST00000297954.4:c.3717G&gt;T</td>\n",
       "      <td>ENSP00000297954.4:p.Leu1239%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>91</td>\n",
       "      <td>9</td>\n",
       "      <td>107288802</td>\n",
       "      <td>107288802</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>0.073</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR13C4</td>\n",
       "      <td>ENST00000277216.3:c.689C&gt;T</td>\n",
       "      <td>ENSP00000277216.3:p.Ser230Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.876)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>92</td>\n",
       "      <td>9</td>\n",
       "      <td>140450159</td>\n",
       "      <td>140450159</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>3</td>\n",
       "      <td>0.097</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DPH7</td>\n",
       "      <td>ENST00000277540.2:c.950-59G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>93</td>\n",
       "      <td>9</td>\n",
       "      <td>140473194</td>\n",
       "      <td>140473194</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>17</td>\n",
       "      <td>3</td>\n",
       "      <td>0.150</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>DPH7</td>\n",
       "      <td>ENST00000277540.2:c.36C&gt;A</td>\n",
       "      <td>ENSP00000277540.2:p.Thr12%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>94</td>\n",
       "      <td>10</td>\n",
       "      <td>5567864</td>\n",
       "      <td>5567864</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>0.107</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CALML3</td>\n",
       "      <td>ENST00000315238.1:c.*366G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>95</td>\n",
       "      <td>10</td>\n",
       "      <td>6528087</td>\n",
       "      <td>6528087</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>4</td>\n",
       "      <td>0.070</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PRKCQ</td>\n",
       "      <td>ENST00000263125.5:c.810T&gt;C</td>\n",
       "      <td>ENSP00000263125.5:p.His270%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>96</td>\n",
       "      <td>10</td>\n",
       "      <td>7621700</td>\n",
       "      <td>7621700</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>11</td>\n",
       "      <td>0.229</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ITIH5</td>\n",
       "      <td>ENST00000256861.6:c.1418+18G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>97</td>\n",
       "      <td>10</td>\n",
       "      <td>22615352</td>\n",
       "      <td>22615352</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>4</td>\n",
       "      <td>0.073</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>BMI1</td>\n",
       "      <td>ENST00000376663.3:c.-19-8T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>98</td>\n",
       "      <td>10</td>\n",
       "      <td>33544735</td>\n",
       "      <td>33544735</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>3</td>\n",
       "      <td>0.064</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NRP1</td>\n",
       "      <td>ENST00000265371.4:c.814+509G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>99</td>\n",
       "      <td>10</td>\n",
       "      <td>115610067</td>\n",
       "      <td>115610067</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>5</td>\n",
       "      <td>0.102</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DCLRE1A</td>\n",
       "      <td>ENST00000361384.2:c.797A&gt;G</td>\n",
       "      <td>ENSP00000355185.2:p.Glu266Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>100</td>\n",
       "      <td>10</td>\n",
       "      <td>116232769</td>\n",
       "      <td>116232769</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>4</td>\n",
       "      <td>0.061</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>ABLIM1</td>\n",
       "      <td>ENST00000277895.5:c.1242T&gt;A</td>\n",
       "      <td>ENSP00000277895.5:p.Tyr414Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>101</td>\n",
       "      <td>10</td>\n",
       "      <td>126490383</td>\n",
       "      <td>126490383</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>25</td>\n",
       "      <td>3</td>\n",
       "      <td>0.107</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FAM175B</td>\n",
       "      <td>ENST00000298492.5:c.-16G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>102</td>\n",
       "      <td>10</td>\n",
       "      <td>128829989</td>\n",
       "      <td>128829989</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>3</td>\n",
       "      <td>0.058</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DOCK1</td>\n",
       "      <td>ENST00000280333.6:c.1637A&gt;G</td>\n",
       "      <td>ENSP00000280333.6:p.Asp546Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.589)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>103</td>\n",
       "      <td>10</td>\n",
       "      <td>128935358</td>\n",
       "      <td>128935358</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>3</td>\n",
       "      <td>0.054</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DOCK1</td>\n",
       "      <td>ENST00000280333.6:c.2784+9330G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>104</td>\n",
       "      <td>10</td>\n",
       "      <td>135003241</td>\n",
       "      <td>135003241</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>13</td>\n",
       "      <td>0.217</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KNDC1</td>\n",
       "      <td>ENST00000304613.3:c.1514-15G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>105</td>\n",
       "      <td>11</td>\n",
       "      <td>562421</td>\n",
       "      <td>562421</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>4</td>\n",
       "      <td>0.063</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C11orf35</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>106</td>\n",
       "      <td>11</td>\n",
       "      <td>6291067</td>\n",
       "      <td>6291067</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>4</td>\n",
       "      <td>0.114</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CCKBR</td>\n",
       "      <td>ENST00000334619.2:c.320G&gt;A</td>\n",
       "      <td>ENSP00000335544.2:p.Cys107Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>107</td>\n",
       "      <td>11</td>\n",
       "      <td>18755185</td>\n",
       "      <td>18755185</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>3</td>\n",
       "      <td>0.046</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PTPN5</td>\n",
       "      <td>ENST00000358540.2:c.1001-3C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>108</td>\n",
       "      <td>11</td>\n",
       "      <td>56015751</td>\n",
       "      <td>56015751</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>OR5T3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>109</td>\n",
       "      <td>11</td>\n",
       "      <td>59271320</td>\n",
       "      <td>59271320</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>0.137</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR4D11</td>\n",
       "      <td>ENST00000313253.1:c.272C&gt;T</td>\n",
       "      <td>ENSP00000320077.1:p.Thr91Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.046)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>110</td>\n",
       "      <td>11</td>\n",
       "      <td>59861497</td>\n",
       "      <td>59861497</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MS4A2</td>\n",
       "      <td>ENST00000278888.3:c.598A&gt;T</td>\n",
       "      <td>ENSP00000278888.3:p.Thr200Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.708)</td>\n",
       "      <td>tolerated(0.37)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>111</td>\n",
       "      <td>11</td>\n",
       "      <td>62543243</td>\n",
       "      <td>62543243</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>4</td>\n",
       "      <td>0.103</td>\n",
       "      <td>splice_region_variant&amp;5_prime_UTR_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TAF6L</td>\n",
       "      <td>ENST00000294168.3:c.-13C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>112</td>\n",
       "      <td>11</td>\n",
       "      <td>68676071</td>\n",
       "      <td>68676071</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IGHMBP2</td>\n",
       "      <td>ENST00000255078.3:c.519A&gt;T</td>\n",
       "      <td>ENSP00000255078.3:p.Arg173Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.87)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>113</td>\n",
       "      <td>11</td>\n",
       "      <td>75438449</td>\n",
       "      <td>75438449</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>7</td>\n",
       "      <td>0.250</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MOGAT2</td>\n",
       "      <td>ENST00000198801.5:c.271-31G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>114</td>\n",
       "      <td>11</td>\n",
       "      <td>86662308</td>\n",
       "      <td>86662308</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>4</td>\n",
       "      <td>0.100</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-736K20.6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>115</td>\n",
       "      <td>11</td>\n",
       "      <td>124189456</td>\n",
       "      <td>124189456</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>9</td>\n",
       "      <td>0.237</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR8D2</td>\n",
       "      <td>ENST00000357438.2:c.638C&gt;T</td>\n",
       "      <td>ENSP00000350022.2:p.Ala213Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.869)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>116</td>\n",
       "      <td>11</td>\n",
       "      <td>133651806</td>\n",
       "      <td>133651806</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>5</td>\n",
       "      <td>0.135</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-448P19.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>117</td>\n",
       "      <td>12</td>\n",
       "      <td>8759488</td>\n",
       "      <td>8759488</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>AICDA</td>\n",
       "      <td>ENST00000229335.6:c.129A&gt;G</td>\n",
       "      <td>ENSP00000229335.6:p.Ser43%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>118</td>\n",
       "      <td>12</td>\n",
       "      <td>9360819</td>\n",
       "      <td>9360819</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>9</td>\n",
       "      <td>0.188</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PZP</td>\n",
       "      <td>ENST00000261336.2:c.83+36G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>119</td>\n",
       "      <td>12</td>\n",
       "      <td>25398281</td>\n",
       "      <td>25398281</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>14</td>\n",
       "      <td>0.326</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>ENST00000256078.4:c.38G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly13Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.44)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>120</td>\n",
       "      <td>12</td>\n",
       "      <td>56477631</td>\n",
       "      <td>56477631</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>71</td>\n",
       "      <td>23</td>\n",
       "      <td>0.245</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ERBB3</td>\n",
       "      <td>ENST00000267101.3:c.179T&gt;A</td>\n",
       "      <td>ENSP00000267101.3:p.Met60Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.147)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>121</td>\n",
       "      <td>12</td>\n",
       "      <td>57881973</td>\n",
       "      <td>57881973</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>12</td>\n",
       "      <td>0.333</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MARS</td>\n",
       "      <td>ENST00000262027.5:c.100G&gt;C</td>\n",
       "      <td>ENSP00000262027.5:p.Gly34Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.38)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>122</td>\n",
       "      <td>12</td>\n",
       "      <td>104190820</td>\n",
       "      <td>104190820</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NT5DC3</td>\n",
       "      <td>ENST00000392876.3:c.616-11C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>123</td>\n",
       "      <td>12</td>\n",
       "      <td>120966921</td>\n",
       "      <td>120966921</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>0.073</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>COQ5</td>\n",
       "      <td>ENST00000288532.6:c.24T&gt;C</td>\n",
       "      <td>ENSP00000288532.6:p.Ala8%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>124</td>\n",
       "      <td>12</td>\n",
       "      <td>124311390</td>\n",
       "      <td>124311390</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>3</td>\n",
       "      <td>0.057</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DNAH10</td>\n",
       "      <td>ENST00000409039.3:c.3982T&gt;C</td>\n",
       "      <td>ENSP00000386770.3:p.Phe1328Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>125</td>\n",
       "      <td>13</td>\n",
       "      <td>58207656</td>\n",
       "      <td>58207656</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>23</td>\n",
       "      <td>0.329</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PCDH17</td>\n",
       "      <td>ENST00000377918.3:c.976C&gt;A</td>\n",
       "      <td>ENSP00000367151.3:p.Leu326Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.962)</td>\n",
       "      <td>tolerated(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>126</td>\n",
       "      <td>13</td>\n",
       "      <td>79218993</td>\n",
       "      <td>79218993</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>76</td>\n",
       "      <td>4</td>\n",
       "      <td>0.050</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>RNF219</td>\n",
       "      <td>ENST00000282003.6:c.208+4A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>127</td>\n",
       "      <td>13</td>\n",
       "      <td>99116014</td>\n",
       "      <td>99116014</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>90</td>\n",
       "      <td>5</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>STK24</td>\n",
       "      <td>ENST00000376547.3:c.896T&gt;C</td>\n",
       "      <td>ENSP00000365730.3:p.Leu299Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>128</td>\n",
       "      <td>13</td>\n",
       "      <td>112763301</td>\n",
       "      <td>112763301</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>6</td>\n",
       "      <td>0.158</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LINC00404</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>129</td>\n",
       "      <td>14</td>\n",
       "      <td>20857468</td>\n",
       "      <td>20857468</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>6</td>\n",
       "      <td>0.240</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TEP1</td>\n",
       "      <td>ENST00000262715.5:c.2466-12T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>130</td>\n",
       "      <td>14</td>\n",
       "      <td>75553697</td>\n",
       "      <td>75553697</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>13</td>\n",
       "      <td>0.289</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NEK9</td>\n",
       "      <td>ENST00000238616.5:c.2817+24C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>131</td>\n",
       "      <td>15</td>\n",
       "      <td>41059329</td>\n",
       "      <td>41059329</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>6</td>\n",
       "      <td>0.136</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DNAJC17</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>132</td>\n",
       "      <td>15</td>\n",
       "      <td>42820589</td>\n",
       "      <td>42820589</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>57</td>\n",
       "      <td>3</td>\n",
       "      <td>0.050</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>SNAP23</td>\n",
       "      <td>ENST00000249647.3:c.396G&gt;A</td>\n",
       "      <td>ENSP00000249647.3:p.Thr132%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>133</td>\n",
       "      <td>15</td>\n",
       "      <td>43821035</td>\n",
       "      <td>43821035</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.095</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MAP1A</td>\n",
       "      <td>ENST00000300231.5:c.7364C&gt;A</td>\n",
       "      <td>ENSP00000300231.5:p.Pro2455Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.964)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>134</td>\n",
       "      <td>15</td>\n",
       "      <td>48779562</td>\n",
       "      <td>48779562</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>7</td>\n",
       "      <td>0.089</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>ENST00000316623.5:c.3410G&gt;A</td>\n",
       "      <td>ENSP00000325527.5:p.Arg1137His</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>possibly_damaging(0.615)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>135</td>\n",
       "      <td>15</td>\n",
       "      <td>64222560</td>\n",
       "      <td>64222560</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>4</td>\n",
       "      <td>0.118</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DAPK2</td>\n",
       "      <td>ENST00000261891.3:c.632+46G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>136</td>\n",
       "      <td>15</td>\n",
       "      <td>68695252</td>\n",
       "      <td>68695252</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ITGA11</td>\n",
       "      <td>ENST00000315757.7:c.164+5G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>137</td>\n",
       "      <td>15</td>\n",
       "      <td>74337173</td>\n",
       "      <td>74337173</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>9</td>\n",
       "      <td>0.220</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PML</td>\n",
       "      <td>ENST00000268058.3:c.2473C&gt;T</td>\n",
       "      <td>ENSP00000268058.3:p.Arg825Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>138</td>\n",
       "      <td>15</td>\n",
       "      <td>78582270</td>\n",
       "      <td>78582270</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>4</td>\n",
       "      <td>0.118</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>WDR61</td>\n",
       "      <td>ENST00000267973.2:c.468+23T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>139</td>\n",
       "      <td>15</td>\n",
       "      <td>86122590</td>\n",
       "      <td>86122590</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>AKAP13</td>\n",
       "      <td>ENST00000361243.2:c.1291G&gt;A</td>\n",
       "      <td>ENSP00000354718.2:p.Gly431Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>140</td>\n",
       "      <td>16</td>\n",
       "      <td>2002875</td>\n",
       "      <td>2002875</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>3</td>\n",
       "      <td>0.047</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RPL3L</td>\n",
       "      <td>ENST00000268661.7:c.365G&gt;T</td>\n",
       "      <td>ENSP00000268661.7:p.Trp122Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.034)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>141</td>\n",
       "      <td>16</td>\n",
       "      <td>3775105</td>\n",
       "      <td>3775105</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>5</td>\n",
       "      <td>0.135</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CREBBP</td>\n",
       "      <td>ENST00000262367.5:c.*2614G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>142</td>\n",
       "      <td>16</td>\n",
       "      <td>4847390</td>\n",
       "      <td>4847390</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>3</td>\n",
       "      <td>0.079</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ROGDI</td>\n",
       "      <td>ENST00000322048.7:c.*71G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>143</td>\n",
       "      <td>16</td>\n",
       "      <td>22361152</td>\n",
       "      <td>22361152</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>4</td>\n",
       "      <td>0.129</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>CDR2</td>\n",
       "      <td>ENST00000268383.2:c.211G&gt;T</td>\n",
       "      <td>ENSP00000268383.2:p.Glu71Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>144</td>\n",
       "      <td>16</td>\n",
       "      <td>23428341</td>\n",
       "      <td>23428341</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>3</td>\n",
       "      <td>0.081</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>COG7</td>\n",
       "      <td>ENST00000307149.5:c.1239C&gt;A</td>\n",
       "      <td>ENSP00000305442.5:p.Thr413%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>145</td>\n",
       "      <td>16</td>\n",
       "      <td>23672585</td>\n",
       "      <td>23672585</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>4</td>\n",
       "      <td>0.060</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DCTN5</td>\n",
       "      <td>ENST00000300087.2:c.331G&gt;A</td>\n",
       "      <td>ENSP00000300087.2:p.Gly111Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.976)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>146</td>\n",
       "      <td>16</td>\n",
       "      <td>23716366</td>\n",
       "      <td>23716366</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>5</td>\n",
       "      <td>0.098</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ERN2</td>\n",
       "      <td>ENST00000256797.4:c.836G&gt;T</td>\n",
       "      <td>ENSP00000256797.4:p.Gly279Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.954)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>147</td>\n",
       "      <td>16</td>\n",
       "      <td>30900143</td>\n",
       "      <td>30900143</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>0.088</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>BCL7C</td>\n",
       "      <td>ENST00000215115.4:c.528+38A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>148</td>\n",
       "      <td>16</td>\n",
       "      <td>31092256</td>\n",
       "      <td>31092256</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PRSS53</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>149</td>\n",
       "      <td>16</td>\n",
       "      <td>34510900</td>\n",
       "      <td>34510900</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>3</td>\n",
       "      <td>0.081</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AC007353.1</td>\n",
       "      <td>ENST00000408605.1:n.38G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>150</td>\n",
       "      <td>16</td>\n",
       "      <td>55363174</td>\n",
       "      <td>55363174</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>7</td>\n",
       "      <td>0.152</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>IRX6</td>\n",
       "      <td>ENST00000290552.7:c.1284G&gt;A</td>\n",
       "      <td>ENSP00000290552.7:p.Pro428%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>151</td>\n",
       "      <td>16</td>\n",
       "      <td>56693058</td>\n",
       "      <td>56693058</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>4</td>\n",
       "      <td>0.091</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MT1F</td>\n",
       "      <td>ENST00000334350.6:c.168G&gt;A</td>\n",
       "      <td>ENSP00000334872.6:p.Lys56%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>152</td>\n",
       "      <td>16</td>\n",
       "      <td>57056150</td>\n",
       "      <td>57056150</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>4</td>\n",
       "      <td>0.148</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NLRC5</td>\n",
       "      <td>ENST00000262510.6:c.296-18C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>153</td>\n",
       "      <td>16</td>\n",
       "      <td>66655363</td>\n",
       "      <td>66655363</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CMTM4</td>\n",
       "      <td>ENST00000330687.4:c.623+602C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>154</td>\n",
       "      <td>16</td>\n",
       "      <td>81929488</td>\n",
       "      <td>81929488</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLCG2</td>\n",
       "      <td>ENST00000359376.3:c.1149C&gt;T</td>\n",
       "      <td>ENSP00000352336.3:p.Asp383%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>155</td>\n",
       "      <td>17</td>\n",
       "      <td>7123922</td>\n",
       "      <td>7123922</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>4</td>\n",
       "      <td>0.070</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DVL2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>156</td>\n",
       "      <td>17</td>\n",
       "      <td>7534912</td>\n",
       "      <td>7534912</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>3</td>\n",
       "      <td>0.120</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SAT2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>157</td>\n",
       "      <td>17</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578395</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TP53</td>\n",
       "      <td>ENST00000269305.4:c.535C&gt;T</td>\n",
       "      <td>ENSP00000269305.4:p.His179Tyr</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>158</td>\n",
       "      <td>17</td>\n",
       "      <td>10352343</td>\n",
       "      <td>10352343</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.095</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MYH4</td>\n",
       "      <td>ENST00000255381.2:c.4203G&gt;T</td>\n",
       "      <td>ENSP00000255381.2:p.Leu1401%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>159</td>\n",
       "      <td>17</td>\n",
       "      <td>18152739</td>\n",
       "      <td>18152739</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LLGL1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>160</td>\n",
       "      <td>17</td>\n",
       "      <td>29162948</td>\n",
       "      <td>29162948</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ATAD5</td>\n",
       "      <td>ENST00000321990.4:c.1849G&gt;A</td>\n",
       "      <td>ENSP00000313171.4:p.Val617Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.037)</td>\n",
       "      <td>tolerated(0.15)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>161</td>\n",
       "      <td>17</td>\n",
       "      <td>29899114</td>\n",
       "      <td>29899114</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>4</td>\n",
       "      <td>0.069</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MIR365B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>162</td>\n",
       "      <td>17</td>\n",
       "      <td>33289680</td>\n",
       "      <td>33289680</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CCT6B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>163</td>\n",
       "      <td>17</td>\n",
       "      <td>46262035</td>\n",
       "      <td>46262035</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SKAP1</td>\n",
       "      <td>ENST00000336915.6:c.567+50T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>164</td>\n",
       "      <td>17</td>\n",
       "      <td>47888885</td>\n",
       "      <td>47888885</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>3</td>\n",
       "      <td>0.125</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>KAT7</td>\n",
       "      <td>ENST00000259021.4:c.801C&gt;G</td>\n",
       "      <td>ENSP00000259021.4:p.Leu267%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>165</td>\n",
       "      <td>17</td>\n",
       "      <td>73845654</td>\n",
       "      <td>73845654</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>80</td>\n",
       "      <td>4</td>\n",
       "      <td>0.048</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>UNC13D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>166</td>\n",
       "      <td>18</td>\n",
       "      <td>12724471</td>\n",
       "      <td>12724471</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>4</td>\n",
       "      <td>0.154</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PSMG2</td>\n",
       "      <td>ENST00000317615.6:c.582-27A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>167</td>\n",
       "      <td>18</td>\n",
       "      <td>21121329</td>\n",
       "      <td>21121329</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.083</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NPC1</td>\n",
       "      <td>ENST00000269228.5:c.2314T&gt;C</td>\n",
       "      <td>ENSP00000269228.4:p.Phe772Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>168</td>\n",
       "      <td>18</td>\n",
       "      <td>22869136</td>\n",
       "      <td>22869136</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>3</td>\n",
       "      <td>0.065</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ZNF521</td>\n",
       "      <td>ENST00000361524.3:c.220+32836C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>169</td>\n",
       "      <td>18</td>\n",
       "      <td>72250923</td>\n",
       "      <td>72250923</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>5</td>\n",
       "      <td>0.106</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CNDP1</td>\n",
       "      <td>ENST00000358821.3:c.1436C&gt;T</td>\n",
       "      <td>ENSP00000351682.3:p.Ser479Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.993)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>170</td>\n",
       "      <td>18</td>\n",
       "      <td>72347107</td>\n",
       "      <td>72347107</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>4</td>\n",
       "      <td>0.133</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF407</td>\n",
       "      <td>ENST00000299687.5:c.4132G&gt;A</td>\n",
       "      <td>ENSP00000299687.4:p.Gly1378Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.01)</td>\n",
       "      <td>tolerated(0.05)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>171</td>\n",
       "      <td>18</td>\n",
       "      <td>74681694</td>\n",
       "      <td>74681694</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>18</td>\n",
       "      <td>4</td>\n",
       "      <td>0.182</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ZNF236</td>\n",
       "      <td>ENST00000253159.8:c.*1399C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>172</td>\n",
       "      <td>19</td>\n",
       "      <td>1976170</td>\n",
       "      <td>1976170</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CSNK1G2</td>\n",
       "      <td>ENST00000255641.8:c.188-2134G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>173</td>\n",
       "      <td>19</td>\n",
       "      <td>11347079</td>\n",
       "      <td>11347079</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>6</td>\n",
       "      <td>0.122</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C19orf80</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>174</td>\n",
       "      <td>19</td>\n",
       "      <td>17949188</td>\n",
       "      <td>17949188</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>4</td>\n",
       "      <td>0.133</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>ENST00000458235.1:c.1453C&gt;A</td>\n",
       "      <td>ENSP00000391676.1:p.Leu485Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.931)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>175</td>\n",
       "      <td>19</td>\n",
       "      <td>18555576</td>\n",
       "      <td>18555576</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>7</td>\n",
       "      <td>0.127</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ELL</td>\n",
       "      <td>ENST00000262809.4:c.1852C&gt;A</td>\n",
       "      <td>ENSP00000262809.3:p.Gln618Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.104)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>176</td>\n",
       "      <td>19</td>\n",
       "      <td>19381214</td>\n",
       "      <td>19381214</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>5</td>\n",
       "      <td>0.074</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TM6SF2</td>\n",
       "      <td>ENST00000389363.4:c.237C&gt;A</td>\n",
       "      <td>ENSP00000374014.2:p.Leu79%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>177</td>\n",
       "      <td>19</td>\n",
       "      <td>30936021</td>\n",
       "      <td>30936021</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF536</td>\n",
       "      <td>ENST00000355537.3:c.1552G&gt;T</td>\n",
       "      <td>ENSP00000347730.1:p.Val518Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.006)</td>\n",
       "      <td>tolerated(0.49)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>178</td>\n",
       "      <td>19</td>\n",
       "      <td>41829716</td>\n",
       "      <td>41829716</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>4</td>\n",
       "      <td>0.121</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CCDC97</td>\n",
       "      <td>ENST00000269967.3:c.*1096C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>179</td>\n",
       "      <td>19</td>\n",
       "      <td>42219585</td>\n",
       "      <td>42219585</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>3</td>\n",
       "      <td>0.055</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>CEACAM5</td>\n",
       "      <td>ENST00000221992.6:c.720C&gt;T</td>\n",
       "      <td>ENSP00000221992.5:p.Pro240%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>180</td>\n",
       "      <td>19</td>\n",
       "      <td>49243761</td>\n",
       "      <td>49243761</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>4</td>\n",
       "      <td>0.061</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RASIP1</td>\n",
       "      <td>ENST00000222145.4:c.-5+17C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>181</td>\n",
       "      <td>19</td>\n",
       "      <td>54802105</td>\n",
       "      <td>54802105</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>8</td>\n",
       "      <td>0.119</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>LILRA3</td>\n",
       "      <td>ENST00000251390.3:c.1083G&gt;T</td>\n",
       "      <td>ENSP00000251390.3:p.Gly361%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>182</td>\n",
       "      <td>19</td>\n",
       "      <td>56372947</td>\n",
       "      <td>56372947</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>3</td>\n",
       "      <td>0.075</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NLRP4</td>\n",
       "      <td>ENST00000301295.6:c.2018+34C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>183</td>\n",
       "      <td>20</td>\n",
       "      <td>3363056</td>\n",
       "      <td>3363056</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>4</td>\n",
       "      <td>0.100</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C20orf194</td>\n",
       "      <td>ENST00000252032.9:c.163+18G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>184</td>\n",
       "      <td>20</td>\n",
       "      <td>3739324</td>\n",
       "      <td>3739324</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>5</td>\n",
       "      <td>0.086</td>\n",
       "      <td>splice_region_variant&amp;synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>C20orf27</td>\n",
       "      <td>ENST00000217195.8:c.96C&gt;A</td>\n",
       "      <td>ENSP00000217195.8:p.Gly32%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>185</td>\n",
       "      <td>20</td>\n",
       "      <td>35555560</td>\n",
       "      <td>35555560</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>0.103</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SAMHD1</td>\n",
       "      <td>ENST00000262878.4:c.696+25C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>186</td>\n",
       "      <td>20</td>\n",
       "      <td>58520992</td>\n",
       "      <td>58520992</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>20</td>\n",
       "      <td>0.263</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FAM217B</td>\n",
       "      <td>ENST00000358293.3:c.*842C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>187</td>\n",
       "      <td>21</td>\n",
       "      <td>17138323</td>\n",
       "      <td>17138323</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>3</td>\n",
       "      <td>0.071</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>USP25</td>\n",
       "      <td>ENST00000285679.6:c.131A&gt;T</td>\n",
       "      <td>ENSP00000285679.6:p.Asn44Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.548)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>188</td>\n",
       "      <td>21</td>\n",
       "      <td>44180524</td>\n",
       "      <td>44180524</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>5</td>\n",
       "      <td>0.114</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PDE9A</td>\n",
       "      <td>ENST00000291539.6:c.984C&gt;A</td>\n",
       "      <td>ENSP00000291539.6:p.Val328%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>189</td>\n",
       "      <td>21</td>\n",
       "      <td>44841662</td>\n",
       "      <td>44841662</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>20</td>\n",
       "      <td>4</td>\n",
       "      <td>0.167</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SIK1</td>\n",
       "      <td>ENST00000270162.6:c.355G&gt;C</td>\n",
       "      <td>ENSP00000270162.6:p.Gly119Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.981)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>190</td>\n",
       "      <td>22</td>\n",
       "      <td>22730549</td>\n",
       "      <td>22730549</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>4</td>\n",
       "      <td>0.143</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>IGLV5-45</td>\n",
       "      <td>ENST00000390296.2:c.47-3C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>191</td>\n",
       "      <td>22</td>\n",
       "      <td>24237610</td>\n",
       "      <td>24237610</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.077</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MIF</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>192</td>\n",
       "      <td>22</td>\n",
       "      <td>32112183</td>\n",
       "      <td>32112183</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>17</td>\n",
       "      <td>0.415</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>PRR14L</td>\n",
       "      <td>ENST00000327423.6:c.1642C&gt;T</td>\n",
       "      <td>ENSP00000331845.6:p.Gln548Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>193</td>\n",
       "      <td>22</td>\n",
       "      <td>47103746</td>\n",
       "      <td>47103746</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CERK</td>\n",
       "      <td>ENST00000216264.8:c.709C&gt;T</td>\n",
       "      <td>ENSP00000216264.8:p.Pro237Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>194</td>\n",
       "      <td>22</td>\n",
       "      <td>50589167</td>\n",
       "      <td>50589167</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>3</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MOV10L1</td>\n",
       "      <td>ENST00000262794.5:c.2731G&gt;A</td>\n",
       "      <td>ENSP00000262794.5:p.Val911Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.987)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>195</td>\n",
       "      <td>22</td>\n",
       "      <td>50591387</td>\n",
       "      <td>50591387</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MOV10L1</td>\n",
       "      <td>ENST00000262794.5:c.2893-87C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>196</td>\n",
       "      <td>22</td>\n",
       "      <td>50751904</td>\n",
       "      <td>50751904</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>0.083</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DENND6B</td>\n",
       "      <td>ENST00000413817.3:c.1318C&gt;A</td>\n",
       "      <td>ENSP00000391524.2:p.Pro440Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.997)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>197</td>\n",
       "      <td>X</td>\n",
       "      <td>11783924</td>\n",
       "      <td>11783924</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>4</td>\n",
       "      <td>0.067</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MSL3</td>\n",
       "      <td>ENST00000312196.4:c.1171+76G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>198</td>\n",
       "      <td>X</td>\n",
       "      <td>13056613</td>\n",
       "      <td>13056613</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>5</td>\n",
       "      <td>0.078</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>FAM9C</td>\n",
       "      <td>ENST00000333995.3:c.448C&gt;T</td>\n",
       "      <td>ENSP00000334430.3:p.Gln150Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>199</td>\n",
       "      <td>X</td>\n",
       "      <td>16170711</td>\n",
       "      <td>16170711</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>4</td>\n",
       "      <td>0.154</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GRPR</td>\n",
       "      <td>ENST00000380289.2:c.1098C&gt;A</td>\n",
       "      <td>ENSP00000369643.2:p.Pro366%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>200</td>\n",
       "      <td>X</td>\n",
       "      <td>34657455</td>\n",
       "      <td>34657455</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TMEM47</td>\n",
       "      <td>ENST00000275954.3:c.276T&gt;A</td>\n",
       "      <td>ENSP00000275954.3:p.Ile92%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>201</td>\n",
       "      <td>X</td>\n",
       "      <td>69261846</td>\n",
       "      <td>69261846</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>0.088</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AWAT2</td>\n",
       "      <td>ENST00000276101.3:c.848-34C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>202</td>\n",
       "      <td>X</td>\n",
       "      <td>69503454</td>\n",
       "      <td>69503454</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>0.103</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PDZD11</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>203</td>\n",
       "      <td>X</td>\n",
       "      <td>106844796</td>\n",
       "      <td>106844796</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FRMPD3</td>\n",
       "      <td>ENST00000276185.4:c.3626A&gt;T</td>\n",
       "      <td>ENSP00000276185.4:p.Gln1209Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated_low_confidence(0.11)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>204</td>\n",
       "      <td>X</td>\n",
       "      <td>134567277</td>\n",
       "      <td>134567277</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>110</td>\n",
       "      <td>18</td>\n",
       "      <td>0.141</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CROCCP1</td>\n",
       "      <td>ENST00000426910.1:n.393G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>205</td>\n",
       "      <td>1</td>\n",
       "      <td>43913688</td>\n",
       "      <td>43913688</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.130</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>HYI</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>206</td>\n",
       "      <td>1</td>\n",
       "      <td>70539561</td>\n",
       "      <td>70539561</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>6</td>\n",
       "      <td>0.202</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LRRC7</td>\n",
       "      <td>ENST00000035383.5:c.4117-2199A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>207</td>\n",
       "      <td>1</td>\n",
       "      <td>197141253</td>\n",
       "      <td>197141253</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>5</td>\n",
       "      <td>0.119</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ZBTB41</td>\n",
       "      <td>ENST00000367405.4:c.2074+37T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>208</td>\n",
       "      <td>2</td>\n",
       "      <td>175289432</td>\n",
       "      <td>175289432</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>10</td>\n",
       "      <td>0.354</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SCRN3</td>\n",
       "      <td>ENST00000272732.6:c.1092+55C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>209</td>\n",
       "      <td>2</td>\n",
       "      <td>201501814</td>\n",
       "      <td>201501814</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>4</td>\n",
       "      <td>0.249</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AOX1</td>\n",
       "      <td>ENST00000374700.2:c.2480+47G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>210</td>\n",
       "      <td>3</td>\n",
       "      <td>14958865</td>\n",
       "      <td>14958865</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>4</td>\n",
       "      <td>0.123</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FGD5</td>\n",
       "      <td>ENST00000285046.5:c.3489+26C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>211</td>\n",
       "      <td>3</td>\n",
       "      <td>108301916</td>\n",
       "      <td>108301916</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.130</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KIAA1524</td>\n",
       "      <td>ENST00000295746.8:c.265A&gt;G</td>\n",
       "      <td>ENSP00000295746.7:p.Thr89Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.108)</td>\n",
       "      <td>tolerated(0.14)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>212</td>\n",
       "      <td>3</td>\n",
       "      <td>178936091</td>\n",
       "      <td>178936091</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>9</td>\n",
       "      <td>0.185</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>ENST00000263967.3:c.1633G&gt;A</td>\n",
       "      <td>ENSP00000263967.3:p.Glu545Lys</td>\n",
       "      <td>not_provided&amp;pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.909)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>213</td>\n",
       "      <td>4</td>\n",
       "      <td>74862239</td>\n",
       "      <td>74862239</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>6</td>\n",
       "      <td>0.317</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CXCL5</td>\n",
       "      <td>ENST00000296027.4:c.*1115A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>214</td>\n",
       "      <td>5</td>\n",
       "      <td>63986568</td>\n",
       "      <td>63986568</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>16</td>\n",
       "      <td>3</td>\n",
       "      <td>0.190</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAM159B</td>\n",
       "      <td>ENST00000389074.5:c.118G&gt;T</td>\n",
       "      <td>ENSP00000373726.5:p.Ala40Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.136)</td>\n",
       "      <td>tolerated(0.83)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>215</td>\n",
       "      <td>5</td>\n",
       "      <td>145838440</td>\n",
       "      <td>145838440</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>4</td>\n",
       "      <td>0.135</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TCERG1</td>\n",
       "      <td>ENST00000296702.5:c.439-7G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>216</td>\n",
       "      <td>6</td>\n",
       "      <td>46107299</td>\n",
       "      <td>46107299</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>4</td>\n",
       "      <td>0.155</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ENPP4</td>\n",
       "      <td>ENST00000321037.4:c.-22G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>217</td>\n",
       "      <td>6</td>\n",
       "      <td>55739553</td>\n",
       "      <td>55739553</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>5</td>\n",
       "      <td>0.120</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>BMP5</td>\n",
       "      <td>ENST00000370830.3:c.111T&gt;C</td>\n",
       "      <td>ENSP00000359866.3:p.Ser37%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>218</td>\n",
       "      <td>7</td>\n",
       "      <td>72658359</td>\n",
       "      <td>72658359</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>20</td>\n",
       "      <td>0.310</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GTF2IRD2P1</td>\n",
       "      <td>ENST00000425256.1:n.1553C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>219</td>\n",
       "      <td>7</td>\n",
       "      <td>150684129</td>\n",
       "      <td>150684129</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>13</td>\n",
       "      <td>0.325</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NOS3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>220</td>\n",
       "      <td>9</td>\n",
       "      <td>101061665</td>\n",
       "      <td>101061665</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>66</td>\n",
       "      <td>4</td>\n",
       "      <td>0.069</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GABBR2</td>\n",
       "      <td>ENST00000259455.2:c.2413-30C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>221</td>\n",
       "      <td>9</td>\n",
       "      <td>139266496</td>\n",
       "      <td>139266496</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>3</td>\n",
       "      <td>0.211</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SNAPC4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>222</td>\n",
       "      <td>11</td>\n",
       "      <td>67132464</td>\n",
       "      <td>67132464</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>6</td>\n",
       "      <td>0.139</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CLCF1</td>\n",
       "      <td>ENST00000312438.7:c.*143C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>223</td>\n",
       "      <td>11</td>\n",
       "      <td>67375868</td>\n",
       "      <td>67375868</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>6</td>\n",
       "      <td>0.113</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NDUFV1</td>\n",
       "      <td>ENST00000322776.6:c.74C&gt;T</td>\n",
       "      <td>ENSP00000322450.6:p.Thr25Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.021)</td>\n",
       "      <td>deleterious_low_confidence(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>224</td>\n",
       "      <td>12</td>\n",
       "      <td>121201409</td>\n",
       "      <td>121201409</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>0.144</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SPPL3</td>\n",
       "      <td>ENST00000353487.2:c.*1393A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>225</td>\n",
       "      <td>13</td>\n",
       "      <td>58240595</td>\n",
       "      <td>58240595</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>74</td>\n",
       "      <td>4</td>\n",
       "      <td>0.062</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PCDH17</td>\n",
       "      <td>ENST00000377918.3:c.2566-7T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>226</td>\n",
       "      <td>16</td>\n",
       "      <td>30125840</td>\n",
       "      <td>30125840</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>5</td>\n",
       "      <td>0.117</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MAPK3</td>\n",
       "      <td>ENST00000263025.4:c.*222C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>227</td>\n",
       "      <td>16</td>\n",
       "      <td>89858916</td>\n",
       "      <td>89858916</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.112</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FANCA</td>\n",
       "      <td>ENST00000389301.3:c.1046C&gt;T</td>\n",
       "      <td>ENSP00000373952.3:p.Ala349Val</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>benign(0.138)</td>\n",
       "      <td>tolerated(0.14)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>228</td>\n",
       "      <td>17</td>\n",
       "      <td>74056413</td>\n",
       "      <td>74056413</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>4</td>\n",
       "      <td>0.110</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>SRP68</td>\n",
       "      <td>ENST00000307877.2:c.813G&gt;A</td>\n",
       "      <td>ENSP00000312066.1:p.Glu271%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>229</td>\n",
       "      <td>17</td>\n",
       "      <td>79571705</td>\n",
       "      <td>79571705</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>0.147</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NPLOC4</td>\n",
       "      <td>ENST00000331134.6:c.703G&gt;T</td>\n",
       "      <td>ENSP00000331487.5:p.Asp235Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.949)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>230</td>\n",
       "      <td>19</td>\n",
       "      <td>2643304</td>\n",
       "      <td>2643304</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>7</td>\n",
       "      <td>0.139</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GNG7</td>\n",
       "      <td>ENST00000382159.3:c.-78+2918C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>231</td>\n",
       "      <td>19</td>\n",
       "      <td>49932044</td>\n",
       "      <td>49932044</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>11</td>\n",
       "      <td>0.255</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SLC17A7</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>232</td>\n",
       "      <td>20</td>\n",
       "      <td>34535373</td>\n",
       "      <td>34535373</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.115</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PHF20</td>\n",
       "      <td>ENST00000374012.3:c.2897-34T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>233</td>\n",
       "      <td>20</td>\n",
       "      <td>61867626</td>\n",
       "      <td>61867626</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>88</td>\n",
       "      <td>5</td>\n",
       "      <td>0.063</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>BIRC7</td>\n",
       "      <td>ENST00000217169.3:c.178C&gt;T</td>\n",
       "      <td>ENSP00000217169.3:p.Arg60Trp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.895)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>234</td>\n",
       "      <td>X</td>\n",
       "      <td>51151007</td>\n",
       "      <td>51151007</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>13</td>\n",
       "      <td>0.258</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CXorf67</td>\n",
       "      <td>ENST00000342995.2:c.1139C&gt;T</td>\n",
       "      <td>ENSP00000342680.2:p.Ala380Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.013)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    seqnames      start        end  width strand   AD  AD.1     AF  \\\n",
       "0          1    1961657    1961657      1      *   51    10  0.164   \n",
       "1          1   16259538   16259538      1      *   33     4  0.108   \n",
       "2          1   16461628   16461628      1      *   26     5  0.161   \n",
       "3          1   25880505   25880505      1      *   41     9  0.180   \n",
       "4          1   28816442   28816442      1      *   50     4  0.074   \n",
       "5          1   46501061   46501061      1      *   22     9  0.290   \n",
       "6          1   53683815   53683815      1      *   29     4  0.121   \n",
       "7          1   55088984   55088984      1      *   71    10  0.123   \n",
       "8          1   86591188   86591188      1      *   38     5  0.116   \n",
       "9          1  112846126  112846126      1      *   22     5  0.185   \n",
       "10         1  114193692  114193692      1      *   43     6  0.122   \n",
       "11         1  118634278  118634278      1      *   53     4  0.070   \n",
       "12         1  149919163  149919163      1      *   63     5  0.074   \n",
       "13         1  149939389  149939389      1      *   58     3  0.049   \n",
       "14         1  152850751  152850751      1      *   87    28  0.243   \n",
       "15         1  152884003  152884003      1      *   49     4  0.075   \n",
       "16         1  153791394  153791394      1      *   35     4  0.103   \n",
       "17         1  166246456  166246456      1      *   70     3  0.041   \n",
       "18         1  172571853  172571853      1      *   28     4  0.125   \n",
       "19         1  180217567  180217567      1      *   46     3  0.061   \n",
       "20         1  193219058  193219058      1      *   54     4  0.069   \n",
       "21         2   24050735   24050735      1      *   79     5  0.060   \n",
       "22         2   54076289   54076289      1      *   44    14  0.241   \n",
       "23         2   56227804   56227804      1      *   41    13  0.241   \n",
       "24         2   64863545   64863545      1      *   33     4  0.108   \n",
       "25         2  120850877  120850877      1      *   42     3  0.067   \n",
       "26         2  122161436  122161436      1      *   34     4  0.105   \n",
       "27         2  189856266  189856266      1      *   38     4  0.095   \n",
       "28         2  209168996  209168996      1      *   61     4  0.062   \n",
       "29         2  220439634  220439634      1      *   83     4  0.046   \n",
       "30         2  239010520  239010520      1      *   49     4  0.075   \n",
       "31         3   31665184   31665184      1      *   28     8  0.222   \n",
       "32         3   37574951   37574951      1      *   32     4  0.111   \n",
       "33         3   41759288   41759288      1      *   50     4  0.074   \n",
       "34         3   49137421   49137421      1      *   48     4  0.077   \n",
       "35         3  113697105  113697105      1      *   37     4  0.098   \n",
       "36         3  124692689  124692689      1      *   54     4  0.069   \n",
       "37         3  132242359  132242359      1      *   65     3  0.044   \n",
       "38         3  134644699  134644699      1      *   61     5  0.076   \n",
       "39         3  151171301  151171301      1      *   43     3  0.065   \n",
       "40         3  180041515  180041515      1      *   37    19  0.339   \n",
       "41         3  190167616  190167616      1      *   60     7  0.104   \n",
       "42         4   55988729   55988729      1      *   46     4  0.080   \n",
       "43         4   56308687   56308687      1      *   52     4  0.071   \n",
       "44         4  140599783  140599783      1      *   51     4  0.073   \n",
       "45         4  142154850  142154850      1      *   27     4  0.129   \n",
       "46         4  177683539  177683539      1      *   23     3  0.115   \n",
       "47         4  185339813  185339813      1      *   56    11  0.164   \n",
       "48         5   10430082   10430082      1      *   50    10  0.167   \n",
       "49         5   10638114   10638114      1      *   33     3  0.083   \n",
       "50         5  121297625  121297625      1      *   49     3  0.058   \n",
       "51         5  126781439  126781439      1      *   30     3  0.091   \n",
       "52         5  131940554  131940554      1      *   42     4  0.087   \n",
       "53         5  140251674  140251674      1      *   32     4  0.111   \n",
       "54         5  140594488  140594488      1      *   42     6  0.125   \n",
       "55         5  150924038  150924038      1      *   57     5  0.081   \n",
       "56         6    3983819    3983819      1      *   25     4  0.138   \n",
       "57         6   24873214   24873214      1      *   32     4  0.111   \n",
       "58         6   29769880   29769880      1      *   32     4  0.111   \n",
       "59         6   33244788   33244788      1      *   31     7  0.184   \n",
       "60         6   34840222   34840222      1      *   25     4  0.138   \n",
       "61         6   35436741   35436741      1      *   33     4  0.108   \n",
       "62         6   36946386   36946386      1      *   39     4  0.093   \n",
       "63         6   42985846   42985846      1      *   48     4  0.077   \n",
       "64         6   43180949   43180949      1      *   54    10  0.156   \n",
       "65         6   52051109   52051109      1      *   44     4  0.083   \n",
       "66         6   80383283   80383283      1      *   35     4  0.103   \n",
       "67         6   89974138   89974138      1      *   50     6  0.107   \n",
       "68         6  111587333  111587333      1      *   30     4  0.118   \n",
       "69         7     565436     565436      1      *   41     4  0.089   \n",
       "70         7   55260474   55260474      1      *   63     3  0.045   \n",
       "71         7   65553885   65553885      1      *   46     4  0.080   \n",
       "72         7   93125190   93125190      1      *   43     4  0.085   \n",
       "73         7  100029092  100029092      1      *   68     9  0.117   \n",
       "74         7  100278679  100278679      1      *   99     5  0.048   \n",
       "75         7  100817842  100817842      1      *  131    21  0.138   \n",
       "76         7  113558433  113558433      1      *   55     3  0.052   \n",
       "77         7  135333354  135333354      1      *   36     3  0.077   \n",
       "78         7  136978123  136978123      1      *   88     8  0.083   \n",
       "79         7  136982347  136982347      1      *   66     5  0.070   \n",
       "80         7  139725986  139725986      1      *   58     4  0.065   \n",
       "81         7  140064270  140064270      1      *   30     3  0.091   \n",
       "82         7  143098692  143098692      1      *   47     4  0.078   \n",
       "83         8   22464198   22464198      1      *   31     3  0.088   \n",
       "84         8   41543691   41543691      1      *   35    18  0.340   \n",
       "85         8   75765469   75765469      1      *   61     3  0.047   \n",
       "86         8   86554927   86554927      1      *   25     3  0.107   \n",
       "87         8  144688268  144688268      1      *   94     4  0.041   \n",
       "88         8  144992103  144992103      1      *   32     4  0.111   \n",
       "89         9   85615980   85615980      1      *   52     6  0.103   \n",
       "90         9   96030048   96030048      1      *   54     5  0.085   \n",
       "91         9  107288802  107288802      1      *   38     3  0.073   \n",
       "92         9  140450159  140450159      1      *   28     3  0.097   \n",
       "93         9  140473194  140473194      1      *   17     3  0.150   \n",
       "94        10    5567864    5567864      1      *   25     3  0.107   \n",
       "95        10    6528087    6528087      1      *   53     4  0.070   \n",
       "96        10    7621700    7621700      1      *   37    11  0.229   \n",
       "97        10   22615352   22615352      1      *   51     4  0.073   \n",
       "98        10   33544735   33544735      1      *   44     3  0.064   \n",
       "99        10  115610067  115610067      1      *   44     5  0.102   \n",
       "100       10  116232769  116232769      1      *   62     4  0.061   \n",
       "101       10  126490383  126490383      1      *   25     3  0.107   \n",
       "102       10  128829989  128829989      1      *   49     3  0.058   \n",
       "103       10  128935358  128935358      1      *   53     3  0.054   \n",
       "104       10  135003241  135003241      1      *   47    13  0.217   \n",
       "105       11     562421     562421      1      *   59     4  0.063   \n",
       "106       11    6291067    6291067      1      *   31     4  0.114   \n",
       "107       11   18755185   18755185      1      *   62     3  0.046   \n",
       "108       11   56015751   56015751      1      *   32     4  0.111   \n",
       "109       11   59271320   59271320      1      *   44     7  0.137   \n",
       "110       11   59861497   59861497      1      *   46     4  0.080   \n",
       "111       11   62543243   62543243      1      *   35     4  0.103   \n",
       "112       11   68676071   68676071      1      *   38     5  0.116   \n",
       "113       11   75438449   75438449      1      *   21     7  0.250   \n",
       "114       11   86662308   86662308      1      *   36     4  0.100   \n",
       "115       11  124189456  124189456      1      *   29     9  0.237   \n",
       "116       11  133651806  133651806      1      *   32     5  0.135   \n",
       "117       12    8759488    8759488      1      *   28     4  0.125   \n",
       "118       12    9360819    9360819      1      *   39     9  0.188   \n",
       "119       12   25398281   25398281      1      *   29    14  0.326   \n",
       "120       12   56477631   56477631      1      *   71    23  0.245   \n",
       "121       12   57881973   57881973      1      *   24    12  0.333   \n",
       "122       12  104190820  104190820      1      *   47     4  0.078   \n",
       "123       12  120966921  120966921      1      *   38     3  0.073   \n",
       "124       12  124311390  124311390      1      *   50     3  0.057   \n",
       "125       13   58207656   58207656      1      *   47    23  0.329   \n",
       "126       13   79218993   79218993      1      *   76     4  0.050   \n",
       "127       13   99116014   99116014      1      *   90     5  0.053   \n",
       "128       13  112763301  112763301      1      *   32     6  0.158   \n",
       "129       14   20857468   20857468      1      *   19     6  0.240   \n",
       "130       14   75553697   75553697      1      *   32    13  0.289   \n",
       "131       15   41059329   41059329      1      *   38     6  0.136   \n",
       "132       15   42820589   42820589      1      *   57     3  0.050   \n",
       "133       15   43821035   43821035      1      *   38     4  0.095   \n",
       "134       15   48779562   48779562      1      *   72     7  0.089   \n",
       "135       15   64222560   64222560      1      *   30     4  0.118   \n",
       "136       15   68695252   68695252      1      *   47     4  0.078   \n",
       "137       15   74337173   74337173      1      *   32     9  0.220   \n",
       "138       15   78582270   78582270      1      *   30     4  0.118   \n",
       "139       15   86122590   86122590      1      *   29     3  0.094   \n",
       "140       16    2002875    2002875      1      *   61     3  0.047   \n",
       "141       16    3775105    3775105      1      *   32     5  0.135   \n",
       "142       16    4847390    4847390      1      *   35     3  0.079   \n",
       "143       16   22361152   22361152      1      *   27     4  0.129   \n",
       "144       16   23428341   23428341      1      *   34     3  0.081   \n",
       "145       16   23672585   23672585      1      *   63     4  0.060   \n",
       "146       16   23716366   23716366      1      *   46     5  0.098   \n",
       "147       16   30900143   30900143      1      *   31     3  0.088   \n",
       "148       16   31092256   31092256      1      *   59     3  0.048   \n",
       "149       16   34510900   34510900      1      *   34     3  0.081   \n",
       "150       16   55363174   55363174      1      *   39     7  0.152   \n",
       "151       16   56693058   56693058      1      *   40     4  0.091   \n",
       "152       16   57056150   57056150      1      *   23     4  0.148   \n",
       "153       16   66655363   66655363      1      *   37     4  0.098   \n",
       "154       16   81929488   81929488      1      *   42     3  0.067   \n",
       "155       17    7123922    7123922      1      *   53     4  0.070   \n",
       "156       17    7534912    7534912      1      *   22     3  0.120   \n",
       "157       17    7578395    7578395      1      *   23     7  0.233   \n",
       "158       17   10352343   10352343      1      *   38     4  0.095   \n",
       "159       17   18152739   18152739      1      *   42     3  0.067   \n",
       "160       17   29162948   29162948      1      *   28     4  0.125   \n",
       "161       17   29899114   29899114      1      *   54     4  0.069   \n",
       "162       17   33289680   33289680      1      *   60     3  0.048   \n",
       "163       17   46262035   46262035      1      *   33     4  0.108   \n",
       "164       17   47888885   47888885      1      *   21     3  0.125   \n",
       "165       17   73845654   73845654      1      *   80     4  0.048   \n",
       "166       18   12724471   12724471      1      *   22     4  0.154   \n",
       "167       18   21121329   21121329      1      *   33     3  0.083   \n",
       "168       18   22869136   22869136      1      *   43     3  0.065   \n",
       "169       18   72250923   72250923      1      *   42     5  0.106   \n",
       "170       18   72347107   72347107      1      *   26     4  0.133   \n",
       "171       18   74681694   74681694      1      *   18     4  0.182   \n",
       "172       19    1976170    1976170      1      *   28     4  0.125   \n",
       "173       19   11347079   11347079      1      *   43     6  0.122   \n",
       "174       19   17949188   17949188      1      *   26     4  0.133   \n",
       "175       19   18555576   18555576      1      *   48     7  0.127   \n",
       "176       19   19381214   19381214      1      *   63     5  0.074   \n",
       "177       19   30936021   30936021      1      *   29     3  0.094   \n",
       "178       19   41829716   41829716      1      *   29     4  0.121   \n",
       "179       19   42219585   42219585      1      *   52     3  0.055   \n",
       "180       19   49243761   49243761      1      *   62     4  0.061   \n",
       "181       19   54802105   54802105      1      *   59     8  0.119   \n",
       "182       19   56372947   56372947      1      *   37     3  0.075   \n",
       "183       20    3363056    3363056      1      *   36     4  0.100   \n",
       "184       20    3739324    3739324      1      *   53     5  0.086   \n",
       "185       20   35555560   35555560      1      *   26     3  0.103   \n",
       "186       20   58520992   58520992      1      *   56    20  0.263   \n",
       "187       21   17138323   17138323      1      *   39     3  0.071   \n",
       "188       21   44180524   44180524      1      *   39     5  0.114   \n",
       "189       21   44841662   44841662      1      *   20     4  0.167   \n",
       "190       22   22730549   22730549      1      *   24     4  0.143   \n",
       "191       22   24237610   24237610      1      *   36     3  0.077   \n",
       "192       22   32112183   32112183      1      *   24    17  0.415   \n",
       "193       22   47103746   47103746      1      *   42     3  0.067   \n",
       "194       22   50589167   50589167      1      *   54     3  0.053   \n",
       "195       22   50591387   50591387      1      *   23     7  0.233   \n",
       "196       22   50751904   50751904      1      *   44     4  0.083   \n",
       "197        X   11783924   11783924      1      *   56     4  0.067   \n",
       "198        X   13056613   13056613      1      *   59     5  0.078   \n",
       "199        X   16170711   16170711      1      *   22     4  0.154   \n",
       "200        X   34657455   34657455      1      *   47     4  0.078   \n",
       "201        X   69261846   69261846      1      *   31     3  0.088   \n",
       "202        X   69503454   69503454      1      *   26     3  0.103   \n",
       "203        X  106844796  106844796      1      *   37     4  0.098   \n",
       "204        X  134567277  134567277      1      *  110    18  0.141   \n",
       "205        1   43913688   43913688      1      *   32     4  0.130   \n",
       "206        1   70539561   70539561      1      *   26     6  0.202   \n",
       "207        1  197141253  197141253      1      *   37     5  0.119   \n",
       "208        2  175289432  175289432      1      *   19    10  0.354   \n",
       "209        2  201501814  201501814      1      *   14     4  0.249   \n",
       "210        3   14958865   14958865      1      *   34     4  0.123   \n",
       "211        3  108301916  108301916      1      *   32     4  0.130   \n",
       "212        3  178936091  178936091      1      *   43     9  0.185   \n",
       "213        4   74862239   74862239      1      *   14     6  0.317   \n",
       "214        5   63986568   63986568      1      *   16     3  0.190   \n",
       "215        5  145838440  145838440      1      *   31     4  0.135   \n",
       "216        6   46107299   46107299      1      *   26     4  0.155   \n",
       "217        6   55739553   55739553      1      *   42     5  0.120   \n",
       "218        7   72658359   72658359      1      *   46    20  0.310   \n",
       "219        7  150684129  150684129      1      *   28    13  0.325   \n",
       "220        9  101061665  101061665      1      *   66     4  0.069   \n",
       "221        9  139266496  139266496      1      *   14     3  0.211   \n",
       "222       11   67132464   67132464      1      *   42     6  0.139   \n",
       "223       11   67375868   67375868      1      *   53     6  0.113   \n",
       "224       12  121201409  121201409      1      *   34     5  0.144   \n",
       "225       13   58240595   58240595      1      *   74     4  0.062   \n",
       "226       16   30125840   30125840      1      *   44     5  0.117   \n",
       "227       16   89858916   89858916      1      *   38     4  0.112   \n",
       "228       17   74056413   74056413      1      *   39     4  0.110   \n",
       "229       17   79571705   79571705      1      *   33     5  0.147   \n",
       "230       19    2643304    2643304      1      *   48     7  0.139   \n",
       "231       19   49932044   49932044      1      *   34    11  0.255   \n",
       "232       20   34535373   34535373      1      *   37     4  0.115   \n",
       "233       20   61867626   61867626      1      *   88     5  0.063   \n",
       "234        X   51151007   51151007      1      *   38    13  0.258   \n",
       "\n",
       "                                      Consequence    IMPACT         SYMBOL  \\\n",
       "0                                missense_variant  MODERATE          GABRD   \n",
       "1                                missense_variant  MODERATE           SPEN   \n",
       "2                              synonymous_variant       LOW          EPHA2   \n",
       "3                                missense_variant  MODERATE        LDLRAP1   \n",
       "4                                  intron_variant  MODIFIER        PHACTR4   \n",
       "5                         downstream_gene_variant  MODIFIER         PIK3R3   \n",
       "6                                  intron_variant  MODIFIER       C1orf123   \n",
       "7                                missense_variant  MODERATE        FAM151A   \n",
       "8                                missense_variant  MODERATE        COL24A1   \n",
       "9    intron_variant&non_coding_transcript_variant  MODIFIER    RP5-965F6.2   \n",
       "10                             synonymous_variant       LOW          MAGI3   \n",
       "11                               missense_variant  MODERATE         SPAG17   \n",
       "12                               missense_variant  MODERATE         OTUD7B   \n",
       "13                               missense_variant  MODERATE         OTUD7B   \n",
       "14                          upstream_gene_variant  MODIFIER           SMCP   \n",
       "15                               missense_variant  MODERATE            IVL   \n",
       "16                               missense_variant  MODERATE        GATAD2B   \n",
       "17             non_coding_transcript_exon_variant  MODIFIER    RP11-9L18.2   \n",
       "18                               missense_variant  MODERATE           SUCO   \n",
       "19                               missense_variant  MODERATE           LHX4   \n",
       "20                                 intron_variant  MODIFIER          CDC73   \n",
       "21                                 intron_variant  MODIFIER         ATAD2B   \n",
       "22                                 intron_variant  MODIFIER     GPR75-ASB3   \n",
       "23                        downstream_gene_variant  MODIFIER        MIR216B   \n",
       "24                               missense_variant  MODERATE        SERTAD2   \n",
       "25                                 intron_variant  MODIFIER        EPB41L5   \n",
       "26                                 intron_variant  MODIFIER         CLASP1   \n",
       "27                                 intron_variant  MODIFIER         COL3A1   \n",
       "28                             synonymous_variant       LOW        PIKFYVE   \n",
       "29                               missense_variant  MODERATE           INHA   \n",
       "30                        downstream_gene_variant  MODIFIER           SCLY   \n",
       "31                               missense_variant  MODERATE          STT3B   \n",
       "32                               missense_variant  MODERATE          ITGA9   \n",
       "33                               missense_variant  MODERATE           ULK4   \n",
       "34                               missense_variant  MODERATE           QARS   \n",
       "35                                    stop_gained      HIGH       KIAA1407   \n",
       "36                             synonymous_variant       LOW           HEG1   \n",
       "37                                 intron_variant  MODIFIER        DNAJC13   \n",
       "38                               missense_variant  MODERATE          EPHB1   \n",
       "39                               missense_variant  MODERATE         IGSF10   \n",
       "40             non_coding_transcript_exon_variant  MODIFIER  RP11-420J11.1   \n",
       "41                            5_prime_UTR_variant  MODIFIER        TMEM207   \n",
       "42                                 intron_variant  MODIFIER            KDR   \n",
       "43                                    stop_gained      HIGH          CLOCK   \n",
       "44                               missense_variant  MODERATE          MGST2   \n",
       "45                                 intron_variant  MODIFIER         ZNF330   \n",
       "46                                 intron_variant  MODIFIER          VEGFC   \n",
       "47                             synonymous_variant       LOW           IRF2   \n",
       "48                               missense_variant  MODERATE         MARCH6   \n",
       "49                                 intron_variant  MODIFIER       ANKRD33B   \n",
       "50                          upstream_gene_variant  MODIFIER         SRFBP1   \n",
       "51                                 intron_variant  MODIFIER         MEGF10   \n",
       "52                               missense_variant  MODERATE          RAD50   \n",
       "53                                 intron_variant  MODIFIER        PCDHA10   \n",
       "54                        downstream_gene_variant  MODIFIER        PCDHB12   \n",
       "55                               missense_variant  MODERATE           FAT2   \n",
       "56                            3_prime_UTR_variant  MODIFIER        C6ORF50   \n",
       "57                                 intron_variant  MODIFIER         FAM65B   \n",
       "58                        downstream_gene_variant  MODIFIER        RPL7AP7   \n",
       "59                        downstream_gene_variant  MODIFIER          WDR46   \n",
       "60                            3_prime_UTR_variant  MODIFIER       UHRF1BP1   \n",
       "61                        downstream_gene_variant  MODIFIER          FANCE   \n",
       "62       splice_region_variant&synonymous_variant       LOW          MTCH1   \n",
       "63                          upstream_gene_variant  MODIFIER          RRP36   \n",
       "64                               missense_variant  MODERATE           CUL9   \n",
       "65                          upstream_gene_variant  MODIFIER          IL17A   \n",
       "66                                 intron_variant  MODIFIER       SH3BGRL2   \n",
       "67                               missense_variant  MODERATE         GABRR2   \n",
       "68                               missense_variant  MODERATE       KIAA1919   \n",
       "69                        downstream_gene_variant  MODIFIER     AC147651.3   \n",
       "70                               missense_variant  MODERATE           EGFR   \n",
       "71                               missense_variant  MODERATE            ASL   \n",
       "72                                 intron_variant  MODIFIER          CALCR   \n",
       "73                        downstream_gene_variant  MODIFIER        PPP1R35   \n",
       "74                            3_prime_UTR_variant  MODIFIER         GIGYF1   \n",
       "75                               missense_variant  MODERATE          NAT16   \n",
       "76                               missense_variant  MODERATE        PPP1R3A   \n",
       "77                            3_prime_UTR_variant  MODIFIER         NUP205   \n",
       "78                                 intron_variant  MODIFIER            PTN   \n",
       "79                                 intron_variant  MODIFIER            PTN   \n",
       "80                                 intron_variant  MODIFIER         PARP12   \n",
       "81                               missense_variant  MODERATE        SLC37A3   \n",
       "82                               missense_variant  MODERATE          EPHA1   \n",
       "83                        downstream_gene_variant  MODIFIER        C8orf58   \n",
       "84                               missense_variant  MODERATE           ANK1   \n",
       "85                            3_prime_UTR_variant  MODIFIER           PI15   \n",
       "86                        downstream_gene_variant  MODIFIER       REXO1L8P   \n",
       "87                               missense_variant  MODERATE          PYCRL   \n",
       "88                             synonymous_variant       LOW           PLEC   \n",
       "89                               missense_variant  MODERATE          RASEF   \n",
       "90                             synonymous_variant       LOW           WNK2   \n",
       "91                               missense_variant  MODERATE         OR13C4   \n",
       "92                                 intron_variant  MODIFIER           DPH7   \n",
       "93                             synonymous_variant       LOW           DPH7   \n",
       "94                            3_prime_UTR_variant  MODIFIER         CALML3   \n",
       "95                             synonymous_variant       LOW          PRKCQ   \n",
       "96                                 intron_variant  MODIFIER          ITIH5   \n",
       "97           splice_region_variant&intron_variant       LOW           BMI1   \n",
       "98                                 intron_variant  MODIFIER           NRP1   \n",
       "99                               missense_variant  MODERATE        DCLRE1A   \n",
       "100                                   stop_gained      HIGH         ABLIM1   \n",
       "101                           5_prime_UTR_variant  MODIFIER        FAM175B   \n",
       "102                              missense_variant  MODERATE          DOCK1   \n",
       "103                                intron_variant  MODIFIER          DOCK1   \n",
       "104                                intron_variant  MODIFIER          KNDC1   \n",
       "105                         upstream_gene_variant  MODIFIER       C11orf35   \n",
       "106                              missense_variant  MODERATE          CCKBR   \n",
       "107          splice_region_variant&intron_variant       LOW          PTPN5   \n",
       "108                         upstream_gene_variant  MODIFIER          OR5T3   \n",
       "109                              missense_variant  MODERATE         OR4D11   \n",
       "110                              missense_variant  MODERATE          MS4A2   \n",
       "111     splice_region_variant&5_prime_UTR_variant       LOW          TAF6L   \n",
       "112                              missense_variant  MODERATE        IGHMBP2   \n",
       "113                                intron_variant  MODIFIER         MOGAT2   \n",
       "114                         upstream_gene_variant  MODIFIER  RP11-736K20.6   \n",
       "115                              missense_variant  MODERATE          OR8D2   \n",
       "116                       downstream_gene_variant  MODIFIER  RP11-448P19.1   \n",
       "117                            synonymous_variant       LOW          AICDA   \n",
       "118                                intron_variant  MODIFIER            PZP   \n",
       "119                              missense_variant  MODERATE           KRAS   \n",
       "120                              missense_variant  MODERATE          ERBB3   \n",
       "121                              missense_variant  MODERATE           MARS   \n",
       "122                                intron_variant  MODIFIER         NT5DC3   \n",
       "123                            synonymous_variant       LOW           COQ5   \n",
       "124                              missense_variant  MODERATE         DNAH10   \n",
       "125                              missense_variant  MODERATE         PCDH17   \n",
       "126          splice_region_variant&intron_variant       LOW         RNF219   \n",
       "127                              missense_variant  MODERATE          STK24   \n",
       "128                         upstream_gene_variant  MODIFIER      LINC00404   \n",
       "129                                intron_variant  MODIFIER           TEP1   \n",
       "130                                intron_variant  MODIFIER           NEK9   \n",
       "131                       downstream_gene_variant  MODIFIER        DNAJC17   \n",
       "132                            synonymous_variant       LOW         SNAP23   \n",
       "133                              missense_variant  MODERATE          MAP1A   \n",
       "134                              missense_variant  MODERATE           FBN1   \n",
       "135                                intron_variant  MODIFIER          DAPK2   \n",
       "136          splice_region_variant&intron_variant       LOW         ITGA11   \n",
       "137                              missense_variant  MODERATE            PML   \n",
       "138                                intron_variant  MODIFIER          WDR61   \n",
       "139                              missense_variant  MODERATE         AKAP13   \n",
       "140        missense_variant&splice_region_variant  MODERATE          RPL3L   \n",
       "141                           3_prime_UTR_variant  MODIFIER         CREBBP   \n",
       "142                           3_prime_UTR_variant  MODIFIER          ROGDI   \n",
       "143                                   stop_gained      HIGH           CDR2   \n",
       "144                            synonymous_variant       LOW           COG7   \n",
       "145                              missense_variant  MODERATE          DCTN5   \n",
       "146                              missense_variant  MODERATE           ERN2   \n",
       "147                                intron_variant  MODIFIER          BCL7C   \n",
       "148                       downstream_gene_variant  MODIFIER         PRSS53   \n",
       "149            non_coding_transcript_exon_variant  MODIFIER     AC007353.1   \n",
       "150                            synonymous_variant       LOW           IRX6   \n",
       "151                            synonymous_variant       LOW           MT1F   \n",
       "152                                intron_variant  MODIFIER          NLRC5   \n",
       "153                                intron_variant  MODIFIER          CMTM4   \n",
       "154                            synonymous_variant       LOW          PLCG2   \n",
       "155                       downstream_gene_variant  MODIFIER           DVL2   \n",
       "156                         upstream_gene_variant  MODIFIER           SAT2   \n",
       "157                              missense_variant  MODERATE           TP53   \n",
       "158                            synonymous_variant       LOW           MYH4   \n",
       "159                       downstream_gene_variant  MODIFIER          LLGL1   \n",
       "160                              missense_variant  MODERATE          ATAD5   \n",
       "161                         upstream_gene_variant  MODIFIER        MIR365B   \n",
       "162                         upstream_gene_variant  MODIFIER          CCT6B   \n",
       "163                                intron_variant  MODIFIER          SKAP1   \n",
       "164                            synonymous_variant       LOW           KAT7   \n",
       "165                         upstream_gene_variant  MODIFIER         UNC13D   \n",
       "166                                intron_variant  MODIFIER          PSMG2   \n",
       "167                              missense_variant  MODERATE           NPC1   \n",
       "168                                intron_variant  MODIFIER         ZNF521   \n",
       "169                              missense_variant  MODERATE          CNDP1   \n",
       "170                              missense_variant  MODERATE         ZNF407   \n",
       "171                           3_prime_UTR_variant  MODIFIER         ZNF236   \n",
       "172                                intron_variant  MODIFIER        CSNK1G2   \n",
       "173                         upstream_gene_variant  MODIFIER       C19orf80   \n",
       "174                              missense_variant  MODERATE           JAK3   \n",
       "175                              missense_variant  MODERATE            ELL   \n",
       "176                            synonymous_variant       LOW         TM6SF2   \n",
       "177                              missense_variant  MODERATE         ZNF536   \n",
       "178                           3_prime_UTR_variant  MODIFIER         CCDC97   \n",
       "179                            synonymous_variant       LOW        CEACAM5   \n",
       "180                                intron_variant  MODIFIER         RASIP1   \n",
       "181                            synonymous_variant       LOW         LILRA3   \n",
       "182                                intron_variant  MODIFIER          NLRP4   \n",
       "183                                intron_variant  MODIFIER      C20orf194   \n",
       "184      splice_region_variant&synonymous_variant       LOW       C20orf27   \n",
       "185                                intron_variant  MODIFIER         SAMHD1   \n",
       "186                           3_prime_UTR_variant  MODIFIER        FAM217B   \n",
       "187                              missense_variant  MODERATE          USP25   \n",
       "188                            synonymous_variant       LOW          PDE9A   \n",
       "189                              missense_variant  MODERATE           SIK1   \n",
       "190          splice_region_variant&intron_variant       LOW       IGLV5-45   \n",
       "191                       downstream_gene_variant  MODIFIER            MIF   \n",
       "192                                   stop_gained      HIGH         PRR14L   \n",
       "193                              missense_variant  MODERATE           CERK   \n",
       "194                              missense_variant  MODERATE        MOV10L1   \n",
       "195                                intron_variant  MODIFIER        MOV10L1   \n",
       "196                              missense_variant  MODERATE        DENND6B   \n",
       "197                                intron_variant  MODIFIER           MSL3   \n",
       "198                                   stop_gained      HIGH          FAM9C   \n",
       "199                            synonymous_variant       LOW           GRPR   \n",
       "200                            synonymous_variant       LOW         TMEM47   \n",
       "201                                intron_variant  MODIFIER          AWAT2   \n",
       "202                       downstream_gene_variant  MODIFIER         PDZD11   \n",
       "203                              missense_variant  MODERATE         FRMPD3   \n",
       "204            non_coding_transcript_exon_variant  MODIFIER        CROCCP1   \n",
       "205                       downstream_gene_variant  MODIFIER            HYI   \n",
       "206                                intron_variant  MODIFIER          LRRC7   \n",
       "207                                intron_variant  MODIFIER         ZBTB41   \n",
       "208                                intron_variant  MODIFIER          SCRN3   \n",
       "209                                intron_variant  MODIFIER           AOX1   \n",
       "210                                intron_variant  MODIFIER           FGD5   \n",
       "211                              missense_variant  MODERATE       KIAA1524   \n",
       "212                              missense_variant  MODERATE         PIK3CA   \n",
       "213                           3_prime_UTR_variant  MODIFIER          CXCL5   \n",
       "214                              missense_variant  MODERATE        FAM159B   \n",
       "215          splice_region_variant&intron_variant       LOW         TCERG1   \n",
       "216                           5_prime_UTR_variant  MODIFIER          ENPP4   \n",
       "217                            synonymous_variant       LOW           BMP5   \n",
       "218            non_coding_transcript_exon_variant  MODIFIER     GTF2IRD2P1   \n",
       "219                         upstream_gene_variant  MODIFIER           NOS3   \n",
       "220                                intron_variant  MODIFIER         GABBR2   \n",
       "221                       downstream_gene_variant  MODIFIER         SNAPC4   \n",
       "222                           3_prime_UTR_variant  MODIFIER          CLCF1   \n",
       "223        missense_variant&splice_region_variant  MODERATE         NDUFV1   \n",
       "224                           3_prime_UTR_variant  MODIFIER          SPPL3   \n",
       "225          splice_region_variant&intron_variant       LOW         PCDH17   \n",
       "226                           3_prime_UTR_variant  MODIFIER          MAPK3   \n",
       "227                              missense_variant  MODERATE          FANCA   \n",
       "228                            synonymous_variant       LOW          SRP68   \n",
       "229                              missense_variant  MODERATE         NPLOC4   \n",
       "230                                intron_variant  MODIFIER           GNG7   \n",
       "231                       downstream_gene_variant  MODIFIER        SLC17A7   \n",
       "232                                intron_variant  MODIFIER          PHF20   \n",
       "233                              missense_variant  MODERATE          BIRC7   \n",
       "234                              missense_variant  MODERATE        CXorf67   \n",
       "\n",
       "                                 HGVSc                           HGVSp  \\\n",
       "0          ENST00000378585.4:c.1295G>A   ENSP00000367848.4:p.Arg432His   \n",
       "1          ENST00000375759.3:c.6803C>A  ENSP00000364912.3:p.Ala2268Asp   \n",
       "2          ENST00000358432.5:c.1485C>T   ENSP00000351209.5:p.Asp495%3D   \n",
       "3           ENST00000374338.4:c.181C>G    ENSP00000363458.4:p.Pro61Ala   \n",
       "4      ENST00000373836.3:c.1846+705T>A                             NaN   \n",
       "5                                  NaN                             NaN   \n",
       "6        ENST00000294360.4:c.261+29G>T                             NaN   \n",
       "7            ENST00000302250.2:c.85G>A    ENSP00000306888.2:p.Ala29Thr   \n",
       "8           ENST00000370571.2:c.831A>C   ENSP00000359603.2:p.Lys277Asn   \n",
       "9     ENST00000427290.1:n.167+56858G>A                             NaN   \n",
       "10         ENST00000307546.9:c.2304C>T   ENSP00000304604.9:p.Arg768%3D   \n",
       "11         ENST00000336338.5:c.1310G>A   ENSP00000337804.5:p.Arg437Gln   \n",
       "12         ENST00000369135.4:c.1312T>C   ENSP00000358131.3:p.Ser438Pro   \n",
       "13          ENST00000369135.4:c.332G>T   ENSP00000358131.3:p.Arg111Leu   \n",
       "14                                 NaN                             NaN   \n",
       "15         ENST00000368764.3:c.1730A>G   ENSP00000357753.3:p.Glu577Gly   \n",
       "16          ENST00000368655.4:c.470A>G   ENSP00000357644.4:p.Lys157Arg   \n",
       "17          ENST00000400979.2:n.379T>G                             NaN   \n",
       "18         ENST00000263688.3:c.3191A>G  ENSP00000263688.3:p.Asp1064Gly   \n",
       "19          ENST00000263726.2:c.224T>C    ENSP00000263726.2:p.Val75Ala   \n",
       "20      ENST00000367435.3:c.1559+57G>T                             NaN   \n",
       "21     ENST00000238789.5:c.1819+984C>A                             NaN   \n",
       "22    ENST00000352846.3:c.101+10676G>A                             NaN   \n",
       "23                                 NaN                             NaN   \n",
       "24          ENST00000313349.3:c.461C>A   ENSP00000326933.3:p.Pro154His   \n",
       "25      ENST00000263713.5:c.1178+25A>G                             NaN   \n",
       "26     ENST00000263710.4:c.2733+511A>G                             NaN   \n",
       "27        ENST00000304636.3:c.897+9G>A                             NaN   \n",
       "28         ENST00000264380.4:c.1422C>T   ENSP00000264380.4:p.Asp474%3D   \n",
       "29          ENST00000243786.2:c.487G>A   ENSP00000243786.2:p.Val163Met   \n",
       "30                                 NaN                             NaN   \n",
       "31         ENST00000295770.2:c.1558C>A   ENSP00000295770.2:p.His520Asn   \n",
       "32         ENST00000264741.5:c.1520G>A   ENSP00000264741.5:p.Gly507Glu   \n",
       "33         ENST00000301831.4:c.2387A>G   ENSP00000301831.4:p.Asn796Ser   \n",
       "34         ENST00000306125.6:c.1268C>G   ENSP00000307567.6:p.Thr423Arg   \n",
       "35         ENST00000295878.3:c.2534G>A   ENSP00000295878.3:p.Trp845Ter   \n",
       "36         ENST00000311127.4:c.3882G>A  ENSP00000311502.3:p.Pro1294%3D   \n",
       "37      ENST00000260818.6:c.5887-25A>G                             NaN   \n",
       "38          ENST00000398015.3:c.100T>A    ENSP00000381097.3:p.Trp34Arg   \n",
       "39          ENST00000282466.3:c.586T>C   ENSP00000282466.3:p.Phe196Leu   \n",
       "40          ENST00000493579.1:n.621G>A                             NaN   \n",
       "41          ENST00000354905.2:c.-18G>A                             NaN   \n",
       "42      ENST00000263923.4:c.68-1372G>T                             NaN   \n",
       "43         ENST00000309964.4:c.2017G>T   ENSP00000308741.4:p.Gly673Ter   \n",
       "44          ENST00000265498.1:c.145G>A    ENSP00000265498.1:p.Val49Ile   \n",
       "45       ENST00000262990.4:c.689-19A>T                             NaN   \n",
       "46    ENST00000280193.2:c.147+29780T>C                             NaN   \n",
       "47          ENST00000393593.3:c.237G>A    ENSP00000377218.3:p.Ala79%3D   \n",
       "48         ENST00000274140.5:c.2584A>G   ENSP00000274140.4:p.Ile862Val   \n",
       "49       ENST00000296657.5:c.497-26A>G                             NaN   \n",
       "50                                 NaN                             NaN   \n",
       "51      ENST00000274473.6:c.2728+54C>T                             NaN   \n",
       "52         ENST00000265335.6:c.2581C>A   ENSP00000265335.6:p.Gln861Lys   \n",
       "53   ENST00000307360.5:c.2388+13653T>C                             NaN   \n",
       "54                                 NaN                             NaN   \n",
       "55         ENST00000261800.5:c.6650C>T  ENSP00000261800.5:p.Thr2217Ile   \n",
       "56         ENST00000356722.3:c.*109C>A                             NaN   \n",
       "57       ENST00000259698.4:c.258-27C>A                             NaN   \n",
       "58                                 NaN                             NaN   \n",
       "59                                 NaN                             NaN   \n",
       "60           ENST00000192788.5:c.*7T>C                             NaN   \n",
       "61                                 NaN                             NaN   \n",
       "62          ENST00000373616.5:c.408C>A   ENSP00000362718.5:p.Ala136%3D   \n",
       "63                                 NaN                             NaN   \n",
       "64         ENST00000252050.4:c.5275C>T  ENSP00000252050.4:p.Arg1759Trp   \n",
       "65                                 NaN                             NaN   \n",
       "66        ENST00000369838.4:c.46-48C>A                             NaN   \n",
       "67         ENST00000402938.3:c.1079G>A   ENSP00000386029.4:p.Arg360Gln   \n",
       "68          ENST00000368847.4:c.568G>T   ENSP00000357840.4:p.Gly190Cys   \n",
       "69                                 NaN                             NaN   \n",
       "70         ENST00000275493.2:c.2641A>G   ENSP00000275493.2:p.Met881Val   \n",
       "71          ENST00000304874.9:c.810C>A   ENSP00000307188.9:p.Phe270Leu   \n",
       "72        ENST00000359558.2:c.28+26C>A                             NaN   \n",
       "73                                 NaN                             NaN   \n",
       "74         ENST00000275732.5:c.*663A>G                             NaN   \n",
       "75          ENST00000300303.2:c.247C>T    ENSP00000300303.2:p.Arg83Cys   \n",
       "76          ENST00000284601.3:c.619T>C   ENSP00000284601.3:p.Phe207Leu   \n",
       "77          ENST00000285968.6:c.*50T>C                             NaN   \n",
       "78     ENST00000348225.2:c.-1-38402G>T                             NaN   \n",
       "79     ENST00000348225.2:c.-1-42626C>T                             NaN   \n",
       "80      ENST00000263549.3:c.1780+11A>G                             NaN   \n",
       "81          ENST00000326232.9:c.313G>A   ENSP00000321498.9:p.Val105Ile   \n",
       "82          ENST00000275815.3:c.157G>A    ENSP00000275815.3:p.Glu53Lys   \n",
       "83                                 NaN                             NaN   \n",
       "84         ENST00000265709.8:c.4492C>T  ENSP00000265709.8:p.Arg1498Cys   \n",
       "85        ENST00000260113.2:c.*3981G>A                             NaN   \n",
       "86                                 NaN                             NaN   \n",
       "87          ENST00000220966.6:c.642C>A   ENSP00000220966.6:p.Ser214Arg   \n",
       "88        ENST00000322810.4:c.12297A>G  ENSP00000323856.4:p.Thr4099%3D   \n",
       "89         ENST00000376447.3:c.1268G>A   ENSP00000365630.3:p.Arg423Lys   \n",
       "90         ENST00000297954.4:c.3717G>T  ENSP00000297954.4:p.Leu1239%3D   \n",
       "91          ENST00000277216.3:c.689C>T   ENSP00000277216.3:p.Ser230Leu   \n",
       "92       ENST00000277540.2:c.950-59G>A                             NaN   \n",
       "93           ENST00000277540.2:c.36C>A    ENSP00000277540.2:p.Thr12%3D   \n",
       "94         ENST00000315238.1:c.*366G>C                             NaN   \n",
       "95          ENST00000263125.5:c.810T>C   ENSP00000263125.5:p.His270%3D   \n",
       "96      ENST00000256861.6:c.1418+18G>A                             NaN   \n",
       "97        ENST00000376663.3:c.-19-8T>C                             NaN   \n",
       "98      ENST00000265371.4:c.814+509G>C                             NaN   \n",
       "99          ENST00000361384.2:c.797A>G   ENSP00000355185.2:p.Glu266Gly   \n",
       "100        ENST00000277895.5:c.1242T>A   ENSP00000277895.5:p.Tyr414Ter   \n",
       "101         ENST00000298492.5:c.-16G>T                             NaN   \n",
       "102        ENST00000280333.6:c.1637A>G   ENSP00000280333.6:p.Asp546Gly   \n",
       "103   ENST00000280333.6:c.2784+9330G>T                             NaN   \n",
       "104     ENST00000304613.3:c.1514-15G>A                             NaN   \n",
       "105                                NaN                             NaN   \n",
       "106         ENST00000334619.2:c.320G>A   ENSP00000335544.2:p.Cys107Tyr   \n",
       "107      ENST00000358540.2:c.1001-3C>A                             NaN   \n",
       "108                                NaN                             NaN   \n",
       "109         ENST00000313253.1:c.272C>T    ENSP00000320077.1:p.Thr91Ile   \n",
       "110         ENST00000278888.3:c.598A>T   ENSP00000278888.3:p.Thr200Ser   \n",
       "111         ENST00000294168.3:c.-13C>A                             NaN   \n",
       "112         ENST00000255078.3:c.519A>T   ENSP00000255078.3:p.Arg173Ser   \n",
       "113      ENST00000198801.5:c.271-31G>A                             NaN   \n",
       "114                                NaN                             NaN   \n",
       "115         ENST00000357438.2:c.638C>T   ENSP00000350022.2:p.Ala213Val   \n",
       "116                                NaN                             NaN   \n",
       "117         ENST00000229335.6:c.129A>G    ENSP00000229335.6:p.Ser43%3D   \n",
       "118       ENST00000261336.2:c.83+36G>A                             NaN   \n",
       "119          ENST00000256078.4:c.38G>A    ENSP00000256078.4:p.Gly13Asp   \n",
       "120         ENST00000267101.3:c.179T>A    ENSP00000267101.3:p.Met60Lys   \n",
       "121         ENST00000262027.5:c.100G>C    ENSP00000262027.5:p.Gly34Arg   \n",
       "122      ENST00000392876.3:c.616-11C>T                             NaN   \n",
       "123          ENST00000288532.6:c.24T>C     ENSP00000288532.6:p.Ala8%3D   \n",
       "124        ENST00000409039.3:c.3982T>C  ENSP00000386770.3:p.Phe1328Leu   \n",
       "125         ENST00000377918.3:c.976C>A   ENSP00000367151.3:p.Leu326Met   \n",
       "126       ENST00000282003.6:c.208+4A>G                             NaN   \n",
       "127         ENST00000376547.3:c.896T>C   ENSP00000365730.3:p.Leu299Pro   \n",
       "128                                NaN                             NaN   \n",
       "129     ENST00000262715.5:c.2466-12T>C                             NaN   \n",
       "130     ENST00000238616.5:c.2817+24C>A                             NaN   \n",
       "131                                NaN                             NaN   \n",
       "132         ENST00000249647.3:c.396G>A   ENSP00000249647.3:p.Thr132%3D   \n",
       "133        ENST00000300231.5:c.7364C>A  ENSP00000300231.5:p.Pro2455Gln   \n",
       "134        ENST00000316623.5:c.3410G>A  ENSP00000325527.5:p.Arg1137His   \n",
       "135      ENST00000261891.3:c.632+46G>A                             NaN   \n",
       "136       ENST00000315757.7:c.164+5G>A                             NaN   \n",
       "137        ENST00000268058.3:c.2473C>T   ENSP00000268058.3:p.Arg825Cys   \n",
       "138      ENST00000267973.2:c.468+23T>C                             NaN   \n",
       "139        ENST00000361243.2:c.1291G>A   ENSP00000354718.2:p.Gly431Arg   \n",
       "140         ENST00000268661.7:c.365G>T   ENSP00000268661.7:p.Trp122Leu   \n",
       "141       ENST00000262367.5:c.*2614G>T                             NaN   \n",
       "142         ENST00000322048.7:c.*71G>T                             NaN   \n",
       "143         ENST00000268383.2:c.211G>T    ENSP00000268383.2:p.Glu71Ter   \n",
       "144        ENST00000307149.5:c.1239C>A   ENSP00000305442.5:p.Thr413%3D   \n",
       "145         ENST00000300087.2:c.331G>A   ENSP00000300087.2:p.Gly111Arg   \n",
       "146         ENST00000256797.4:c.836G>T   ENSP00000256797.4:p.Gly279Val   \n",
       "147      ENST00000215115.4:c.528+38A>G                             NaN   \n",
       "148                                NaN                             NaN   \n",
       "149          ENST00000408605.1:n.38G>A                             NaN   \n",
       "150        ENST00000290552.7:c.1284G>A   ENSP00000290552.7:p.Pro428%3D   \n",
       "151         ENST00000334350.6:c.168G>A    ENSP00000334872.6:p.Lys56%3D   \n",
       "152      ENST00000262510.6:c.296-18C>T                             NaN   \n",
       "153     ENST00000330687.4:c.623+602C>A                             NaN   \n",
       "154        ENST00000359376.3:c.1149C>T   ENSP00000352336.3:p.Asp383%3D   \n",
       "155                                NaN                             NaN   \n",
       "156                                NaN                             NaN   \n",
       "157         ENST00000269305.4:c.535C>T   ENSP00000269305.4:p.His179Tyr   \n",
       "158        ENST00000255381.2:c.4203G>T  ENSP00000255381.2:p.Leu1401%3D   \n",
       "159                                NaN                             NaN   \n",
       "160        ENST00000321990.4:c.1849G>A   ENSP00000313171.4:p.Val617Ile   \n",
       "161                                NaN                             NaN   \n",
       "162                                NaN                             NaN   \n",
       "163      ENST00000336915.6:c.567+50T>C                             NaN   \n",
       "164         ENST00000259021.4:c.801C>G   ENSP00000259021.4:p.Leu267%3D   \n",
       "165                                NaN                             NaN   \n",
       "166      ENST00000317615.6:c.582-27A>G                             NaN   \n",
       "167        ENST00000269228.5:c.2314T>C   ENSP00000269228.4:p.Phe772Leu   \n",
       "168   ENST00000361524.3:c.220+32836C>G                             NaN   \n",
       "169        ENST00000358821.3:c.1436C>T   ENSP00000351682.3:p.Ser479Leu   \n",
       "170        ENST00000299687.5:c.4132G>A  ENSP00000299687.4:p.Gly1378Ser   \n",
       "171       ENST00000253159.8:c.*1399C>A                             NaN   \n",
       "172    ENST00000255641.8:c.188-2134G>A                             NaN   \n",
       "173                                NaN                             NaN   \n",
       "174        ENST00000458235.1:c.1453C>A   ENSP00000391676.1:p.Leu485Met   \n",
       "175        ENST00000262809.4:c.1852C>A   ENSP00000262809.3:p.Gln618Lys   \n",
       "176         ENST00000389363.4:c.237C>A    ENSP00000374014.2:p.Leu79%3D   \n",
       "177        ENST00000355537.3:c.1552G>T   ENSP00000347730.1:p.Val518Leu   \n",
       "178       ENST00000269967.3:c.*1096C>T                             NaN   \n",
       "179         ENST00000221992.6:c.720C>T   ENSP00000221992.5:p.Pro240%3D   \n",
       "180       ENST00000222145.4:c.-5+17C>T                             NaN   \n",
       "181        ENST00000251390.3:c.1083G>T   ENSP00000251390.3:p.Gly361%3D   \n",
       "182     ENST00000301295.6:c.2018+34C>G                             NaN   \n",
       "183      ENST00000252032.9:c.163+18G>T                             NaN   \n",
       "184          ENST00000217195.8:c.96C>A    ENSP00000217195.8:p.Gly32%3D   \n",
       "185      ENST00000262878.4:c.696+25C>A                             NaN   \n",
       "186        ENST00000358293.3:c.*842C>A                             NaN   \n",
       "187         ENST00000285679.6:c.131A>T    ENSP00000285679.6:p.Asn44Ile   \n",
       "188         ENST00000291539.6:c.984C>A   ENSP00000291539.6:p.Val328%3D   \n",
       "189         ENST00000270162.6:c.355G>C   ENSP00000270162.6:p.Gly119Arg   \n",
       "190        ENST00000390296.2:c.47-3C>T                             NaN   \n",
       "191                                NaN                             NaN   \n",
       "192        ENST00000327423.6:c.1642C>T   ENSP00000331845.6:p.Gln548Ter   \n",
       "193         ENST00000216264.8:c.709C>T   ENSP00000216264.8:p.Pro237Ser   \n",
       "194        ENST00000262794.5:c.2731G>A   ENSP00000262794.5:p.Val911Met   \n",
       "195     ENST00000262794.5:c.2893-87C>T                             NaN   \n",
       "196        ENST00000413817.3:c.1318C>A   ENSP00000391524.2:p.Pro440Thr   \n",
       "197     ENST00000312196.4:c.1171+76G>T                             NaN   \n",
       "198         ENST00000333995.3:c.448C>T   ENSP00000334430.3:p.Gln150Ter   \n",
       "199        ENST00000380289.2:c.1098C>A   ENSP00000369643.2:p.Pro366%3D   \n",
       "200         ENST00000275954.3:c.276T>A    ENSP00000275954.3:p.Ile92%3D   \n",
       "201      ENST00000276101.3:c.848-34C>T                             NaN   \n",
       "202                                NaN                             NaN   \n",
       "203        ENST00000276185.4:c.3626A>T  ENSP00000276185.4:p.Gln1209Leu   \n",
       "204         ENST00000426910.1:n.393G>A                             NaN   \n",
       "205                                NaN                             NaN   \n",
       "206   ENST00000035383.5:c.4117-2199A>T                             NaN   \n",
       "207     ENST00000367405.4:c.2074+37T>G                             NaN   \n",
       "208     ENST00000272732.6:c.1092+55C>T                             NaN   \n",
       "209     ENST00000374700.2:c.2480+47G>C                             NaN   \n",
       "210     ENST00000285046.5:c.3489+26C>T                             NaN   \n",
       "211         ENST00000295746.8:c.265A>G    ENSP00000295746.7:p.Thr89Ala   \n",
       "212        ENST00000263967.3:c.1633G>A   ENSP00000263967.3:p.Glu545Lys   \n",
       "213       ENST00000296027.4:c.*1115A>C                             NaN   \n",
       "214         ENST00000389074.5:c.118G>T    ENSP00000373726.5:p.Ala40Ser   \n",
       "215       ENST00000296702.5:c.439-7G>A                             NaN   \n",
       "216         ENST00000321037.4:c.-22G>T                             NaN   \n",
       "217         ENST00000370830.3:c.111T>C    ENSP00000359866.3:p.Ser37%3D   \n",
       "218        ENST00000425256.1:n.1553C>T                             NaN   \n",
       "219                                NaN                             NaN   \n",
       "220     ENST00000259455.2:c.2413-30C>G                             NaN   \n",
       "221                                NaN                             NaN   \n",
       "222        ENST00000312438.7:c.*143C>T                             NaN   \n",
       "223          ENST00000322776.6:c.74C>T    ENSP00000322450.6:p.Thr25Ile   \n",
       "224       ENST00000353487.2:c.*1393A>C                             NaN   \n",
       "225      ENST00000377918.3:c.2566-7T>G                             NaN   \n",
       "226        ENST00000263025.4:c.*222C>G                             NaN   \n",
       "227        ENST00000389301.3:c.1046C>T   ENSP00000373952.3:p.Ala349Val   \n",
       "228         ENST00000307877.2:c.813G>A   ENSP00000312066.1:p.Glu271%3D   \n",
       "229         ENST00000331134.6:c.703G>T   ENSP00000331487.5:p.Asp235Tyr   \n",
       "230    ENST00000382159.3:c.-78+2918C>T                             NaN   \n",
       "231                                NaN                             NaN   \n",
       "232     ENST00000374012.3:c.2897-34T>C                             NaN   \n",
       "233         ENST00000217169.3:c.178C>T    ENSP00000217169.3:p.Arg60Trp   \n",
       "234        ENST00000342995.2:c.1139C>T   ENSP00000342680.2:p.Ala380Val   \n",
       "\n",
       "                                      CLIN_SIG                  PolyPhen  \\\n",
       "0                                          NaN  probably_damaging(0.991)   \n",
       "1                                          NaN  probably_damaging(0.991)   \n",
       "2                                          NaN                       NaN   \n",
       "3                                          NaN  probably_damaging(0.914)   \n",
       "4                                          NaN                       NaN   \n",
       "5                                          NaN                       NaN   \n",
       "6                                          NaN                       NaN   \n",
       "7                                          NaN                 benign(0)   \n",
       "8                                          NaN                 benign(0)   \n",
       "9                                          NaN                       NaN   \n",
       "10                                         NaN                       NaN   \n",
       "11                                         NaN                 benign(0)   \n",
       "12                                         NaN             benign(0.367)   \n",
       "13                                         NaN  probably_damaging(0.999)   \n",
       "14                                         NaN                       NaN   \n",
       "15                                         NaN  probably_damaging(0.942)   \n",
       "16                                         NaN  probably_damaging(0.956)   \n",
       "17                                         NaN                       NaN   \n",
       "18                                         NaN  probably_damaging(0.998)   \n",
       "19                                         NaN  probably_damaging(0.991)   \n",
       "20                                         NaN                       NaN   \n",
       "21                                         NaN                       NaN   \n",
       "22                                         NaN                       NaN   \n",
       "23                                         NaN                       NaN   \n",
       "24                                         NaN  possibly_damaging(0.818)   \n",
       "25                                         NaN                       NaN   \n",
       "26                                         NaN                       NaN   \n",
       "27                                         NaN                       NaN   \n",
       "28                                         NaN                       NaN   \n",
       "29                                         NaN             benign(0.003)   \n",
       "30                                         NaN                       NaN   \n",
       "31                                         NaN             benign(0.028)   \n",
       "32                                      benign              benign(0.02)   \n",
       "33                                         NaN             benign(0.003)   \n",
       "34                                         NaN      probably_damaging(1)   \n",
       "35                                         NaN                       NaN   \n",
       "36                                         NaN                       NaN   \n",
       "37                                         NaN                       NaN   \n",
       "38                                         NaN  probably_damaging(0.999)   \n",
       "39                                         NaN             benign(0.005)   \n",
       "40                                         NaN                       NaN   \n",
       "41                                         NaN                       NaN   \n",
       "42                                         NaN                       NaN   \n",
       "43                                         NaN                       NaN   \n",
       "44                                         NaN                 benign(0)   \n",
       "45                                         NaN                       NaN   \n",
       "46                                         NaN                       NaN   \n",
       "47                                         NaN                       NaN   \n",
       "48                                         NaN                 benign(0)   \n",
       "49                                         NaN                       NaN   \n",
       "50                                         NaN                       NaN   \n",
       "51                                         NaN                       NaN   \n",
       "52                                         NaN             benign(0.103)   \n",
       "53                                         NaN                       NaN   \n",
       "54                                         NaN                       NaN   \n",
       "55                                         NaN             benign(0.255)   \n",
       "56                                         NaN                       NaN   \n",
       "57                                         NaN                       NaN   \n",
       "58                                         NaN                       NaN   \n",
       "59                                         NaN                       NaN   \n",
       "60                                         NaN                       NaN   \n",
       "61                                         NaN                       NaN   \n",
       "62                                         NaN                       NaN   \n",
       "63                                         NaN                       NaN   \n",
       "64                                         NaN                 benign(0)   \n",
       "65                                         NaN                       NaN   \n",
       "66                                         NaN                       NaN   \n",
       "67                                         NaN             benign(0.034)   \n",
       "68                                         NaN  probably_damaging(0.998)   \n",
       "69                                         NaN                       NaN   \n",
       "70                                         NaN  possibly_damaging(0.583)   \n",
       "71                                         NaN  probably_damaging(0.982)   \n",
       "72                                         NaN                       NaN   \n",
       "73                                         NaN                       NaN   \n",
       "74                                         NaN                       NaN   \n",
       "75                                         NaN             benign(0.077)   \n",
       "76                                         NaN  probably_damaging(0.971)   \n",
       "77                                         NaN                       NaN   \n",
       "78                                         NaN                       NaN   \n",
       "79                                         NaN                       NaN   \n",
       "80                                         NaN                       NaN   \n",
       "81                                         NaN             benign(0.001)   \n",
       "82                                         NaN             benign(0.043)   \n",
       "83                                         NaN                       NaN   \n",
       "84                                         NaN  possibly_damaging(0.902)   \n",
       "85                                         NaN                       NaN   \n",
       "86                                         NaN                       NaN   \n",
       "87                                         NaN  possibly_damaging(0.851)   \n",
       "88                                      benign                       NaN   \n",
       "89                                         NaN             benign(0.018)   \n",
       "90                                         NaN                       NaN   \n",
       "91                                         NaN  possibly_damaging(0.876)   \n",
       "92                                         NaN                       NaN   \n",
       "93                                         NaN                       NaN   \n",
       "94                                         NaN                       NaN   \n",
       "95                                         NaN                       NaN   \n",
       "96                                         NaN                       NaN   \n",
       "97                                         NaN                       NaN   \n",
       "98                                         NaN                       NaN   \n",
       "99                                         NaN                 benign(0)   \n",
       "100                                        NaN                       NaN   \n",
       "101                                        NaN                       NaN   \n",
       "102                                        NaN  possibly_damaging(0.589)   \n",
       "103                                        NaN                       NaN   \n",
       "104                                        NaN                       NaN   \n",
       "105                                        NaN                       NaN   \n",
       "106                                        NaN  probably_damaging(0.998)   \n",
       "107                                        NaN                       NaN   \n",
       "108                                        NaN                       NaN   \n",
       "109                                        NaN             benign(0.046)   \n",
       "110                                        NaN  possibly_damaging(0.708)   \n",
       "111                                        NaN                       NaN   \n",
       "112                                        NaN             benign(0.001)   \n",
       "113                                        NaN                       NaN   \n",
       "114                                        NaN                       NaN   \n",
       "115                                        NaN  possibly_damaging(0.869)   \n",
       "116                                        NaN                       NaN   \n",
       "117                                        NaN                       NaN   \n",
       "118                                        NaN                       NaN   \n",
       "119               pathogenic&likely_pathogenic              benign(0.44)   \n",
       "120                                        NaN             benign(0.147)   \n",
       "121                                        NaN             benign(0.003)   \n",
       "122                                        NaN                       NaN   \n",
       "123                                        NaN                       NaN   \n",
       "124                                        NaN  probably_damaging(0.991)   \n",
       "125                                        NaN  probably_damaging(0.962)   \n",
       "126                                        NaN                       NaN   \n",
       "127                                        NaN      probably_damaging(1)   \n",
       "128                                        NaN                       NaN   \n",
       "129                                        NaN                       NaN   \n",
       "130                                        NaN                       NaN   \n",
       "131                                        NaN                       NaN   \n",
       "132                                        NaN                       NaN   \n",
       "133                                        NaN  probably_damaging(0.964)   \n",
       "134                     uncertain_significance  possibly_damaging(0.615)   \n",
       "135                                        NaN                       NaN   \n",
       "136                                        NaN                       NaN   \n",
       "137                                        NaN             benign(0.003)   \n",
       "138                                        NaN                       NaN   \n",
       "139                                        NaN             benign(0.001)   \n",
       "140                                        NaN             benign(0.034)   \n",
       "141                                        NaN                       NaN   \n",
       "142                                        NaN                       NaN   \n",
       "143                                        NaN                       NaN   \n",
       "144                                        NaN                       NaN   \n",
       "145                                        NaN  probably_damaging(0.976)   \n",
       "146                                        NaN  probably_damaging(0.954)   \n",
       "147                                        NaN                       NaN   \n",
       "148                                        NaN                       NaN   \n",
       "149                                        NaN                       NaN   \n",
       "150                                        NaN                       NaN   \n",
       "151                                        NaN                       NaN   \n",
       "152                                        NaN                       NaN   \n",
       "153                                        NaN                       NaN   \n",
       "154                                     benign                       NaN   \n",
       "155                                        NaN                       NaN   \n",
       "156                                        NaN                       NaN   \n",
       "157               pathogenic&likely_pathogenic  probably_damaging(0.991)   \n",
       "158                                        NaN                       NaN   \n",
       "159                                        NaN                       NaN   \n",
       "160                                        NaN             benign(0.037)   \n",
       "161                                        NaN                       NaN   \n",
       "162                                        NaN                       NaN   \n",
       "163                                        NaN                       NaN   \n",
       "164                                        NaN                       NaN   \n",
       "165                                        NaN                       NaN   \n",
       "166                                        NaN                       NaN   \n",
       "167                                        NaN  probably_damaging(0.991)   \n",
       "168                                        NaN                       NaN   \n",
       "169                                        NaN  probably_damaging(0.993)   \n",
       "170                                        NaN              benign(0.01)   \n",
       "171                                        NaN                       NaN   \n",
       "172                                        NaN                       NaN   \n",
       "173                                        NaN                       NaN   \n",
       "174                                        NaN  probably_damaging(0.931)   \n",
       "175                                        NaN             benign(0.104)   \n",
       "176                                        NaN                       NaN   \n",
       "177                                        NaN             benign(0.006)   \n",
       "178                                        NaN                       NaN   \n",
       "179                                        NaN                       NaN   \n",
       "180                                        NaN                       NaN   \n",
       "181                                        NaN                       NaN   \n",
       "182                                        NaN                       NaN   \n",
       "183                                        NaN                       NaN   \n",
       "184                                        NaN                       NaN   \n",
       "185                                        NaN                       NaN   \n",
       "186                                        NaN                       NaN   \n",
       "187                                        NaN  possibly_damaging(0.548)   \n",
       "188                                        NaN                       NaN   \n",
       "189                                        NaN  probably_damaging(0.981)   \n",
       "190                                        NaN                       NaN   \n",
       "191                                        NaN                       NaN   \n",
       "192                                        NaN                       NaN   \n",
       "193                                        NaN  probably_damaging(0.999)   \n",
       "194                                        NaN  probably_damaging(0.987)   \n",
       "195                                        NaN                       NaN   \n",
       "196                                        NaN  probably_damaging(0.997)   \n",
       "197                                        NaN                       NaN   \n",
       "198                                        NaN                       NaN   \n",
       "199                                        NaN                       NaN   \n",
       "200                                        NaN                       NaN   \n",
       "201                                        NaN                       NaN   \n",
       "202                                        NaN                       NaN   \n",
       "203                                        NaN                 benign(0)   \n",
       "204                                        NaN                       NaN   \n",
       "205                                        NaN                       NaN   \n",
       "206                                        NaN                       NaN   \n",
       "207                                        NaN                       NaN   \n",
       "208                                        NaN                       NaN   \n",
       "209                                        NaN                       NaN   \n",
       "210                                        NaN                       NaN   \n",
       "211                                        NaN             benign(0.108)   \n",
       "212  not_provided&pathogenic&likely_pathogenic  probably_damaging(0.909)   \n",
       "213                                        NaN                       NaN   \n",
       "214                                        NaN             benign(0.136)   \n",
       "215                                        NaN                       NaN   \n",
       "216                                        NaN                       NaN   \n",
       "217                                        NaN                       NaN   \n",
       "218                                        NaN                       NaN   \n",
       "219                                        NaN                       NaN   \n",
       "220                                        NaN                       NaN   \n",
       "221                                     benign                       NaN   \n",
       "222                                        NaN                       NaN   \n",
       "223                                        NaN             benign(0.021)   \n",
       "224                                        NaN                       NaN   \n",
       "225                                        NaN                       NaN   \n",
       "226                                        NaN                       NaN   \n",
       "227                     uncertain_significance             benign(0.138)   \n",
       "228                                        NaN                       NaN   \n",
       "229                                        NaN  probably_damaging(0.949)   \n",
       "230                                        NaN                       NaN   \n",
       "231                                        NaN                       NaN   \n",
       "232                                        NaN                       NaN   \n",
       "233                                        NaN  possibly_damaging(0.895)   \n",
       "234                                        NaN             benign(0.013)   \n",
       "\n",
       "                                 SIFT  \n",
       "0                      deleterious(0)  \n",
       "1      tolerated_low_confidence(0.07)  \n",
       "2                                 NaN  \n",
       "3                     tolerated(0.08)  \n",
       "4                                 NaN  \n",
       "5                                 NaN  \n",
       "6                                 NaN  \n",
       "7                     tolerated(0.75)  \n",
       "8                      tolerated(0.4)  \n",
       "9                                 NaN  \n",
       "10                                NaN  \n",
       "11                     deleterious(0)  \n",
       "12                    tolerated(0.11)  \n",
       "13                     deleterious(0)  \n",
       "14                                NaN  \n",
       "15     tolerated_low_confidence(0.09)  \n",
       "16                    tolerated(0.25)  \n",
       "17                                NaN  \n",
       "18                     deleterious(0)  \n",
       "19                     deleterious(0)  \n",
       "20                                NaN  \n",
       "21                                NaN  \n",
       "22                                NaN  \n",
       "23                                NaN  \n",
       "24                  deleterious(0.02)  \n",
       "25                                NaN  \n",
       "26                                NaN  \n",
       "27                                NaN  \n",
       "28                                NaN  \n",
       "29                    tolerated(0.46)  \n",
       "30                                NaN  \n",
       "31                    tolerated(0.44)  \n",
       "32                  deleterious(0.01)  \n",
       "33                       tolerated(1)  \n",
       "34                    tolerated(0.06)  \n",
       "35                                NaN  \n",
       "36                                NaN  \n",
       "37                                NaN  \n",
       "38                     deleterious(0)  \n",
       "39                    tolerated(0.13)  \n",
       "40                                NaN  \n",
       "41                                NaN  \n",
       "42                                NaN  \n",
       "43                                NaN  \n",
       "44                    tolerated(0.96)  \n",
       "45                                NaN  \n",
       "46                                NaN  \n",
       "47                                NaN  \n",
       "48                    tolerated(0.78)  \n",
       "49                                NaN  \n",
       "50                                NaN  \n",
       "51                                NaN  \n",
       "52                    tolerated(0.12)  \n",
       "53                                NaN  \n",
       "54                                NaN  \n",
       "55                  deleterious(0.04)  \n",
       "56                                NaN  \n",
       "57                                NaN  \n",
       "58                                NaN  \n",
       "59                                NaN  \n",
       "60                                NaN  \n",
       "61                                NaN  \n",
       "62                                NaN  \n",
       "63                                NaN  \n",
       "64                    tolerated(0.13)  \n",
       "65                                NaN  \n",
       "66                                NaN  \n",
       "67                    tolerated(0.17)  \n",
       "68                     deleterious(0)  \n",
       "69                                NaN  \n",
       "70                  deleterious(0.03)  \n",
       "71                     deleterious(0)  \n",
       "72                                NaN  \n",
       "73                                NaN  \n",
       "74                                NaN  \n",
       "75                  deleterious(0.03)  \n",
       "76                     deleterious(0)  \n",
       "77                                NaN  \n",
       "78                                NaN  \n",
       "79                                NaN  \n",
       "80                                NaN  \n",
       "81                    tolerated(0.65)  \n",
       "82                    tolerated(0.44)  \n",
       "83                                NaN  \n",
       "84                     deleterious(0)  \n",
       "85                                NaN  \n",
       "86                                NaN  \n",
       "87                       tolerated(1)  \n",
       "88                                NaN  \n",
       "89                     tolerated(0.1)  \n",
       "90                                NaN  \n",
       "91                  deleterious(0.02)  \n",
       "92                                NaN  \n",
       "93                                NaN  \n",
       "94                                NaN  \n",
       "95                                NaN  \n",
       "96                                NaN  \n",
       "97                                NaN  \n",
       "98                                NaN  \n",
       "99                     tolerated(0.3)  \n",
       "100                               NaN  \n",
       "101                               NaN  \n",
       "102                    deleterious(0)  \n",
       "103                               NaN  \n",
       "104                               NaN  \n",
       "105                               NaN  \n",
       "106                    deleterious(0)  \n",
       "107                               NaN  \n",
       "108                               NaN  \n",
       "109                    tolerated(0.1)  \n",
       "110                   tolerated(0.37)  \n",
       "111                               NaN  \n",
       "112                   tolerated(0.87)  \n",
       "113                               NaN  \n",
       "114                               NaN  \n",
       "115                 deleterious(0.04)  \n",
       "116                               NaN  \n",
       "117                               NaN  \n",
       "118                               NaN  \n",
       "119                 deleterious(0.04)  \n",
       "120                    deleterious(0)  \n",
       "121                   tolerated(0.38)  \n",
       "122                               NaN  \n",
       "123                               NaN  \n",
       "124                    deleterious(0)  \n",
       "125                   tolerated(0.07)  \n",
       "126                               NaN  \n",
       "127                 deleterious(0.01)  \n",
       "128                               NaN  \n",
       "129                               NaN  \n",
       "130                               NaN  \n",
       "131                               NaN  \n",
       "132                               NaN  \n",
       "133                    deleterious(0)  \n",
       "134                   tolerated(0.17)  \n",
       "135                               NaN  \n",
       "136                               NaN  \n",
       "137                   tolerated(0.09)  \n",
       "138                               NaN  \n",
       "139                   tolerated(0.12)  \n",
       "140                    deleterious(0)  \n",
       "141                               NaN  \n",
       "142                               NaN  \n",
       "143                               NaN  \n",
       "144                               NaN  \n",
       "145                    deleterious(0)  \n",
       "146                    deleterious(0)  \n",
       "147                               NaN  \n",
       "148                               NaN  \n",
       "149                               NaN  \n",
       "150                               NaN  \n",
       "151                               NaN  \n",
       "152                               NaN  \n",
       "153                               NaN  \n",
       "154                               NaN  \n",
       "155                               NaN  \n",
       "156                               NaN  \n",
       "157                    deleterious(0)  \n",
       "158                               NaN  \n",
       "159                               NaN  \n",
       "160                   tolerated(0.15)  \n",
       "161                               NaN  \n",
       "162                               NaN  \n",
       "163                               NaN  \n",
       "164                               NaN  \n",
       "165                               NaN  \n",
       "166                               NaN  \n",
       "167                    deleterious(0)  \n",
       "168                               NaN  \n",
       "169                    deleterious(0)  \n",
       "170                   tolerated(0.05)  \n",
       "171                               NaN  \n",
       "172                               NaN  \n",
       "173                               NaN  \n",
       "174                 deleterious(0.02)  \n",
       "175                   tolerated(0.13)  \n",
       "176                               NaN  \n",
       "177                   tolerated(0.49)  \n",
       "178                               NaN  \n",
       "179                               NaN  \n",
       "180                               NaN  \n",
       "181                               NaN  \n",
       "182                               NaN  \n",
       "183                               NaN  \n",
       "184                               NaN  \n",
       "185                               NaN  \n",
       "186                               NaN  \n",
       "187                    deleterious(0)  \n",
       "188                               NaN  \n",
       "189                 deleterious(0.01)  \n",
       "190                               NaN  \n",
       "191                               NaN  \n",
       "192                               NaN  \n",
       "193                    deleterious(0)  \n",
       "194                    deleterious(0)  \n",
       "195                               NaN  \n",
       "196                    deleterious(0)  \n",
       "197                               NaN  \n",
       "198                               NaN  \n",
       "199                               NaN  \n",
       "200                               NaN  \n",
       "201                               NaN  \n",
       "202                               NaN  \n",
       "203    tolerated_low_confidence(0.11)  \n",
       "204                               NaN  \n",
       "205                               NaN  \n",
       "206                               NaN  \n",
       "207                               NaN  \n",
       "208                               NaN  \n",
       "209                               NaN  \n",
       "210                               NaN  \n",
       "211                   tolerated(0.14)  \n",
       "212                 deleterious(0.02)  \n",
       "213                               NaN  \n",
       "214                   tolerated(0.83)  \n",
       "215                               NaN  \n",
       "216                               NaN  \n",
       "217                               NaN  \n",
       "218                               NaN  \n",
       "219                               NaN  \n",
       "220                               NaN  \n",
       "221                               NaN  \n",
       "222                               NaN  \n",
       "223  deleterious_low_confidence(0.04)  \n",
       "224                               NaN  \n",
       "225                               NaN  \n",
       "226                               NaN  \n",
       "227                   tolerated(0.14)  \n",
       "228                               NaN  \n",
       "229                    deleterious(0)  \n",
       "230                               NaN  \n",
       "231                               NaN  \n",
       "232                               NaN  \n",
       "233                    deleterious(0)  \n",
       "234                      tolerated(1)  "
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Index(['seqnames', 'start', 'end', 'width', 'strand', 'AD', 'AD.1', 'AF',\n",
       "       'Consequence', 'IMPACT', 'SYMBOL', 'HGVSc', 'HGVSp', 'CLIN_SIG',\n",
       "       'PolyPhen', 'SIFT'],\n",
       "      dtype='object')"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf.columns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "#First filter Keep the vaiants in clinvar\n",
    "vcf_clinvar=vcf[vcf['CLIN_SIG'].notnull()]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>width</th>\n",
       "      <th>strand</th>\n",
       "      <th>AD</th>\n",
       "      <th>AD.1</th>\n",
       "      <th>AF</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>SYMBOL</th>\n",
       "      <th>HGVSc</th>\n",
       "      <th>HGVSp</th>\n",
       "      <th>CLIN_SIG</th>\n",
       "      <th>PolyPhen</th>\n",
       "      <th>SIFT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>37574951</td>\n",
       "      <td>37574951</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ITGA9</td>\n",
       "      <td>ENST00000264741.5:c.1520G&gt;A</td>\n",
       "      <td>ENSP00000264741.5:p.Gly507Glu</td>\n",
       "      <td>benign</td>\n",
       "      <td>benign(0.02)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>88</td>\n",
       "      <td>8</td>\n",
       "      <td>144992103</td>\n",
       "      <td>144992103</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLEC</td>\n",
       "      <td>ENST00000322810.4:c.12297A&gt;G</td>\n",
       "      <td>ENSP00000323856.4:p.Thr4099%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>119</td>\n",
       "      <td>12</td>\n",
       "      <td>25398281</td>\n",
       "      <td>25398281</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>14</td>\n",
       "      <td>0.326</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>ENST00000256078.4:c.38G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly13Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.44)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>134</td>\n",
       "      <td>15</td>\n",
       "      <td>48779562</td>\n",
       "      <td>48779562</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>7</td>\n",
       "      <td>0.089</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>ENST00000316623.5:c.3410G&gt;A</td>\n",
       "      <td>ENSP00000325527.5:p.Arg1137His</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>possibly_damaging(0.615)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>154</td>\n",
       "      <td>16</td>\n",
       "      <td>81929488</td>\n",
       "      <td>81929488</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLCG2</td>\n",
       "      <td>ENST00000359376.3:c.1149C&gt;T</td>\n",
       "      <td>ENSP00000352336.3:p.Asp383%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>157</td>\n",
       "      <td>17</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578395</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TP53</td>\n",
       "      <td>ENST00000269305.4:c.535C&gt;T</td>\n",
       "      <td>ENSP00000269305.4:p.His179Tyr</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>212</td>\n",
       "      <td>3</td>\n",
       "      <td>178936091</td>\n",
       "      <td>178936091</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>9</td>\n",
       "      <td>0.185</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>ENST00000263967.3:c.1633G&gt;A</td>\n",
       "      <td>ENSP00000263967.3:p.Glu545Lys</td>\n",
       "      <td>not_provided&amp;pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.909)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>221</td>\n",
       "      <td>9</td>\n",
       "      <td>139266496</td>\n",
       "      <td>139266496</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>3</td>\n",
       "      <td>0.211</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SNAPC4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>227</td>\n",
       "      <td>16</td>\n",
       "      <td>89858916</td>\n",
       "      <td>89858916</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.112</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FANCA</td>\n",
       "      <td>ENST00000389301.3:c.1046C&gt;T</td>\n",
       "      <td>ENSP00000373952.3:p.Ala349Val</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>benign(0.138)</td>\n",
       "      <td>tolerated(0.14)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    seqnames      start        end  width strand  AD  AD.1     AF  \\\n",
       "32         3   37574951   37574951      1      *  32     4  0.111   \n",
       "88         8  144992103  144992103      1      *  32     4  0.111   \n",
       "119       12   25398281   25398281      1      *  29    14  0.326   \n",
       "134       15   48779562   48779562      1      *  72     7  0.089   \n",
       "154       16   81929488   81929488      1      *  42     3  0.067   \n",
       "157       17    7578395    7578395      1      *  23     7  0.233   \n",
       "212        3  178936091  178936091      1      *  43     9  0.185   \n",
       "221        9  139266496  139266496      1      *  14     3  0.211   \n",
       "227       16   89858916   89858916      1      *  38     4  0.112   \n",
       "\n",
       "                 Consequence    IMPACT  SYMBOL                         HGVSc  \\\n",
       "32          missense_variant  MODERATE   ITGA9   ENST00000264741.5:c.1520G>A   \n",
       "88        synonymous_variant       LOW    PLEC  ENST00000322810.4:c.12297A>G   \n",
       "119         missense_variant  MODERATE    KRAS     ENST00000256078.4:c.38G>A   \n",
       "134         missense_variant  MODERATE    FBN1   ENST00000316623.5:c.3410G>A   \n",
       "154       synonymous_variant       LOW   PLCG2   ENST00000359376.3:c.1149C>T   \n",
       "157         missense_variant  MODERATE    TP53    ENST00000269305.4:c.535C>T   \n",
       "212         missense_variant  MODERATE  PIK3CA   ENST00000263967.3:c.1633G>A   \n",
       "221  downstream_gene_variant  MODIFIER  SNAPC4                           NaN   \n",
       "227         missense_variant  MODERATE   FANCA   ENST00000389301.3:c.1046C>T   \n",
       "\n",
       "                              HGVSp  \\\n",
       "32    ENSP00000264741.5:p.Gly507Glu   \n",
       "88   ENSP00000323856.4:p.Thr4099%3D   \n",
       "119    ENSP00000256078.4:p.Gly13Asp   \n",
       "134  ENSP00000325527.5:p.Arg1137His   \n",
       "154   ENSP00000352336.3:p.Asp383%3D   \n",
       "157   ENSP00000269305.4:p.His179Tyr   \n",
       "212   ENSP00000263967.3:p.Glu545Lys   \n",
       "221                             NaN   \n",
       "227   ENSP00000373952.3:p.Ala349Val   \n",
       "\n",
       "                                      CLIN_SIG                  PolyPhen  \\\n",
       "32                                      benign              benign(0.02)   \n",
       "88                                      benign                       NaN   \n",
       "119               pathogenic&likely_pathogenic              benign(0.44)   \n",
       "134                     uncertain_significance  possibly_damaging(0.615)   \n",
       "154                                     benign                       NaN   \n",
       "157               pathogenic&likely_pathogenic  probably_damaging(0.991)   \n",
       "212  not_provided&pathogenic&likely_pathogenic  probably_damaging(0.909)   \n",
       "221                                     benign                       NaN   \n",
       "227                     uncertain_significance             benign(0.138)   \n",
       "\n",
       "                  SIFT  \n",
       "32   deleterious(0.01)  \n",
       "88                 NaN  \n",
       "119  deleterious(0.04)  \n",
       "134    tolerated(0.17)  \n",
       "154                NaN  \n",
       "157     deleterious(0)  \n",
       "212  deleterious(0.02)  \n",
       "221                NaN  \n",
       "227    tolerated(0.14)  "
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf_clinvar"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['missense_variant', 'synonymous_variant', 'intron_variant',\n",
       "       'downstream_gene_variant',\n",
       "       'intron_variant&non_coding_transcript_variant',\n",
       "       'upstream_gene_variant', 'non_coding_transcript_exon_variant',\n",
       "       'stop_gained', '5_prime_UTR_variant', '3_prime_UTR_variant',\n",
       "       'splice_region_variant&synonymous_variant',\n",
       "       'splice_region_variant&intron_variant',\n",
       "       'splice_region_variant&5_prime_UTR_variant',\n",
       "       'missense_variant&splice_region_variant'], dtype=object)"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "#type of variants in the vcf file\n",
    "vcf['Consequence'].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "non_coding_mutations=['intron_variant&non_coding_transcript_variant','splice_region_variant&5_prime_UTR_variant', '3_prime_UTR_variant','non_coding_transcript_exon_variant','splice_acceptor_variant','splice_region_variant&intron_variant', '5_prime_UTR_variant']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['intron_variant&non_coding_transcript_variant',\n",
       " 'splice_region_variant&5_prime_UTR_variant',\n",
       " '3_prime_UTR_variant',\n",
       " 'non_coding_transcript_exon_variant',\n",
       " 'splice_acceptor_variant',\n",
       " 'splice_region_variant&intron_variant',\n",
       " '5_prime_UTR_variant']"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "non_coding_mutations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#drop non-coding variants\n",
    "coding_variants=variants[variants['Consequence'].apply(lambda x:x not in ['intron_variant&non_coding_transcript_variant','splice_region_variant&5_prime_UTR_variant', '3_prime_UTR_variant','non_coding_transcript_exon_variant','splice_acceptor_variant','splice_region_variant&intron_variant', '5_prime_UTR_variant'])]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#generating matrix for required field\n",
    "selection=coding_variants[['SYMBOL', 'seqnames','start','end','Consequence','IMPACT','CLIN_SIG','PolyPhen', 'SIFT',]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Checking common positions across two samples"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "vcf_1=pd.read_table('APD002_T2.HM.consensus.tab', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>width</th>\n",
       "      <th>strand</th>\n",
       "      <th>AD</th>\n",
       "      <th>AD.1</th>\n",
       "      <th>AF</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>SYMBOL</th>\n",
       "      <th>HGVSc</th>\n",
       "      <th>HGVSp</th>\n",
       "      <th>CLIN_SIG</th>\n",
       "      <th>PolyPhen</th>\n",
       "      <th>SIFT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1961657</td>\n",
       "      <td>1961657</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>10</td>\n",
       "      <td>0.164</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GABRD</td>\n",
       "      <td>ENST00000378585.4:c.1295G&gt;A</td>\n",
       "      <td>ENSP00000367848.4:p.Arg432His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>16259538</td>\n",
       "      <td>16259538</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPEN</td>\n",
       "      <td>ENST00000375759.3:c.6803C&gt;A</td>\n",
       "      <td>ENSP00000364912.3:p.Ala2268Asp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>tolerated_low_confidence(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>25880505</td>\n",
       "      <td>25880505</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>9</td>\n",
       "      <td>0.180</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>LDLRAP1</td>\n",
       "      <td>ENST00000374338.4:c.181C&gt;G</td>\n",
       "      <td>ENSP00000363458.4:p.Pro61Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.914)</td>\n",
       "      <td>tolerated(0.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>1</td>\n",
       "      <td>55088984</td>\n",
       "      <td>55088984</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>71</td>\n",
       "      <td>10</td>\n",
       "      <td>0.123</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAM151A</td>\n",
       "      <td>ENST00000302250.2:c.85G&gt;A</td>\n",
       "      <td>ENSP00000306888.2:p.Ala29Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.75)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>1</td>\n",
       "      <td>86591188</td>\n",
       "      <td>86591188</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>COL24A1</td>\n",
       "      <td>ENST00000370571.2:c.831A&gt;C</td>\n",
       "      <td>ENSP00000359603.2:p.Lys277Asn</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.4)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>1</td>\n",
       "      <td>118634278</td>\n",
       "      <td>118634278</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>4</td>\n",
       "      <td>0.070</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPAG17</td>\n",
       "      <td>ENST00000336338.5:c.1310G&gt;A</td>\n",
       "      <td>ENSP00000337804.5:p.Arg437Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      <td>1</td>\n",
       "      <td>149919163</td>\n",
       "      <td>149919163</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>5</td>\n",
       "      <td>0.074</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OTUD7B</td>\n",
       "      <td>ENST00000369135.4:c.1312T&gt;C</td>\n",
       "      <td>ENSP00000358131.3:p.Ser438Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.367)</td>\n",
       "      <td>tolerated(0.11)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>1</td>\n",
       "      <td>149939389</td>\n",
       "      <td>149939389</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>3</td>\n",
       "      <td>0.049</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OTUD7B</td>\n",
       "      <td>ENST00000369135.4:c.332G&gt;T</td>\n",
       "      <td>ENSP00000358131.3:p.Arg111Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      <td>1</td>\n",
       "      <td>152884003</td>\n",
       "      <td>152884003</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>0.075</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IVL</td>\n",
       "      <td>ENST00000368764.3:c.1730A&gt;G</td>\n",
       "      <td>ENSP00000357753.3:p.Glu577Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.942)</td>\n",
       "      <td>tolerated_low_confidence(0.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>1</td>\n",
       "      <td>153791394</td>\n",
       "      <td>153791394</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>4</td>\n",
       "      <td>0.103</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GATAD2B</td>\n",
       "      <td>ENST00000368655.4:c.470A&gt;G</td>\n",
       "      <td>ENSP00000357644.4:p.Lys157Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.956)</td>\n",
       "      <td>tolerated(0.25)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      <td>1</td>\n",
       "      <td>172571853</td>\n",
       "      <td>172571853</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SUCO</td>\n",
       "      <td>ENST00000263688.3:c.3191A&gt;G</td>\n",
       "      <td>ENSP00000263688.3:p.Asp1064Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      <td>1</td>\n",
       "      <td>180217567</td>\n",
       "      <td>180217567</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>3</td>\n",
       "      <td>0.061</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>LHX4</td>\n",
       "      <td>ENST00000263726.2:c.224T&gt;C</td>\n",
       "      <td>ENSP00000263726.2:p.Val75Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      <td>2</td>\n",
       "      <td>64863545</td>\n",
       "      <td>64863545</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SERTAD2</td>\n",
       "      <td>ENST00000313349.3:c.461C&gt;A</td>\n",
       "      <td>ENSP00000326933.3:p.Pro154His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.818)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      <td>2</td>\n",
       "      <td>220439634</td>\n",
       "      <td>220439634</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>83</td>\n",
       "      <td>4</td>\n",
       "      <td>0.046</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>INHA</td>\n",
       "      <td>ENST00000243786.2:c.487G&gt;A</td>\n",
       "      <td>ENSP00000243786.2:p.Val163Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.46)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      <td>3</td>\n",
       "      <td>31665184</td>\n",
       "      <td>31665184</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>8</td>\n",
       "      <td>0.222</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>STT3B</td>\n",
       "      <td>ENST00000295770.2:c.1558C&gt;A</td>\n",
       "      <td>ENSP00000295770.2:p.His520Asn</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.028)</td>\n",
       "      <td>tolerated(0.44)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      <td>3</td>\n",
       "      <td>37574951</td>\n",
       "      <td>37574951</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.111</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ITGA9</td>\n",
       "      <td>ENST00000264741.5:c.1520G&gt;A</td>\n",
       "      <td>ENSP00000264741.5:p.Gly507Glu</td>\n",
       "      <td>benign</td>\n",
       "      <td>benign(0.02)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      <td>3</td>\n",
       "      <td>41759288</td>\n",
       "      <td>41759288</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>4</td>\n",
       "      <td>0.074</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ULK4</td>\n",
       "      <td>ENST00000301831.4:c.2387A&gt;G</td>\n",
       "      <td>ENSP00000301831.4:p.Asn796Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      <td>3</td>\n",
       "      <td>49137421</td>\n",
       "      <td>49137421</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>4</td>\n",
       "      <td>0.077</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>QARS</td>\n",
       "      <td>ENST00000306125.6:c.1268C&gt;G</td>\n",
       "      <td>ENSP00000307567.6:p.Thr423Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>tolerated(0.06)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      <td>3</td>\n",
       "      <td>113697105</td>\n",
       "      <td>113697105</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>KIAA1407</td>\n",
       "      <td>ENST00000295878.3:c.2534G&gt;A</td>\n",
       "      <td>ENSP00000295878.3:p.Trp845Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      <td>3</td>\n",
       "      <td>134644699</td>\n",
       "      <td>134644699</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>5</td>\n",
       "      <td>0.076</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHB1</td>\n",
       "      <td>ENST00000398015.3:c.100T&gt;A</td>\n",
       "      <td>ENSP00000381097.3:p.Trp34Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      <td>3</td>\n",
       "      <td>151171301</td>\n",
       "      <td>151171301</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>3</td>\n",
       "      <td>0.065</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IGSF10</td>\n",
       "      <td>ENST00000282466.3:c.586T&gt;C</td>\n",
       "      <td>ENSP00000282466.3:p.Phe196Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.005)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      <td>4</td>\n",
       "      <td>56308687</td>\n",
       "      <td>56308687</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>4</td>\n",
       "      <td>0.071</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>CLOCK</td>\n",
       "      <td>ENST00000309964.4:c.2017G&gt;T</td>\n",
       "      <td>ENSP00000308741.4:p.Gly673Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      <td>4</td>\n",
       "      <td>140599783</td>\n",
       "      <td>140599783</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>4</td>\n",
       "      <td>0.073</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MGST2</td>\n",
       "      <td>ENST00000265498.1:c.145G&gt;A</td>\n",
       "      <td>ENSP00000265498.1:p.Val49Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.96)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>5</td>\n",
       "      <td>10430082</td>\n",
       "      <td>10430082</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>10</td>\n",
       "      <td>0.167</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MARCH6</td>\n",
       "      <td>ENST00000274140.5:c.2584A&gt;G</td>\n",
       "      <td>ENSP00000274140.4:p.Ile862Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.78)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      <td>5</td>\n",
       "      <td>131940554</td>\n",
       "      <td>131940554</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>4</td>\n",
       "      <td>0.087</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RAD50</td>\n",
       "      <td>ENST00000265335.6:c.2581C&gt;A</td>\n",
       "      <td>ENSP00000265335.6:p.Gln861Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.103)</td>\n",
       "      <td>tolerated(0.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      <td>5</td>\n",
       "      <td>150924038</td>\n",
       "      <td>150924038</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>57</td>\n",
       "      <td>5</td>\n",
       "      <td>0.081</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAT2</td>\n",
       "      <td>ENST00000261800.5:c.6650C&gt;T</td>\n",
       "      <td>ENSP00000261800.5:p.Thr2217Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.255)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      <td>6</td>\n",
       "      <td>43180949</td>\n",
       "      <td>43180949</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>10</td>\n",
       "      <td>0.156</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CUL9</td>\n",
       "      <td>ENST00000252050.4:c.5275C&gt;T</td>\n",
       "      <td>ENSP00000252050.4:p.Arg1759Trp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      <td>6</td>\n",
       "      <td>89974138</td>\n",
       "      <td>89974138</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>6</td>\n",
       "      <td>0.107</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GABRR2</td>\n",
       "      <td>ENST00000402938.3:c.1079G&gt;A</td>\n",
       "      <td>ENSP00000386029.4:p.Arg360Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.034)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      <td>6</td>\n",
       "      <td>111587333</td>\n",
       "      <td>111587333</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>4</td>\n",
       "      <td>0.118</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KIAA1919</td>\n",
       "      <td>ENST00000368847.4:c.568G&gt;T</td>\n",
       "      <td>ENSP00000357840.4:p.Gly190Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      <td>7</td>\n",
       "      <td>55260474</td>\n",
       "      <td>55260474</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>3</td>\n",
       "      <td>0.045</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EGFR</td>\n",
       "      <td>ENST00000275493.2:c.2641A&gt;G</td>\n",
       "      <td>ENSP00000275493.2:p.Met881Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.583)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>30</td>\n",
       "      <td>7</td>\n",
       "      <td>65553885</td>\n",
       "      <td>65553885</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ASL</td>\n",
       "      <td>ENST00000304874.9:c.810C&gt;A</td>\n",
       "      <td>ENSP00000307188.9:p.Phe270Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.982)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>31</td>\n",
       "      <td>7</td>\n",
       "      <td>100817842</td>\n",
       "      <td>100817842</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>131</td>\n",
       "      <td>21</td>\n",
       "      <td>0.138</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NAT16</td>\n",
       "      <td>ENST00000300303.2:c.247C&gt;T</td>\n",
       "      <td>ENSP00000300303.2:p.Arg83Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.077)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>32</td>\n",
       "      <td>7</td>\n",
       "      <td>113558433</td>\n",
       "      <td>113558433</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>55</td>\n",
       "      <td>3</td>\n",
       "      <td>0.052</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PPP1R3A</td>\n",
       "      <td>ENST00000284601.3:c.619T&gt;C</td>\n",
       "      <td>ENSP00000284601.3:p.Phe207Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.971)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>33</td>\n",
       "      <td>7</td>\n",
       "      <td>140064270</td>\n",
       "      <td>140064270</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>0.091</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SLC37A3</td>\n",
       "      <td>ENST00000326232.9:c.313G&gt;A</td>\n",
       "      <td>ENSP00000321498.9:p.Val105Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.65)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>34</td>\n",
       "      <td>7</td>\n",
       "      <td>143098692</td>\n",
       "      <td>143098692</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHA1</td>\n",
       "      <td>ENST00000275815.3:c.157G&gt;A</td>\n",
       "      <td>ENSP00000275815.3:p.Glu53Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.043)</td>\n",
       "      <td>tolerated(0.44)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35</td>\n",
       "      <td>8</td>\n",
       "      <td>41543691</td>\n",
       "      <td>41543691</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>18</td>\n",
       "      <td>0.340</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ANK1</td>\n",
       "      <td>ENST00000265709.8:c.4492C&gt;T</td>\n",
       "      <td>ENSP00000265709.8:p.Arg1498Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.902)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36</td>\n",
       "      <td>8</td>\n",
       "      <td>144688268</td>\n",
       "      <td>144688268</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>94</td>\n",
       "      <td>4</td>\n",
       "      <td>0.041</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PYCRL</td>\n",
       "      <td>ENST00000220966.6:c.642C&gt;A</td>\n",
       "      <td>ENSP00000220966.6:p.Ser214Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.851)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>37</td>\n",
       "      <td>9</td>\n",
       "      <td>85615980</td>\n",
       "      <td>85615980</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>6</td>\n",
       "      <td>0.103</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RASEF</td>\n",
       "      <td>ENST00000376447.3:c.1268G&gt;A</td>\n",
       "      <td>ENSP00000365630.3:p.Arg423Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.018)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>38</td>\n",
       "      <td>9</td>\n",
       "      <td>107288802</td>\n",
       "      <td>107288802</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>0.073</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR13C4</td>\n",
       "      <td>ENST00000277216.3:c.689C&gt;T</td>\n",
       "      <td>ENSP00000277216.3:p.Ser230Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.876)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>39</td>\n",
       "      <td>10</td>\n",
       "      <td>115610067</td>\n",
       "      <td>115610067</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>5</td>\n",
       "      <td>0.102</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DCLRE1A</td>\n",
       "      <td>ENST00000361384.2:c.797A&gt;G</td>\n",
       "      <td>ENSP00000355185.2:p.Glu266Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.3)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>40</td>\n",
       "      <td>10</td>\n",
       "      <td>116232769</td>\n",
       "      <td>116232769</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>4</td>\n",
       "      <td>0.061</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>ABLIM1</td>\n",
       "      <td>ENST00000277895.5:c.1242T&gt;A</td>\n",
       "      <td>ENSP00000277895.5:p.Tyr414Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>41</td>\n",
       "      <td>10</td>\n",
       "      <td>128829989</td>\n",
       "      <td>128829989</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>3</td>\n",
       "      <td>0.058</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DOCK1</td>\n",
       "      <td>ENST00000280333.6:c.1637A&gt;G</td>\n",
       "      <td>ENSP00000280333.6:p.Asp546Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.589)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>42</td>\n",
       "      <td>11</td>\n",
       "      <td>6291067</td>\n",
       "      <td>6291067</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>4</td>\n",
       "      <td>0.114</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CCKBR</td>\n",
       "      <td>ENST00000334619.2:c.320G&gt;A</td>\n",
       "      <td>ENSP00000335544.2:p.Cys107Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>43</td>\n",
       "      <td>11</td>\n",
       "      <td>59271320</td>\n",
       "      <td>59271320</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>7</td>\n",
       "      <td>0.137</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR4D11</td>\n",
       "      <td>ENST00000313253.1:c.272C&gt;T</td>\n",
       "      <td>ENSP00000320077.1:p.Thr91Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.046)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>44</td>\n",
       "      <td>11</td>\n",
       "      <td>59861497</td>\n",
       "      <td>59861497</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MS4A2</td>\n",
       "      <td>ENST00000278888.3:c.598A&gt;T</td>\n",
       "      <td>ENSP00000278888.3:p.Thr200Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.708)</td>\n",
       "      <td>tolerated(0.37)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>45</td>\n",
       "      <td>11</td>\n",
       "      <td>68676071</td>\n",
       "      <td>68676071</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>IGHMBP2</td>\n",
       "      <td>ENST00000255078.3:c.519A&gt;T</td>\n",
       "      <td>ENSP00000255078.3:p.Arg173Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.87)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>46</td>\n",
       "      <td>11</td>\n",
       "      <td>124189456</td>\n",
       "      <td>124189456</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>9</td>\n",
       "      <td>0.237</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR8D2</td>\n",
       "      <td>ENST00000357438.2:c.638C&gt;T</td>\n",
       "      <td>ENSP00000350022.2:p.Ala213Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.869)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>47</td>\n",
       "      <td>12</td>\n",
       "      <td>25398281</td>\n",
       "      <td>25398281</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>14</td>\n",
       "      <td>0.326</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>ENST00000256078.4:c.38G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly13Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.44)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>48</td>\n",
       "      <td>12</td>\n",
       "      <td>56477631</td>\n",
       "      <td>56477631</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>71</td>\n",
       "      <td>23</td>\n",
       "      <td>0.245</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ERBB3</td>\n",
       "      <td>ENST00000267101.3:c.179T&gt;A</td>\n",
       "      <td>ENSP00000267101.3:p.Met60Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.147)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>49</td>\n",
       "      <td>12</td>\n",
       "      <td>57881973</td>\n",
       "      <td>57881973</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>12</td>\n",
       "      <td>0.333</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MARS</td>\n",
       "      <td>ENST00000262027.5:c.100G&gt;C</td>\n",
       "      <td>ENSP00000262027.5:p.Gly34Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.38)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>50</td>\n",
       "      <td>12</td>\n",
       "      <td>124311390</td>\n",
       "      <td>124311390</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>3</td>\n",
       "      <td>0.057</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DNAH10</td>\n",
       "      <td>ENST00000409039.3:c.3982T&gt;C</td>\n",
       "      <td>ENSP00000386770.3:p.Phe1328Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>51</td>\n",
       "      <td>13</td>\n",
       "      <td>58207656</td>\n",
       "      <td>58207656</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>23</td>\n",
       "      <td>0.329</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PCDH17</td>\n",
       "      <td>ENST00000377918.3:c.976C&gt;A</td>\n",
       "      <td>ENSP00000367151.3:p.Leu326Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.962)</td>\n",
       "      <td>tolerated(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>52</td>\n",
       "      <td>13</td>\n",
       "      <td>99116014</td>\n",
       "      <td>99116014</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>90</td>\n",
       "      <td>5</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>STK24</td>\n",
       "      <td>ENST00000376547.3:c.896T&gt;C</td>\n",
       "      <td>ENSP00000365730.3:p.Leu299Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>53</td>\n",
       "      <td>15</td>\n",
       "      <td>43821035</td>\n",
       "      <td>43821035</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.095</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MAP1A</td>\n",
       "      <td>ENST00000300231.5:c.7364C&gt;A</td>\n",
       "      <td>ENSP00000300231.5:p.Pro2455Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.964)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>54</td>\n",
       "      <td>15</td>\n",
       "      <td>48779562</td>\n",
       "      <td>48779562</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>7</td>\n",
       "      <td>0.089</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FBN1</td>\n",
       "      <td>ENST00000316623.5:c.3410G&gt;A</td>\n",
       "      <td>ENSP00000325527.5:p.Arg1137His</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>possibly_damaging(0.615)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>55</td>\n",
       "      <td>15</td>\n",
       "      <td>74337173</td>\n",
       "      <td>74337173</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>9</td>\n",
       "      <td>0.220</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PML</td>\n",
       "      <td>ENST00000268058.3:c.2473C&gt;T</td>\n",
       "      <td>ENSP00000268058.3:p.Arg825Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.09)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>56</td>\n",
       "      <td>15</td>\n",
       "      <td>86122590</td>\n",
       "      <td>86122590</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>AKAP13</td>\n",
       "      <td>ENST00000361243.2:c.1291G&gt;A</td>\n",
       "      <td>ENSP00000354718.2:p.Gly431Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.001)</td>\n",
       "      <td>tolerated(0.12)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>57</td>\n",
       "      <td>16</td>\n",
       "      <td>2002875</td>\n",
       "      <td>2002875</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>3</td>\n",
       "      <td>0.047</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RPL3L</td>\n",
       "      <td>ENST00000268661.7:c.365G&gt;T</td>\n",
       "      <td>ENSP00000268661.7:p.Trp122Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.034)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>58</td>\n",
       "      <td>16</td>\n",
       "      <td>22361152</td>\n",
       "      <td>22361152</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>4</td>\n",
       "      <td>0.129</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>CDR2</td>\n",
       "      <td>ENST00000268383.2:c.211G&gt;T</td>\n",
       "      <td>ENSP00000268383.2:p.Glu71Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>59</td>\n",
       "      <td>16</td>\n",
       "      <td>23672585</td>\n",
       "      <td>23672585</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>4</td>\n",
       "      <td>0.060</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DCTN5</td>\n",
       "      <td>ENST00000300087.2:c.331G&gt;A</td>\n",
       "      <td>ENSP00000300087.2:p.Gly111Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.976)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>60</td>\n",
       "      <td>16</td>\n",
       "      <td>23716366</td>\n",
       "      <td>23716366</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>5</td>\n",
       "      <td>0.098</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ERN2</td>\n",
       "      <td>ENST00000256797.4:c.836G&gt;T</td>\n",
       "      <td>ENSP00000256797.4:p.Gly279Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.954)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>61</td>\n",
       "      <td>17</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578395</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TP53</td>\n",
       "      <td>ENST00000269305.4:c.535C&gt;T</td>\n",
       "      <td>ENSP00000269305.4:p.His179Tyr</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>62</td>\n",
       "      <td>17</td>\n",
       "      <td>29162948</td>\n",
       "      <td>29162948</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ATAD5</td>\n",
       "      <td>ENST00000321990.4:c.1849G&gt;A</td>\n",
       "      <td>ENSP00000313171.4:p.Val617Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.037)</td>\n",
       "      <td>tolerated(0.15)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>63</td>\n",
       "      <td>18</td>\n",
       "      <td>21121329</td>\n",
       "      <td>21121329</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.083</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NPC1</td>\n",
       "      <td>ENST00000269228.5:c.2314T&gt;C</td>\n",
       "      <td>ENSP00000269228.4:p.Phe772Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>64</td>\n",
       "      <td>18</td>\n",
       "      <td>72250923</td>\n",
       "      <td>72250923</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>5</td>\n",
       "      <td>0.106</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CNDP1</td>\n",
       "      <td>ENST00000358821.3:c.1436C&gt;T</td>\n",
       "      <td>ENSP00000351682.3:p.Ser479Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.993)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>65</td>\n",
       "      <td>18</td>\n",
       "      <td>72347107</td>\n",
       "      <td>72347107</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>4</td>\n",
       "      <td>0.133</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF407</td>\n",
       "      <td>ENST00000299687.5:c.4132G&gt;A</td>\n",
       "      <td>ENSP00000299687.4:p.Gly1378Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.01)</td>\n",
       "      <td>tolerated(0.05)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>66</td>\n",
       "      <td>19</td>\n",
       "      <td>17949188</td>\n",
       "      <td>17949188</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>4</td>\n",
       "      <td>0.133</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>JAK3</td>\n",
       "      <td>ENST00000458235.1:c.1453C&gt;A</td>\n",
       "      <td>ENSP00000391676.1:p.Leu485Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.931)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>67</td>\n",
       "      <td>19</td>\n",
       "      <td>18555576</td>\n",
       "      <td>18555576</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>7</td>\n",
       "      <td>0.127</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ELL</td>\n",
       "      <td>ENST00000262809.4:c.1852C&gt;A</td>\n",
       "      <td>ENSP00000262809.3:p.Gln618Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.104)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>68</td>\n",
       "      <td>19</td>\n",
       "      <td>30936021</td>\n",
       "      <td>30936021</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF536</td>\n",
       "      <td>ENST00000355537.3:c.1552G&gt;T</td>\n",
       "      <td>ENSP00000347730.1:p.Val518Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.006)</td>\n",
       "      <td>tolerated(0.49)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>69</td>\n",
       "      <td>21</td>\n",
       "      <td>17138323</td>\n",
       "      <td>17138323</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>3</td>\n",
       "      <td>0.071</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>USP25</td>\n",
       "      <td>ENST00000285679.6:c.131A&gt;T</td>\n",
       "      <td>ENSP00000285679.6:p.Asn44Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.548)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>70</td>\n",
       "      <td>21</td>\n",
       "      <td>44841662</td>\n",
       "      <td>44841662</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>20</td>\n",
       "      <td>4</td>\n",
       "      <td>0.167</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SIK1</td>\n",
       "      <td>ENST00000270162.6:c.355G&gt;C</td>\n",
       "      <td>ENSP00000270162.6:p.Gly119Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.981)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>71</td>\n",
       "      <td>22</td>\n",
       "      <td>32112183</td>\n",
       "      <td>32112183</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>17</td>\n",
       "      <td>0.415</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>PRR14L</td>\n",
       "      <td>ENST00000327423.6:c.1642C&gt;T</td>\n",
       "      <td>ENSP00000331845.6:p.Gln548Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>72</td>\n",
       "      <td>22</td>\n",
       "      <td>47103746</td>\n",
       "      <td>47103746</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>0.067</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CERK</td>\n",
       "      <td>ENST00000216264.8:c.709C&gt;T</td>\n",
       "      <td>ENSP00000216264.8:p.Pro237Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>73</td>\n",
       "      <td>22</td>\n",
       "      <td>50589167</td>\n",
       "      <td>50589167</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>3</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MOV10L1</td>\n",
       "      <td>ENST00000262794.5:c.2731G&gt;A</td>\n",
       "      <td>ENSP00000262794.5:p.Val911Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.987)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74</td>\n",
       "      <td>22</td>\n",
       "      <td>50751904</td>\n",
       "      <td>50751904</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>0.083</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DENND6B</td>\n",
       "      <td>ENST00000413817.3:c.1318C&gt;A</td>\n",
       "      <td>ENSP00000391524.2:p.Pro440Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.997)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>75</td>\n",
       "      <td>X</td>\n",
       "      <td>13056613</td>\n",
       "      <td>13056613</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>5</td>\n",
       "      <td>0.078</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>FAM9C</td>\n",
       "      <td>ENST00000333995.3:c.448C&gt;T</td>\n",
       "      <td>ENSP00000334430.3:p.Gln150Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>76</td>\n",
       "      <td>X</td>\n",
       "      <td>106844796</td>\n",
       "      <td>106844796</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FRMPD3</td>\n",
       "      <td>ENST00000276185.4:c.3626A&gt;T</td>\n",
       "      <td>ENSP00000276185.4:p.Gln1209Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated_low_confidence(0.11)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>77</td>\n",
       "      <td>3</td>\n",
       "      <td>108301916</td>\n",
       "      <td>108301916</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>4</td>\n",
       "      <td>0.130</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KIAA1524</td>\n",
       "      <td>ENST00000295746.8:c.265A&gt;G</td>\n",
       "      <td>ENSP00000295746.7:p.Thr89Ala</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.108)</td>\n",
       "      <td>tolerated(0.14)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>78</td>\n",
       "      <td>3</td>\n",
       "      <td>178936091</td>\n",
       "      <td>178936091</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>9</td>\n",
       "      <td>0.185</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>ENST00000263967.3:c.1633G&gt;A</td>\n",
       "      <td>ENSP00000263967.3:p.Glu545Lys</td>\n",
       "      <td>not_provided&amp;pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.909)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>79</td>\n",
       "      <td>5</td>\n",
       "      <td>63986568</td>\n",
       "      <td>63986568</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>16</td>\n",
       "      <td>3</td>\n",
       "      <td>0.190</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAM159B</td>\n",
       "      <td>ENST00000389074.5:c.118G&gt;T</td>\n",
       "      <td>ENSP00000373726.5:p.Ala40Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.136)</td>\n",
       "      <td>tolerated(0.83)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80</td>\n",
       "      <td>11</td>\n",
       "      <td>67375868</td>\n",
       "      <td>67375868</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>6</td>\n",
       "      <td>0.113</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NDUFV1</td>\n",
       "      <td>ENST00000322776.6:c.74C&gt;T</td>\n",
       "      <td>ENSP00000322450.6:p.Thr25Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.021)</td>\n",
       "      <td>deleterious_low_confidence(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>81</td>\n",
       "      <td>16</td>\n",
       "      <td>89858916</td>\n",
       "      <td>89858916</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>4</td>\n",
       "      <td>0.112</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FANCA</td>\n",
       "      <td>ENST00000389301.3:c.1046C&gt;T</td>\n",
       "      <td>ENSP00000373952.3:p.Ala349Val</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>benign(0.138)</td>\n",
       "      <td>tolerated(0.14)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>82</td>\n",
       "      <td>17</td>\n",
       "      <td>79571705</td>\n",
       "      <td>79571705</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>5</td>\n",
       "      <td>0.147</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NPLOC4</td>\n",
       "      <td>ENST00000331134.6:c.703G&gt;T</td>\n",
       "      <td>ENSP00000331487.5:p.Asp235Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.949)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>83</td>\n",
       "      <td>20</td>\n",
       "      <td>61867626</td>\n",
       "      <td>61867626</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>88</td>\n",
       "      <td>5</td>\n",
       "      <td>0.063</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>BIRC7</td>\n",
       "      <td>ENST00000217169.3:c.178C&gt;T</td>\n",
       "      <td>ENSP00000217169.3:p.Arg60Trp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.895)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>84</td>\n",
       "      <td>X</td>\n",
       "      <td>51151007</td>\n",
       "      <td>51151007</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>13</td>\n",
       "      <td>0.258</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CXorf67</td>\n",
       "      <td>ENST00000342995.2:c.1139C&gt;T</td>\n",
       "      <td>ENSP00000342680.2:p.Ala380Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.013)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   seqnames      start        end  width strand   AD  AD.1     AF  \\\n",
       "0         1    1961657    1961657      1      *   51    10  0.164   \n",
       "1         1   16259538   16259538      1      *   33     4  0.108   \n",
       "2         1   25880505   25880505      1      *   41     9  0.180   \n",
       "3         1   55088984   55088984      1      *   71    10  0.123   \n",
       "4         1   86591188   86591188      1      *   38     5  0.116   \n",
       "5         1  118634278  118634278      1      *   53     4  0.070   \n",
       "6         1  149919163  149919163      1      *   63     5  0.074   \n",
       "7         1  149939389  149939389      1      *   58     3  0.049   \n",
       "8         1  152884003  152884003      1      *   49     4  0.075   \n",
       "9         1  153791394  153791394      1      *   35     4  0.103   \n",
       "10        1  172571853  172571853      1      *   28     4  0.125   \n",
       "11        1  180217567  180217567      1      *   46     3  0.061   \n",
       "12        2   64863545   64863545      1      *   33     4  0.108   \n",
       "13        2  220439634  220439634      1      *   83     4  0.046   \n",
       "14        3   31665184   31665184      1      *   28     8  0.222   \n",
       "15        3   37574951   37574951      1      *   32     4  0.111   \n",
       "16        3   41759288   41759288      1      *   50     4  0.074   \n",
       "17        3   49137421   49137421      1      *   48     4  0.077   \n",
       "18        3  113697105  113697105      1      *   37     4  0.098   \n",
       "19        3  134644699  134644699      1      *   61     5  0.076   \n",
       "20        3  151171301  151171301      1      *   43     3  0.065   \n",
       "21        4   56308687   56308687      1      *   52     4  0.071   \n",
       "22        4  140599783  140599783      1      *   51     4  0.073   \n",
       "23        5   10430082   10430082      1      *   50    10  0.167   \n",
       "24        5  131940554  131940554      1      *   42     4  0.087   \n",
       "25        5  150924038  150924038      1      *   57     5  0.081   \n",
       "26        6   43180949   43180949      1      *   54    10  0.156   \n",
       "27        6   89974138   89974138      1      *   50     6  0.107   \n",
       "28        6  111587333  111587333      1      *   30     4  0.118   \n",
       "29        7   55260474   55260474      1      *   63     3  0.045   \n",
       "30        7   65553885   65553885      1      *   46     4  0.080   \n",
       "31        7  100817842  100817842      1      *  131    21  0.138   \n",
       "32        7  113558433  113558433      1      *   55     3  0.052   \n",
       "33        7  140064270  140064270      1      *   30     3  0.091   \n",
       "34        7  143098692  143098692      1      *   47     4  0.078   \n",
       "35        8   41543691   41543691      1      *   35    18  0.340   \n",
       "36        8  144688268  144688268      1      *   94     4  0.041   \n",
       "37        9   85615980   85615980      1      *   52     6  0.103   \n",
       "38        9  107288802  107288802      1      *   38     3  0.073   \n",
       "39       10  115610067  115610067      1      *   44     5  0.102   \n",
       "40       10  116232769  116232769      1      *   62     4  0.061   \n",
       "41       10  128829989  128829989      1      *   49     3  0.058   \n",
       "42       11    6291067    6291067      1      *   31     4  0.114   \n",
       "43       11   59271320   59271320      1      *   44     7  0.137   \n",
       "44       11   59861497   59861497      1      *   46     4  0.080   \n",
       "45       11   68676071   68676071      1      *   38     5  0.116   \n",
       "46       11  124189456  124189456      1      *   29     9  0.237   \n",
       "47       12   25398281   25398281      1      *   29    14  0.326   \n",
       "48       12   56477631   56477631      1      *   71    23  0.245   \n",
       "49       12   57881973   57881973      1      *   24    12  0.333   \n",
       "50       12  124311390  124311390      1      *   50     3  0.057   \n",
       "51       13   58207656   58207656      1      *   47    23  0.329   \n",
       "52       13   99116014   99116014      1      *   90     5  0.053   \n",
       "53       15   43821035   43821035      1      *   38     4  0.095   \n",
       "54       15   48779562   48779562      1      *   72     7  0.089   \n",
       "55       15   74337173   74337173      1      *   32     9  0.220   \n",
       "56       15   86122590   86122590      1      *   29     3  0.094   \n",
       "57       16    2002875    2002875      1      *   61     3  0.047   \n",
       "58       16   22361152   22361152      1      *   27     4  0.129   \n",
       "59       16   23672585   23672585      1      *   63     4  0.060   \n",
       "60       16   23716366   23716366      1      *   46     5  0.098   \n",
       "61       17    7578395    7578395      1      *   23     7  0.233   \n",
       "62       17   29162948   29162948      1      *   28     4  0.125   \n",
       "63       18   21121329   21121329      1      *   33     3  0.083   \n",
       "64       18   72250923   72250923      1      *   42     5  0.106   \n",
       "65       18   72347107   72347107      1      *   26     4  0.133   \n",
       "66       19   17949188   17949188      1      *   26     4  0.133   \n",
       "67       19   18555576   18555576      1      *   48     7  0.127   \n",
       "68       19   30936021   30936021      1      *   29     3  0.094   \n",
       "69       21   17138323   17138323      1      *   39     3  0.071   \n",
       "70       21   44841662   44841662      1      *   20     4  0.167   \n",
       "71       22   32112183   32112183      1      *   24    17  0.415   \n",
       "72       22   47103746   47103746      1      *   42     3  0.067   \n",
       "73       22   50589167   50589167      1      *   54     3  0.053   \n",
       "74       22   50751904   50751904      1      *   44     4  0.083   \n",
       "75        X   13056613   13056613      1      *   59     5  0.078   \n",
       "76        X  106844796  106844796      1      *   37     4  0.098   \n",
       "77        3  108301916  108301916      1      *   32     4  0.130   \n",
       "78        3  178936091  178936091      1      *   43     9  0.185   \n",
       "79        5   63986568   63986568      1      *   16     3  0.190   \n",
       "80       11   67375868   67375868      1      *   53     6  0.113   \n",
       "81       16   89858916   89858916      1      *   38     4  0.112   \n",
       "82       17   79571705   79571705      1      *   33     5  0.147   \n",
       "83       20   61867626   61867626      1      *   88     5  0.063   \n",
       "84        X   51151007   51151007      1      *   38    13  0.258   \n",
       "\n",
       "                               Consequence    IMPACT    SYMBOL  \\\n",
       "0                         missense_variant  MODERATE     GABRD   \n",
       "1                         missense_variant  MODERATE      SPEN   \n",
       "2                         missense_variant  MODERATE   LDLRAP1   \n",
       "3                         missense_variant  MODERATE   FAM151A   \n",
       "4                         missense_variant  MODERATE   COL24A1   \n",
       "5                         missense_variant  MODERATE    SPAG17   \n",
       "6                         missense_variant  MODERATE    OTUD7B   \n",
       "7                         missense_variant  MODERATE    OTUD7B   \n",
       "8                         missense_variant  MODERATE       IVL   \n",
       "9                         missense_variant  MODERATE   GATAD2B   \n",
       "10                        missense_variant  MODERATE      SUCO   \n",
       "11                        missense_variant  MODERATE      LHX4   \n",
       "12                        missense_variant  MODERATE   SERTAD2   \n",
       "13                        missense_variant  MODERATE      INHA   \n",
       "14                        missense_variant  MODERATE     STT3B   \n",
       "15                        missense_variant  MODERATE     ITGA9   \n",
       "16                        missense_variant  MODERATE      ULK4   \n",
       "17                        missense_variant  MODERATE      QARS   \n",
       "18                             stop_gained      HIGH  KIAA1407   \n",
       "19                        missense_variant  MODERATE     EPHB1   \n",
       "20                        missense_variant  MODERATE    IGSF10   \n",
       "21                             stop_gained      HIGH     CLOCK   \n",
       "22                        missense_variant  MODERATE     MGST2   \n",
       "23                        missense_variant  MODERATE    MARCH6   \n",
       "24                        missense_variant  MODERATE     RAD50   \n",
       "25                        missense_variant  MODERATE      FAT2   \n",
       "26                        missense_variant  MODERATE      CUL9   \n",
       "27                        missense_variant  MODERATE    GABRR2   \n",
       "28                        missense_variant  MODERATE  KIAA1919   \n",
       "29                        missense_variant  MODERATE      EGFR   \n",
       "30                        missense_variant  MODERATE       ASL   \n",
       "31                        missense_variant  MODERATE     NAT16   \n",
       "32                        missense_variant  MODERATE   PPP1R3A   \n",
       "33                        missense_variant  MODERATE   SLC37A3   \n",
       "34                        missense_variant  MODERATE     EPHA1   \n",
       "35                        missense_variant  MODERATE      ANK1   \n",
       "36                        missense_variant  MODERATE     PYCRL   \n",
       "37                        missense_variant  MODERATE     RASEF   \n",
       "38                        missense_variant  MODERATE    OR13C4   \n",
       "39                        missense_variant  MODERATE   DCLRE1A   \n",
       "40                             stop_gained      HIGH    ABLIM1   \n",
       "41                        missense_variant  MODERATE     DOCK1   \n",
       "42                        missense_variant  MODERATE     CCKBR   \n",
       "43                        missense_variant  MODERATE    OR4D11   \n",
       "44                        missense_variant  MODERATE     MS4A2   \n",
       "45                        missense_variant  MODERATE   IGHMBP2   \n",
       "46                        missense_variant  MODERATE     OR8D2   \n",
       "47                        missense_variant  MODERATE      KRAS   \n",
       "48                        missense_variant  MODERATE     ERBB3   \n",
       "49                        missense_variant  MODERATE      MARS   \n",
       "50                        missense_variant  MODERATE    DNAH10   \n",
       "51                        missense_variant  MODERATE    PCDH17   \n",
       "52                        missense_variant  MODERATE     STK24   \n",
       "53                        missense_variant  MODERATE     MAP1A   \n",
       "54                        missense_variant  MODERATE      FBN1   \n",
       "55                        missense_variant  MODERATE       PML   \n",
       "56                        missense_variant  MODERATE    AKAP13   \n",
       "57  missense_variant&splice_region_variant  MODERATE     RPL3L   \n",
       "58                             stop_gained      HIGH      CDR2   \n",
       "59                        missense_variant  MODERATE     DCTN5   \n",
       "60                        missense_variant  MODERATE      ERN2   \n",
       "61                        missense_variant  MODERATE      TP53   \n",
       "62                        missense_variant  MODERATE     ATAD5   \n",
       "63                        missense_variant  MODERATE      NPC1   \n",
       "64                        missense_variant  MODERATE     CNDP1   \n",
       "65                        missense_variant  MODERATE    ZNF407   \n",
       "66                        missense_variant  MODERATE      JAK3   \n",
       "67                        missense_variant  MODERATE       ELL   \n",
       "68                        missense_variant  MODERATE    ZNF536   \n",
       "69                        missense_variant  MODERATE     USP25   \n",
       "70                        missense_variant  MODERATE      SIK1   \n",
       "71                             stop_gained      HIGH    PRR14L   \n",
       "72                        missense_variant  MODERATE      CERK   \n",
       "73                        missense_variant  MODERATE   MOV10L1   \n",
       "74                        missense_variant  MODERATE   DENND6B   \n",
       "75                             stop_gained      HIGH     FAM9C   \n",
       "76                        missense_variant  MODERATE    FRMPD3   \n",
       "77                        missense_variant  MODERATE  KIAA1524   \n",
       "78                        missense_variant  MODERATE    PIK3CA   \n",
       "79                        missense_variant  MODERATE   FAM159B   \n",
       "80  missense_variant&splice_region_variant  MODERATE    NDUFV1   \n",
       "81                        missense_variant  MODERATE     FANCA   \n",
       "82                        missense_variant  MODERATE    NPLOC4   \n",
       "83                        missense_variant  MODERATE     BIRC7   \n",
       "84                        missense_variant  MODERATE   CXorf67   \n",
       "\n",
       "                          HGVSc                           HGVSp  \\\n",
       "0   ENST00000378585.4:c.1295G>A   ENSP00000367848.4:p.Arg432His   \n",
       "1   ENST00000375759.3:c.6803C>A  ENSP00000364912.3:p.Ala2268Asp   \n",
       "2    ENST00000374338.4:c.181C>G    ENSP00000363458.4:p.Pro61Ala   \n",
       "3     ENST00000302250.2:c.85G>A    ENSP00000306888.2:p.Ala29Thr   \n",
       "4    ENST00000370571.2:c.831A>C   ENSP00000359603.2:p.Lys277Asn   \n",
       "5   ENST00000336338.5:c.1310G>A   ENSP00000337804.5:p.Arg437Gln   \n",
       "6   ENST00000369135.4:c.1312T>C   ENSP00000358131.3:p.Ser438Pro   \n",
       "7    ENST00000369135.4:c.332G>T   ENSP00000358131.3:p.Arg111Leu   \n",
       "8   ENST00000368764.3:c.1730A>G   ENSP00000357753.3:p.Glu577Gly   \n",
       "9    ENST00000368655.4:c.470A>G   ENSP00000357644.4:p.Lys157Arg   \n",
       "10  ENST00000263688.3:c.3191A>G  ENSP00000263688.3:p.Asp1064Gly   \n",
       "11   ENST00000263726.2:c.224T>C    ENSP00000263726.2:p.Val75Ala   \n",
       "12   ENST00000313349.3:c.461C>A   ENSP00000326933.3:p.Pro154His   \n",
       "13   ENST00000243786.2:c.487G>A   ENSP00000243786.2:p.Val163Met   \n",
       "14  ENST00000295770.2:c.1558C>A   ENSP00000295770.2:p.His520Asn   \n",
       "15  ENST00000264741.5:c.1520G>A   ENSP00000264741.5:p.Gly507Glu   \n",
       "16  ENST00000301831.4:c.2387A>G   ENSP00000301831.4:p.Asn796Ser   \n",
       "17  ENST00000306125.6:c.1268C>G   ENSP00000307567.6:p.Thr423Arg   \n",
       "18  ENST00000295878.3:c.2534G>A   ENSP00000295878.3:p.Trp845Ter   \n",
       "19   ENST00000398015.3:c.100T>A    ENSP00000381097.3:p.Trp34Arg   \n",
       "20   ENST00000282466.3:c.586T>C   ENSP00000282466.3:p.Phe196Leu   \n",
       "21  ENST00000309964.4:c.2017G>T   ENSP00000308741.4:p.Gly673Ter   \n",
       "22   ENST00000265498.1:c.145G>A    ENSP00000265498.1:p.Val49Ile   \n",
       "23  ENST00000274140.5:c.2584A>G   ENSP00000274140.4:p.Ile862Val   \n",
       "24  ENST00000265335.6:c.2581C>A   ENSP00000265335.6:p.Gln861Lys   \n",
       "25  ENST00000261800.5:c.6650C>T  ENSP00000261800.5:p.Thr2217Ile   \n",
       "26  ENST00000252050.4:c.5275C>T  ENSP00000252050.4:p.Arg1759Trp   \n",
       "27  ENST00000402938.3:c.1079G>A   ENSP00000386029.4:p.Arg360Gln   \n",
       "28   ENST00000368847.4:c.568G>T   ENSP00000357840.4:p.Gly190Cys   \n",
       "29  ENST00000275493.2:c.2641A>G   ENSP00000275493.2:p.Met881Val   \n",
       "30   ENST00000304874.9:c.810C>A   ENSP00000307188.9:p.Phe270Leu   \n",
       "31   ENST00000300303.2:c.247C>T    ENSP00000300303.2:p.Arg83Cys   \n",
       "32   ENST00000284601.3:c.619T>C   ENSP00000284601.3:p.Phe207Leu   \n",
       "33   ENST00000326232.9:c.313G>A   ENSP00000321498.9:p.Val105Ile   \n",
       "34   ENST00000275815.3:c.157G>A    ENSP00000275815.3:p.Glu53Lys   \n",
       "35  ENST00000265709.8:c.4492C>T  ENSP00000265709.8:p.Arg1498Cys   \n",
       "36   ENST00000220966.6:c.642C>A   ENSP00000220966.6:p.Ser214Arg   \n",
       "37  ENST00000376447.3:c.1268G>A   ENSP00000365630.3:p.Arg423Lys   \n",
       "38   ENST00000277216.3:c.689C>T   ENSP00000277216.3:p.Ser230Leu   \n",
       "39   ENST00000361384.2:c.797A>G   ENSP00000355185.2:p.Glu266Gly   \n",
       "40  ENST00000277895.5:c.1242T>A   ENSP00000277895.5:p.Tyr414Ter   \n",
       "41  ENST00000280333.6:c.1637A>G   ENSP00000280333.6:p.Asp546Gly   \n",
       "42   ENST00000334619.2:c.320G>A   ENSP00000335544.2:p.Cys107Tyr   \n",
       "43   ENST00000313253.1:c.272C>T    ENSP00000320077.1:p.Thr91Ile   \n",
       "44   ENST00000278888.3:c.598A>T   ENSP00000278888.3:p.Thr200Ser   \n",
       "45   ENST00000255078.3:c.519A>T   ENSP00000255078.3:p.Arg173Ser   \n",
       "46   ENST00000357438.2:c.638C>T   ENSP00000350022.2:p.Ala213Val   \n",
       "47    ENST00000256078.4:c.38G>A    ENSP00000256078.4:p.Gly13Asp   \n",
       "48   ENST00000267101.3:c.179T>A    ENSP00000267101.3:p.Met60Lys   \n",
       "49   ENST00000262027.5:c.100G>C    ENSP00000262027.5:p.Gly34Arg   \n",
       "50  ENST00000409039.3:c.3982T>C  ENSP00000386770.3:p.Phe1328Leu   \n",
       "51   ENST00000377918.3:c.976C>A   ENSP00000367151.3:p.Leu326Met   \n",
       "52   ENST00000376547.3:c.896T>C   ENSP00000365730.3:p.Leu299Pro   \n",
       "53  ENST00000300231.5:c.7364C>A  ENSP00000300231.5:p.Pro2455Gln   \n",
       "54  ENST00000316623.5:c.3410G>A  ENSP00000325527.5:p.Arg1137His   \n",
       "55  ENST00000268058.3:c.2473C>T   ENSP00000268058.3:p.Arg825Cys   \n",
       "56  ENST00000361243.2:c.1291G>A   ENSP00000354718.2:p.Gly431Arg   \n",
       "57   ENST00000268661.7:c.365G>T   ENSP00000268661.7:p.Trp122Leu   \n",
       "58   ENST00000268383.2:c.211G>T    ENSP00000268383.2:p.Glu71Ter   \n",
       "59   ENST00000300087.2:c.331G>A   ENSP00000300087.2:p.Gly111Arg   \n",
       "60   ENST00000256797.4:c.836G>T   ENSP00000256797.4:p.Gly279Val   \n",
       "61   ENST00000269305.4:c.535C>T   ENSP00000269305.4:p.His179Tyr   \n",
       "62  ENST00000321990.4:c.1849G>A   ENSP00000313171.4:p.Val617Ile   \n",
       "63  ENST00000269228.5:c.2314T>C   ENSP00000269228.4:p.Phe772Leu   \n",
       "64  ENST00000358821.3:c.1436C>T   ENSP00000351682.3:p.Ser479Leu   \n",
       "65  ENST00000299687.5:c.4132G>A  ENSP00000299687.4:p.Gly1378Ser   \n",
       "66  ENST00000458235.1:c.1453C>A   ENSP00000391676.1:p.Leu485Met   \n",
       "67  ENST00000262809.4:c.1852C>A   ENSP00000262809.3:p.Gln618Lys   \n",
       "68  ENST00000355537.3:c.1552G>T   ENSP00000347730.1:p.Val518Leu   \n",
       "69   ENST00000285679.6:c.131A>T    ENSP00000285679.6:p.Asn44Ile   \n",
       "70   ENST00000270162.6:c.355G>C   ENSP00000270162.6:p.Gly119Arg   \n",
       "71  ENST00000327423.6:c.1642C>T   ENSP00000331845.6:p.Gln548Ter   \n",
       "72   ENST00000216264.8:c.709C>T   ENSP00000216264.8:p.Pro237Ser   \n",
       "73  ENST00000262794.5:c.2731G>A   ENSP00000262794.5:p.Val911Met   \n",
       "74  ENST00000413817.3:c.1318C>A   ENSP00000391524.2:p.Pro440Thr   \n",
       "75   ENST00000333995.3:c.448C>T   ENSP00000334430.3:p.Gln150Ter   \n",
       "76  ENST00000276185.4:c.3626A>T  ENSP00000276185.4:p.Gln1209Leu   \n",
       "77   ENST00000295746.8:c.265A>G    ENSP00000295746.7:p.Thr89Ala   \n",
       "78  ENST00000263967.3:c.1633G>A   ENSP00000263967.3:p.Glu545Lys   \n",
       "79   ENST00000389074.5:c.118G>T    ENSP00000373726.5:p.Ala40Ser   \n",
       "80    ENST00000322776.6:c.74C>T    ENSP00000322450.6:p.Thr25Ile   \n",
       "81  ENST00000389301.3:c.1046C>T   ENSP00000373952.3:p.Ala349Val   \n",
       "82   ENST00000331134.6:c.703G>T   ENSP00000331487.5:p.Asp235Tyr   \n",
       "83   ENST00000217169.3:c.178C>T    ENSP00000217169.3:p.Arg60Trp   \n",
       "84  ENST00000342995.2:c.1139C>T   ENSP00000342680.2:p.Ala380Val   \n",
       "\n",
       "                                     CLIN_SIG                  PolyPhen  \\\n",
       "0                                         NaN  probably_damaging(0.991)   \n",
       "1                                         NaN  probably_damaging(0.991)   \n",
       "2                                         NaN  probably_damaging(0.914)   \n",
       "3                                         NaN                 benign(0)   \n",
       "4                                         NaN                 benign(0)   \n",
       "5                                         NaN                 benign(0)   \n",
       "6                                         NaN             benign(0.367)   \n",
       "7                                         NaN  probably_damaging(0.999)   \n",
       "8                                         NaN  probably_damaging(0.942)   \n",
       "9                                         NaN  probably_damaging(0.956)   \n",
       "10                                        NaN  probably_damaging(0.998)   \n",
       "11                                        NaN  probably_damaging(0.991)   \n",
       "12                                        NaN  possibly_damaging(0.818)   \n",
       "13                                        NaN             benign(0.003)   \n",
       "14                                        NaN             benign(0.028)   \n",
       "15                                     benign              benign(0.02)   \n",
       "16                                        NaN             benign(0.003)   \n",
       "17                                        NaN      probably_damaging(1)   \n",
       "18                                        NaN                       NaN   \n",
       "19                                        NaN  probably_damaging(0.999)   \n",
       "20                                        NaN             benign(0.005)   \n",
       "21                                        NaN                       NaN   \n",
       "22                                        NaN                 benign(0)   \n",
       "23                                        NaN                 benign(0)   \n",
       "24                                        NaN             benign(0.103)   \n",
       "25                                        NaN             benign(0.255)   \n",
       "26                                        NaN                 benign(0)   \n",
       "27                                        NaN             benign(0.034)   \n",
       "28                                        NaN  probably_damaging(0.998)   \n",
       "29                                        NaN  possibly_damaging(0.583)   \n",
       "30                                        NaN  probably_damaging(0.982)   \n",
       "31                                        NaN             benign(0.077)   \n",
       "32                                        NaN  probably_damaging(0.971)   \n",
       "33                                        NaN             benign(0.001)   \n",
       "34                                        NaN             benign(0.043)   \n",
       "35                                        NaN  possibly_damaging(0.902)   \n",
       "36                                        NaN  possibly_damaging(0.851)   \n",
       "37                                        NaN             benign(0.018)   \n",
       "38                                        NaN  possibly_damaging(0.876)   \n",
       "39                                        NaN                 benign(0)   \n",
       "40                                        NaN                       NaN   \n",
       "41                                        NaN  possibly_damaging(0.589)   \n",
       "42                                        NaN  probably_damaging(0.998)   \n",
       "43                                        NaN             benign(0.046)   \n",
       "44                                        NaN  possibly_damaging(0.708)   \n",
       "45                                        NaN             benign(0.001)   \n",
       "46                                        NaN  possibly_damaging(0.869)   \n",
       "47               pathogenic&likely_pathogenic              benign(0.44)   \n",
       "48                                        NaN             benign(0.147)   \n",
       "49                                        NaN             benign(0.003)   \n",
       "50                                        NaN  probably_damaging(0.991)   \n",
       "51                                        NaN  probably_damaging(0.962)   \n",
       "52                                        NaN      probably_damaging(1)   \n",
       "53                                        NaN  probably_damaging(0.964)   \n",
       "54                     uncertain_significance  possibly_damaging(0.615)   \n",
       "55                                        NaN             benign(0.003)   \n",
       "56                                        NaN             benign(0.001)   \n",
       "57                                        NaN             benign(0.034)   \n",
       "58                                        NaN                       NaN   \n",
       "59                                        NaN  probably_damaging(0.976)   \n",
       "60                                        NaN  probably_damaging(0.954)   \n",
       "61               pathogenic&likely_pathogenic  probably_damaging(0.991)   \n",
       "62                                        NaN             benign(0.037)   \n",
       "63                                        NaN  probably_damaging(0.991)   \n",
       "64                                        NaN  probably_damaging(0.993)   \n",
       "65                                        NaN              benign(0.01)   \n",
       "66                                        NaN  probably_damaging(0.931)   \n",
       "67                                        NaN             benign(0.104)   \n",
       "68                                        NaN             benign(0.006)   \n",
       "69                                        NaN  possibly_damaging(0.548)   \n",
       "70                                        NaN  probably_damaging(0.981)   \n",
       "71                                        NaN                       NaN   \n",
       "72                                        NaN  probably_damaging(0.999)   \n",
       "73                                        NaN  probably_damaging(0.987)   \n",
       "74                                        NaN  probably_damaging(0.997)   \n",
       "75                                        NaN                       NaN   \n",
       "76                                        NaN                 benign(0)   \n",
       "77                                        NaN             benign(0.108)   \n",
       "78  not_provided&pathogenic&likely_pathogenic  probably_damaging(0.909)   \n",
       "79                                        NaN             benign(0.136)   \n",
       "80                                        NaN             benign(0.021)   \n",
       "81                     uncertain_significance             benign(0.138)   \n",
       "82                                        NaN  probably_damaging(0.949)   \n",
       "83                                        NaN  possibly_damaging(0.895)   \n",
       "84                                        NaN             benign(0.013)   \n",
       "\n",
       "                                SIFT  \n",
       "0                     deleterious(0)  \n",
       "1     tolerated_low_confidence(0.07)  \n",
       "2                    tolerated(0.08)  \n",
       "3                    tolerated(0.75)  \n",
       "4                     tolerated(0.4)  \n",
       "5                     deleterious(0)  \n",
       "6                    tolerated(0.11)  \n",
       "7                     deleterious(0)  \n",
       "8     tolerated_low_confidence(0.09)  \n",
       "9                    tolerated(0.25)  \n",
       "10                    deleterious(0)  \n",
       "11                    deleterious(0)  \n",
       "12                 deleterious(0.02)  \n",
       "13                   tolerated(0.46)  \n",
       "14                   tolerated(0.44)  \n",
       "15                 deleterious(0.01)  \n",
       "16                      tolerated(1)  \n",
       "17                   tolerated(0.06)  \n",
       "18                               NaN  \n",
       "19                    deleterious(0)  \n",
       "20                   tolerated(0.13)  \n",
       "21                               NaN  \n",
       "22                   tolerated(0.96)  \n",
       "23                   tolerated(0.78)  \n",
       "24                   tolerated(0.12)  \n",
       "25                 deleterious(0.04)  \n",
       "26                   tolerated(0.13)  \n",
       "27                   tolerated(0.17)  \n",
       "28                    deleterious(0)  \n",
       "29                 deleterious(0.03)  \n",
       "30                    deleterious(0)  \n",
       "31                 deleterious(0.03)  \n",
       "32                    deleterious(0)  \n",
       "33                   tolerated(0.65)  \n",
       "34                   tolerated(0.44)  \n",
       "35                    deleterious(0)  \n",
       "36                      tolerated(1)  \n",
       "37                    tolerated(0.1)  \n",
       "38                 deleterious(0.02)  \n",
       "39                    tolerated(0.3)  \n",
       "40                               NaN  \n",
       "41                    deleterious(0)  \n",
       "42                    deleterious(0)  \n",
       "43                    tolerated(0.1)  \n",
       "44                   tolerated(0.37)  \n",
       "45                   tolerated(0.87)  \n",
       "46                 deleterious(0.04)  \n",
       "47                 deleterious(0.04)  \n",
       "48                    deleterious(0)  \n",
       "49                   tolerated(0.38)  \n",
       "50                    deleterious(0)  \n",
       "51                   tolerated(0.07)  \n",
       "52                 deleterious(0.01)  \n",
       "53                    deleterious(0)  \n",
       "54                   tolerated(0.17)  \n",
       "55                   tolerated(0.09)  \n",
       "56                   tolerated(0.12)  \n",
       "57                    deleterious(0)  \n",
       "58                               NaN  \n",
       "59                    deleterious(0)  \n",
       "60                    deleterious(0)  \n",
       "61                    deleterious(0)  \n",
       "62                   tolerated(0.15)  \n",
       "63                    deleterious(0)  \n",
       "64                    deleterious(0)  \n",
       "65                   tolerated(0.05)  \n",
       "66                 deleterious(0.02)  \n",
       "67                   tolerated(0.13)  \n",
       "68                   tolerated(0.49)  \n",
       "69                    deleterious(0)  \n",
       "70                 deleterious(0.01)  \n",
       "71                               NaN  \n",
       "72                    deleterious(0)  \n",
       "73                    deleterious(0)  \n",
       "74                    deleterious(0)  \n",
       "75                               NaN  \n",
       "76    tolerated_low_confidence(0.11)  \n",
       "77                   tolerated(0.14)  \n",
       "78                 deleterious(0.02)  \n",
       "79                   tolerated(0.83)  \n",
       "80  deleterious_low_confidence(0.04)  \n",
       "81                   tolerated(0.14)  \n",
       "82                    deleterious(0)  \n",
       "83                    deleterious(0)  \n",
       "84                      tolerated(1)  "
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf_1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [],
   "source": [
    "vcf_2=pd.read_table('APD006_T2.ALL.consensus.tab', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames</th>\n",
       "      <th>start</th>\n",
       "      <th>end</th>\n",
       "      <th>width</th>\n",
       "      <th>strand</th>\n",
       "      <th>AD</th>\n",
       "      <th>AD.1</th>\n",
       "      <th>AF</th>\n",
       "      <th>Consequence</th>\n",
       "      <th>IMPACT</th>\n",
       "      <th>SYMBOL</th>\n",
       "      <th>HGVSc</th>\n",
       "      <th>HGVSp</th>\n",
       "      <th>CLIN_SIG</th>\n",
       "      <th>PolyPhen</th>\n",
       "      <th>SIFT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>1</td>\n",
       "      <td>1267294</td>\n",
       "      <td>1267294</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>8</td>\n",
       "      <td>0.182</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TAS1R3</td>\n",
       "      <td>ENST00000339381.5:c.468C&gt;A</td>\n",
       "      <td>ENSP00000344411.5:p.Phe156Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.007)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>16161836</td>\n",
       "      <td>16161836</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>3</td>\n",
       "      <td>0.064</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-169K16.9</td>\n",
       "      <td>ENST00000317122.1:n.1606A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>37324811</td>\n",
       "      <td>37324811</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>4</td>\n",
       "      <td>0.105</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GRIK3</td>\n",
       "      <td>ENST00000373091.3:c.1002C&gt;T</td>\n",
       "      <td>ENSP00000362183.3:p.Ser334%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>1</td>\n",
       "      <td>89654458</td>\n",
       "      <td>89654458</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>4</td>\n",
       "      <td>0.089</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GBP4</td>\n",
       "      <td>ENST00000355754.6:c.1217A&gt;G</td>\n",
       "      <td>ENSP00000359490.5:p.Lys406Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.208)</td>\n",
       "      <td>tolerated(0.37)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>1</td>\n",
       "      <td>94488912</td>\n",
       "      <td>94488912</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ABCA4</td>\n",
       "      <td>ENST00000370225.3:c.4667+30T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>5</td>\n",
       "      <td>1</td>\n",
       "      <td>149859333</td>\n",
       "      <td>149859333</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>3</td>\n",
       "      <td>0.091</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>HIST2H2AB</td>\n",
       "      <td>ENST00000331128.3:c.134G&gt;T</td>\n",
       "      <td>ENSP00000332790.3:p.Gly45Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.992)</td>\n",
       "      <td>deleterious_low_confidence(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>6</td>\n",
       "      <td>1</td>\n",
       "      <td>154841903</td>\n",
       "      <td>154841903</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>3</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KCNN3</td>\n",
       "      <td>ENST00000271915.4:c.538T&gt;G</td>\n",
       "      <td>ENSP00000271915.3:p.Tyr180Asp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.997)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>7</td>\n",
       "      <td>1</td>\n",
       "      <td>158623085</td>\n",
       "      <td>158623085</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>10</td>\n",
       "      <td>0.185</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPTA1</td>\n",
       "      <td>ENST00000368147.4:c.3167G&gt;A</td>\n",
       "      <td>ENSP00000357129.4:p.Arg1056His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.776)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>8</td>\n",
       "      <td>1</td>\n",
       "      <td>158632710</td>\n",
       "      <td>158632710</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>9</td>\n",
       "      <td>0.164</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SPTA1</td>\n",
       "      <td>ENST00000368147.4:c.2246T&gt;C</td>\n",
       "      <td>ENSP00000357129.4:p.Leu749Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.811)</td>\n",
       "      <td>tolerated(0.28)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>9</td>\n",
       "      <td>1</td>\n",
       "      <td>169781280</td>\n",
       "      <td>169781280</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>12</td>\n",
       "      <td>0.190</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C1orf112</td>\n",
       "      <td>ENST00000286031.6:c.702+4210A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>10</td>\n",
       "      <td>1</td>\n",
       "      <td>169890543</td>\n",
       "      <td>169890543</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>3</td>\n",
       "      <td>0.068</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KIFAP3</td>\n",
       "      <td>ENST00000361580.2:c.*274G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>11</td>\n",
       "      <td>1</td>\n",
       "      <td>177902482</td>\n",
       "      <td>177902482</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>93</td>\n",
       "      <td>5</td>\n",
       "      <td>0.051</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SEC16B</td>\n",
       "      <td>ENST00000308284.6:c.2725-35C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>12</td>\n",
       "      <td>1</td>\n",
       "      <td>208200122</td>\n",
       "      <td>208200122</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>16</td>\n",
       "      <td>6</td>\n",
       "      <td>0.273</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PLXNA2</td>\n",
       "      <td>ENST00000367033.3:c.*466T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>13</td>\n",
       "      <td>1</td>\n",
       "      <td>248004756</td>\n",
       "      <td>248004756</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>5</td>\n",
       "      <td>0.086</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR11L1</td>\n",
       "      <td>ENST00000355784.2:c.443C&gt;T</td>\n",
       "      <td>ENSP00000348033.2:p.Ser148Phe</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.992)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>14</td>\n",
       "      <td>2</td>\n",
       "      <td>27457500</td>\n",
       "      <td>27457500</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>19</td>\n",
       "      <td>0.302</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CAD</td>\n",
       "      <td>ENST00000264705.4:c.3733G&gt;C</td>\n",
       "      <td>ENSP00000264705.3:p.Glu1245Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.146)</td>\n",
       "      <td>tolerated(0.56)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>15</td>\n",
       "      <td>2</td>\n",
       "      <td>32692725</td>\n",
       "      <td>32692725</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>4</td>\n",
       "      <td>0.060</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>BIRC6</td>\n",
       "      <td>ENST00000421745.2:c.5489G&gt;T</td>\n",
       "      <td>ENSP00000393596.2:p.Arg1830Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.982)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>16</td>\n",
       "      <td>2</td>\n",
       "      <td>62451310</td>\n",
       "      <td>62451310</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>10</td>\n",
       "      <td>0.263</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>B3GNT2</td>\n",
       "      <td>ENST00000301998.4:c.*761G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>17</td>\n",
       "      <td>2</td>\n",
       "      <td>64862518</td>\n",
       "      <td>64862518</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>8</td>\n",
       "      <td>0.258</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SERTAD2</td>\n",
       "      <td>ENST00000313349.3:c.*543T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>18</td>\n",
       "      <td>2</td>\n",
       "      <td>71062648</td>\n",
       "      <td>71062648</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>67</td>\n",
       "      <td>3</td>\n",
       "      <td>0.043</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CD207</td>\n",
       "      <td>ENST00000410009.3:c.164C&gt;T</td>\n",
       "      <td>ENSP00000386378.3:p.Ala55Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.115)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>19</td>\n",
       "      <td>2</td>\n",
       "      <td>74185876</td>\n",
       "      <td>74185876</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>64</td>\n",
       "      <td>3</td>\n",
       "      <td>0.045</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DGUOK</td>\n",
       "      <td>ENST00000264093.4:c.*13A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>20</td>\n",
       "      <td>2</td>\n",
       "      <td>87085411</td>\n",
       "      <td>87085411</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>15</td>\n",
       "      <td>0.288</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CD8B</td>\n",
       "      <td>ENST00000331469.2:c.172C&gt;T</td>\n",
       "      <td>ENSP00000331172.2:p.Arg58Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.035)</td>\n",
       "      <td>tolerated(0.31)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>21</td>\n",
       "      <td>2</td>\n",
       "      <td>137814138</td>\n",
       "      <td>137814138</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>91</td>\n",
       "      <td>4</td>\n",
       "      <td>0.042</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>THSD7B</td>\n",
       "      <td>ENST00000272643.3:c.288T&gt;C</td>\n",
       "      <td>ENSP00000272643.3:p.Val96%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>22</td>\n",
       "      <td>2</td>\n",
       "      <td>137852449</td>\n",
       "      <td>137852449</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>9</td>\n",
       "      <td>0.138</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>THSD7B</td>\n",
       "      <td>ENST00000272643.3:c.957C&gt;T</td>\n",
       "      <td>ENSP00000272643.3:p.Cys319%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>23</td>\n",
       "      <td>2</td>\n",
       "      <td>165996101</td>\n",
       "      <td>165996101</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>13</td>\n",
       "      <td>0.178</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>SCN3A</td>\n",
       "      <td>ENST00000283254.7:c.2037G&gt;A</td>\n",
       "      <td>ENSP00000283254.7:p.Thr679%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>24</td>\n",
       "      <td>2</td>\n",
       "      <td>167266247</td>\n",
       "      <td>167266247</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>7</td>\n",
       "      <td>0.184</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SCN7A</td>\n",
       "      <td>ENST00000409855.1:c.3910G&gt;A</td>\n",
       "      <td>ENSP00000386796.1:p.Ala1304Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.461)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>25</td>\n",
       "      <td>2</td>\n",
       "      <td>168921838</td>\n",
       "      <td>168921838</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>STK39</td>\n",
       "      <td>ENST00000355999.4:c.1296C&gt;T</td>\n",
       "      <td>ENSP00000348278.4:p.His432%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>26</td>\n",
       "      <td>2</td>\n",
       "      <td>170402703</td>\n",
       "      <td>170402703</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>5</td>\n",
       "      <td>0.147</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FASTKD1</td>\n",
       "      <td>ENST00000453153.2:c.1701+25A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>27</td>\n",
       "      <td>2</td>\n",
       "      <td>183623602</td>\n",
       "      <td>183623602</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DNAJC10</td>\n",
       "      <td>ENST00000264065.7:c.1982G&gt;C</td>\n",
       "      <td>ENSP00000264065.6:p.Arg661Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.587)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>28</td>\n",
       "      <td>2</td>\n",
       "      <td>198570729</td>\n",
       "      <td>198570729</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>97</td>\n",
       "      <td>8</td>\n",
       "      <td>0.076</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MARS2</td>\n",
       "      <td>ENST00000282276.6:c.600G&gt;A</td>\n",
       "      <td>ENSP00000282276.6:p.Gly200%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>29</td>\n",
       "      <td>2</td>\n",
       "      <td>201434487</td>\n",
       "      <td>201434487</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>11</td>\n",
       "      <td>0.275</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SGOL2</td>\n",
       "      <td>ENST00000357799.4:c.575G&gt;T</td>\n",
       "      <td>ENSP00000350447.4:p.Gly192Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>30</td>\n",
       "      <td>2</td>\n",
       "      <td>218762536</td>\n",
       "      <td>218762536</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>11</td>\n",
       "      <td>0.169</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TNS1</td>\n",
       "      <td>ENST00000171887.4:c.153T&gt;C</td>\n",
       "      <td>ENSP00000171887.4:p.His51%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>31</td>\n",
       "      <td>2</td>\n",
       "      <td>226378136</td>\n",
       "      <td>226378136</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>14</td>\n",
       "      <td>0.226</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NYAP2</td>\n",
       "      <td>ENST00000272907.6:c.271C&gt;T</td>\n",
       "      <td>ENSP00000272907.6:p.His91Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.946)</td>\n",
       "      <td>deleterious(0.03)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>32</td>\n",
       "      <td>3</td>\n",
       "      <td>4810190</td>\n",
       "      <td>4810190</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>3</td>\n",
       "      <td>0.049</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ITPR1</td>\n",
       "      <td>ENST00000302640.8:c.5681-5T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>39136589</td>\n",
       "      <td>39136589</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>92</td>\n",
       "      <td>4</td>\n",
       "      <td>0.042</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>WDR48</td>\n",
       "      <td>ENST00000302313.5:c.*355G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>34</td>\n",
       "      <td>3</td>\n",
       "      <td>44438259</td>\n",
       "      <td>44438259</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>8</td>\n",
       "      <td>0.129</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TCAIM</td>\n",
       "      <td>ENST00000342649.4:c.818G&gt;T</td>\n",
       "      <td>ENSP00000341539.4:p.Arg273Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.034)</td>\n",
       "      <td>tolerated(0.48)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>35</td>\n",
       "      <td>3</td>\n",
       "      <td>57875754</td>\n",
       "      <td>57875754</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>35</td>\n",
       "      <td>4</td>\n",
       "      <td>0.103</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SLMAP</td>\n",
       "      <td>ENST00000295951.3:c.1199-14A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>60898389</td>\n",
       "      <td>60898389</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>0.083</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FHIT</td>\n",
       "      <td>ENST00000468189.1:c.-110-90684T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>37</td>\n",
       "      <td>3</td>\n",
       "      <td>84339405</td>\n",
       "      <td>84339405</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>65</td>\n",
       "      <td>6</td>\n",
       "      <td>0.085</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AC108696.1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>134967276</td>\n",
       "      <td>134967276</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>74</td>\n",
       "      <td>25</td>\n",
       "      <td>0.253</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHB1</td>\n",
       "      <td>ENST00000398015.3:c.2615C&gt;T</td>\n",
       "      <td>ENSP00000381097.3:p.Ala872Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.137)</td>\n",
       "      <td>deleterious(0.05)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>39</td>\n",
       "      <td>3</td>\n",
       "      <td>140401955</td>\n",
       "      <td>140401955</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>18</td>\n",
       "      <td>0.310</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TRIM42</td>\n",
       "      <td>ENST00000286349.3:c.993C&gt;T</td>\n",
       "      <td>ENSP00000286349.3:p.Ser331%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>40</td>\n",
       "      <td>3</td>\n",
       "      <td>151012419</td>\n",
       "      <td>151012419</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>7</td>\n",
       "      <td>0.189</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GPR87</td>\n",
       "      <td>ENST00000260843.4:c.615G&gt;A</td>\n",
       "      <td>ENSP00000260843.4:p.Thr205%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>41</td>\n",
       "      <td>3</td>\n",
       "      <td>159703343</td>\n",
       "      <td>159703343</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>7</td>\n",
       "      <td>0.111</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IL12A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>42</td>\n",
       "      <td>3</td>\n",
       "      <td>178916930</td>\n",
       "      <td>178916930</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>4</td>\n",
       "      <td>0.174</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>ENST00000263967.3:c.317G&gt;T</td>\n",
       "      <td>ENSP00000263967.3:p.Gly106Val</td>\n",
       "      <td>uncertain_significance&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>1737062</td>\n",
       "      <td>1737062</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>71</td>\n",
       "      <td>4</td>\n",
       "      <td>0.053</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TACC3</td>\n",
       "      <td>ENST00000313288.4:c.1644+10A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>1737109</td>\n",
       "      <td>1737109</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>5</td>\n",
       "      <td>0.098</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TACC3</td>\n",
       "      <td>ENST00000313288.4:c.1644+57G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>45</td>\n",
       "      <td>4</td>\n",
       "      <td>73933754</td>\n",
       "      <td>73933754</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>3</td>\n",
       "      <td>0.125</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>COX18</td>\n",
       "      <td>ENST00000295890.4:c.426A&gt;G</td>\n",
       "      <td>ENSP00000295890.4:p.Arg142%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>96123981</td>\n",
       "      <td>96123981</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>UNC5C</td>\n",
       "      <td>ENST00000453304.1:c.2037G&gt;A</td>\n",
       "      <td>ENSP00000406022.1:p.Ala679%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>123745305</td>\n",
       "      <td>123745305</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>107</td>\n",
       "      <td>12</td>\n",
       "      <td>0.101</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FGF2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>48</td>\n",
       "      <td>4</td>\n",
       "      <td>151772031</td>\n",
       "      <td>151772031</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>6</td>\n",
       "      <td>0.214</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>LRBA</td>\n",
       "      <td>ENST00000357115.3:c.3849A&gt;C</td>\n",
       "      <td>ENSP00000349629.3:p.Pro1283%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>156764877</td>\n",
       "      <td>156764877</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>4</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ASIC5</td>\n",
       "      <td>ENST00000537611.2:c.817T&gt;C</td>\n",
       "      <td>ENSP00000442477.2:p.Ser273Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.062)</td>\n",
       "      <td>tolerated(0.08)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>50</td>\n",
       "      <td>4</td>\n",
       "      <td>158074147</td>\n",
       "      <td>158074147</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GLRB</td>\n",
       "      <td>ENST00000264428.4:c.1182T&gt;C</td>\n",
       "      <td>ENSP00000264428.4:p.His394%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>51</td>\n",
       "      <td>5</td>\n",
       "      <td>38481688</td>\n",
       "      <td>38481688</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>13</td>\n",
       "      <td>0.217</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LIFR</td>\n",
       "      <td>ENST00000263409.4:c.*9C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>52</td>\n",
       "      <td>5</td>\n",
       "      <td>50084445</td>\n",
       "      <td>50084445</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>11</td>\n",
       "      <td>0.234</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PARP8</td>\n",
       "      <td>ENST00000281631.5:c.737+22A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>53</td>\n",
       "      <td>5</td>\n",
       "      <td>99921931</td>\n",
       "      <td>99921931</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>5</td>\n",
       "      <td>0.185</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FAM174A</td>\n",
       "      <td>ENST00000312637.4:c.*43A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>54</td>\n",
       "      <td>5</td>\n",
       "      <td>126769146</td>\n",
       "      <td>126769146</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>3</td>\n",
       "      <td>0.053</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MEGF10</td>\n",
       "      <td>ENST00000274473.6:c.1785T&gt;C</td>\n",
       "      <td>ENSP00000274473.6:p.Pro595%3D</td>\n",
       "      <td>likely_benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>55</td>\n",
       "      <td>5</td>\n",
       "      <td>129520330</td>\n",
       "      <td>129520330</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>9</td>\n",
       "      <td>0.138</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CHSY3</td>\n",
       "      <td>ENST00000305031.4:c.1495G&gt;A</td>\n",
       "      <td>ENSP00000302629.4:p.Val499Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.042)</td>\n",
       "      <td>tolerated(0.46)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>56</td>\n",
       "      <td>5</td>\n",
       "      <td>132005592</td>\n",
       "      <td>132005592</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>8</td>\n",
       "      <td>0.250</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IL4</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>57</td>\n",
       "      <td>5</td>\n",
       "      <td>140536977</td>\n",
       "      <td>140536977</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>12</td>\n",
       "      <td>8</td>\n",
       "      <td>0.400</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PCDHB6</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>58</td>\n",
       "      <td>5</td>\n",
       "      <td>141027663</td>\n",
       "      <td>141027663</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>17</td>\n",
       "      <td>3</td>\n",
       "      <td>0.150</td>\n",
       "      <td>splice_region_variant&amp;synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>FCHSD1</td>\n",
       "      <td>ENST00000435817.2:c.577C&gt;T</td>\n",
       "      <td>ENSP00000399259.2:p.Leu193%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>59</td>\n",
       "      <td>5</td>\n",
       "      <td>154296729</td>\n",
       "      <td>154296729</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>41</td>\n",
       "      <td>20</td>\n",
       "      <td>0.328</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GEMIN5</td>\n",
       "      <td>ENST00000285873.7:c.1684A&gt;G</td>\n",
       "      <td>ENSP00000285873.6:p.Ile562Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.015)</td>\n",
       "      <td>tolerated(0.86)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>60</td>\n",
       "      <td>5</td>\n",
       "      <td>177612983</td>\n",
       "      <td>177612983</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>79</td>\n",
       "      <td>10</td>\n",
       "      <td>0.112</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GMCL1P1</td>\n",
       "      <td>ENST00000463439.1:n.1318C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>61</td>\n",
       "      <td>6</td>\n",
       "      <td>31431757</td>\n",
       "      <td>31431757</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>4</td>\n",
       "      <td>0.067</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>HCP5</td>\n",
       "      <td>ENST00000414046.2:n.829A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>62</td>\n",
       "      <td>6</td>\n",
       "      <td>44311563</td>\n",
       "      <td>44311563</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>108</td>\n",
       "      <td>21</td>\n",
       "      <td>0.163</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SPATS1</td>\n",
       "      <td>ENST00000288390.2:c.139+592G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>63</td>\n",
       "      <td>6</td>\n",
       "      <td>51491959</td>\n",
       "      <td>51491959</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>14</td>\n",
       "      <td>0.259</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PKHD1</td>\n",
       "      <td>ENST00000371117.3:c.11666-45C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>64</td>\n",
       "      <td>6</td>\n",
       "      <td>74001806</td>\n",
       "      <td>74001806</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>6</td>\n",
       "      <td>0.167</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KHDC1</td>\n",
       "      <td>ENST00000370384.3:c.164-43T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>65</td>\n",
       "      <td>6</td>\n",
       "      <td>118801734</td>\n",
       "      <td>118801734</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>18</td>\n",
       "      <td>11</td>\n",
       "      <td>0.379</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CEP85L</td>\n",
       "      <td>ENST00000368488.5:c.1755-58C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>66</td>\n",
       "      <td>6</td>\n",
       "      <td>130497280</td>\n",
       "      <td>130497280</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SAMD3</td>\n",
       "      <td>ENST00000368134.2:c.655-127T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>67</td>\n",
       "      <td>6</td>\n",
       "      <td>136403705</td>\n",
       "      <td>136403705</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>12</td>\n",
       "      <td>0.235</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PDE7B</td>\n",
       "      <td>ENST00000308191.6:c.83-26164C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>68</td>\n",
       "      <td>6</td>\n",
       "      <td>139598024</td>\n",
       "      <td>139598024</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>4</td>\n",
       "      <td>0.154</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TXLNB</td>\n",
       "      <td>ENST00000358430.3:c.459C&gt;T</td>\n",
       "      <td>ENSP00000351206.3:p.Asn153%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>69</td>\n",
       "      <td>6</td>\n",
       "      <td>160831788</td>\n",
       "      <td>160831788</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>15</td>\n",
       "      <td>0.197</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>SLC22A3</td>\n",
       "      <td>ENST00000275300.2:c.885G&gt;C</td>\n",
       "      <td>ENSP00000275300.2:p.Leu295%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>70</td>\n",
       "      <td>7</td>\n",
       "      <td>12399204</td>\n",
       "      <td>12399204</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>3</td>\n",
       "      <td>0.047</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>VWDE</td>\n",
       "      <td>ENST00000275358.3:c.3274A&gt;T</td>\n",
       "      <td>ENSP00000275358.3:p.Asn1092Tyr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.005)</td>\n",
       "      <td>tolerated(0.11)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>71</td>\n",
       "      <td>7</td>\n",
       "      <td>31746856</td>\n",
       "      <td>31746856</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>18</td>\n",
       "      <td>0.250</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PPP1R17</td>\n",
       "      <td>ENST00000342032.3:c.427G&gt;A</td>\n",
       "      <td>ENSP00000340125.3:p.Val143Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated_low_confidence(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>72</td>\n",
       "      <td>7</td>\n",
       "      <td>36763672</td>\n",
       "      <td>36763672</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>3</td>\n",
       "      <td>0.065</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>AOAH</td>\n",
       "      <td>ENST00000258749.5:c.82G&gt;A</td>\n",
       "      <td>ENSP00000258749.5:p.Asp28Asn</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.24)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>73</td>\n",
       "      <td>7</td>\n",
       "      <td>37272689</td>\n",
       "      <td>37272689</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>12</td>\n",
       "      <td>0.222</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ELMO1</td>\n",
       "      <td>ENST00000310758.4:c.549+11C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>74</td>\n",
       "      <td>7</td>\n",
       "      <td>43519218</td>\n",
       "      <td>43519218</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>74</td>\n",
       "      <td>19</td>\n",
       "      <td>0.204</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>HECW1</td>\n",
       "      <td>ENST00000395891.2:c.3109G&gt;A</td>\n",
       "      <td>ENSP00000379228.1:p.Val1037Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.744)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>75</td>\n",
       "      <td>7</td>\n",
       "      <td>64439575</td>\n",
       "      <td>64439575</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>11</td>\n",
       "      <td>0.204</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF117</td>\n",
       "      <td>ENST00000282869.6:c.374C&gt;T</td>\n",
       "      <td>ENSP00000282869.5:p.Thr125Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.202)</td>\n",
       "      <td>tolerated(0.13)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>76</td>\n",
       "      <td>7</td>\n",
       "      <td>72756743</td>\n",
       "      <td>72756743</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>3</td>\n",
       "      <td>0.055</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FKBP6</td>\n",
       "      <td>ENST00000252037.4:c.894-64G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>77</td>\n",
       "      <td>7</td>\n",
       "      <td>87083892</td>\n",
       "      <td>87083892</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>5</td>\n",
       "      <td>0.111</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ABCB4</td>\n",
       "      <td>ENST00000265723.4:c.303G&gt;A</td>\n",
       "      <td>ENSP00000265723.4:p.Ser101%3D</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>78</td>\n",
       "      <td>7</td>\n",
       "      <td>136978168</td>\n",
       "      <td>136978168</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>93</td>\n",
       "      <td>6</td>\n",
       "      <td>0.061</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PTN</td>\n",
       "      <td>ENST00000348225.2:c.-1-38447A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>79</td>\n",
       "      <td>7</td>\n",
       "      <td>137015603</td>\n",
       "      <td>137015603</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>3</td>\n",
       "      <td>0.054</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PTN</td>\n",
       "      <td>ENST00000348225.2:c.-2+12582T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>80</td>\n",
       "      <td>7</td>\n",
       "      <td>137092658</td>\n",
       "      <td>137092658</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>80</td>\n",
       "      <td>18</td>\n",
       "      <td>0.184</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>DGKI</td>\n",
       "      <td>ENST00000288490.5:c.2907C&gt;T</td>\n",
       "      <td>ENSP00000288490.4:p.Thr969%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>81</td>\n",
       "      <td>7</td>\n",
       "      <td>150164063</td>\n",
       "      <td>150164063</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>76</td>\n",
       "      <td>20</td>\n",
       "      <td>0.208</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>GIMAP8</td>\n",
       "      <td>ENST00000307271.3:c.277A&gt;T</td>\n",
       "      <td>ENSP00000305107.3:p.Ser93Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.979)</td>\n",
       "      <td>tolerated(0.06)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>82</td>\n",
       "      <td>8</td>\n",
       "      <td>23225607</td>\n",
       "      <td>23225607</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>66</td>\n",
       "      <td>4</td>\n",
       "      <td>0.057</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>LOXL2</td>\n",
       "      <td>ENST00000389131.3:c.258C&gt;T</td>\n",
       "      <td>ENSP00000373783.3:p.Asp86%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>83</td>\n",
       "      <td>8</td>\n",
       "      <td>69026619</td>\n",
       "      <td>69026619</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>18</td>\n",
       "      <td>7</td>\n",
       "      <td>0.280</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PREX2</td>\n",
       "      <td>ENST00000288368.4:c.3147-1369G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>84</td>\n",
       "      <td>8</td>\n",
       "      <td>87671840</td>\n",
       "      <td>87671840</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>77</td>\n",
       "      <td>19</td>\n",
       "      <td>0.198</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CNGB3</td>\n",
       "      <td>ENST00000320005.5:c.853-5550G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>85</td>\n",
       "      <td>8</td>\n",
       "      <td>97514016</td>\n",
       "      <td>97514016</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>3</td>\n",
       "      <td>0.115</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SDC2</td>\n",
       "      <td>ENST00000302190.4:c.60+7457T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>86</td>\n",
       "      <td>8</td>\n",
       "      <td>109455863</td>\n",
       "      <td>109455863</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>EMC2</td>\n",
       "      <td>ENST00000220853.3:c.-25C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>87</td>\n",
       "      <td>9</td>\n",
       "      <td>26923301</td>\n",
       "      <td>26923301</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>74</td>\n",
       "      <td>4</td>\n",
       "      <td>0.051</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PLAA</td>\n",
       "      <td>ENST00000397292.3:c.914C&gt;T</td>\n",
       "      <td>ENSP00000380460.3:p.Ala305Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>88</td>\n",
       "      <td>9</td>\n",
       "      <td>27103306</td>\n",
       "      <td>27103306</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>11</td>\n",
       "      <td>0.172</td>\n",
       "      <td>intron_variant&amp;non_coding_transcript_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-56F10.3</td>\n",
       "      <td>ENST00000522960.1:n.447+232A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>89</td>\n",
       "      <td>9</td>\n",
       "      <td>73000586</td>\n",
       "      <td>73000586</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>13</td>\n",
       "      <td>0.277</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KLF9</td>\n",
       "      <td>ENST00000377126.2:c.*2106G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>90</td>\n",
       "      <td>9</td>\n",
       "      <td>110087166</td>\n",
       "      <td>110087166</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>5</td>\n",
       "      <td>0.192</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RAD23B</td>\n",
       "      <td>ENST00000358015.3:c.997C&gt;G</td>\n",
       "      <td>ENSP00000350708.3:p.Gln333Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.055)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>91</td>\n",
       "      <td>9</td>\n",
       "      <td>116731456</td>\n",
       "      <td>116731456</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>6</td>\n",
       "      <td>0.094</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ZNF618</td>\n",
       "      <td>ENST00000288466.7:c.77+16C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>92</td>\n",
       "      <td>9</td>\n",
       "      <td>130506868</td>\n",
       "      <td>130506868</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>6</td>\n",
       "      <td>0.167</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SH2D3C</td>\n",
       "      <td>ENST00000314830.8:c.1775G&gt;A</td>\n",
       "      <td>ENSP00000317817.8:p.Arg592Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(0.59)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>93</td>\n",
       "      <td>9</td>\n",
       "      <td>136907079</td>\n",
       "      <td>136907079</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>79</td>\n",
       "      <td>4</td>\n",
       "      <td>0.048</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>BRD3</td>\n",
       "      <td>ENST00000303407.7:c.1216-6C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>94</td>\n",
       "      <td>9</td>\n",
       "      <td>137705778</td>\n",
       "      <td>137705778</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>113</td>\n",
       "      <td>12</td>\n",
       "      <td>0.096</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>COL5A1</td>\n",
       "      <td>ENST00000371817.3:c.3853-51C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>95</td>\n",
       "      <td>9</td>\n",
       "      <td>137711921</td>\n",
       "      <td>137711921</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>12</td>\n",
       "      <td>0.214</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>COL5A1</td>\n",
       "      <td>ENST00000371817.3:c.4447-41G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>96</td>\n",
       "      <td>10</td>\n",
       "      <td>33506179</td>\n",
       "      <td>33506179</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>4</td>\n",
       "      <td>0.053</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NRP1</td>\n",
       "      <td>ENST00000265371.4:c.1283-3534G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>97</td>\n",
       "      <td>10</td>\n",
       "      <td>33561788</td>\n",
       "      <td>33561788</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>6</td>\n",
       "      <td>0.100</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NRP1</td>\n",
       "      <td>ENST00000265371.4:c.249-2004G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>98</td>\n",
       "      <td>10</td>\n",
       "      <td>50339754</td>\n",
       "      <td>50339754</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FAM170B</td>\n",
       "      <td>ENST00000311787.5:c.756C&gt;A</td>\n",
       "      <td>ENSP00000308292.5:p.Asp252Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.009)</td>\n",
       "      <td>tolerated(0.34)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>99</td>\n",
       "      <td>10</td>\n",
       "      <td>71390392</td>\n",
       "      <td>71390392</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>3</td>\n",
       "      <td>0.103</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C10orf35</td>\n",
       "      <td>ENST00000373279.4:c.-104+323C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>100</td>\n",
       "      <td>10</td>\n",
       "      <td>74671439</td>\n",
       "      <td>74671439</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>80</td>\n",
       "      <td>16</td>\n",
       "      <td>0.167</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>OIT3</td>\n",
       "      <td>ENST00000334011.5:c.668-36C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>101</td>\n",
       "      <td>10</td>\n",
       "      <td>75530013</td>\n",
       "      <td>75530013</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>4</td>\n",
       "      <td>0.071</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SEC24C</td>\n",
       "      <td>ENST00000339365.2:c.2863-25C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>102</td>\n",
       "      <td>10</td>\n",
       "      <td>105912432</td>\n",
       "      <td>105912432</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>15</td>\n",
       "      <td>0.192</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>WDR96</td>\n",
       "      <td>ENST00000357060.3:c.3593A&gt;T</td>\n",
       "      <td>ENSP00000349568.3:p.Gln1198Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.813)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>103</td>\n",
       "      <td>10</td>\n",
       "      <td>115958855</td>\n",
       "      <td>115958855</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>45</td>\n",
       "      <td>3</td>\n",
       "      <td>0.062</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TDRD1</td>\n",
       "      <td>ENST00000251864.2:c.385-77G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>104</td>\n",
       "      <td>10</td>\n",
       "      <td>115971806</td>\n",
       "      <td>115971806</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>7</td>\n",
       "      <td>0.241</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TDRD1</td>\n",
       "      <td>ENST00000251864.2:c.1831+11T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>105</td>\n",
       "      <td>10</td>\n",
       "      <td>118220506</td>\n",
       "      <td>118220506</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>18</td>\n",
       "      <td>0.225</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PNLIPRP3</td>\n",
       "      <td>ENST00000369230.3:c.594C&gt;T</td>\n",
       "      <td>ENSP00000358232.3:p.His198%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>106</td>\n",
       "      <td>10</td>\n",
       "      <td>124459112</td>\n",
       "      <td>124459112</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>11</td>\n",
       "      <td>0.208</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C10orf120</td>\n",
       "      <td>ENST00000329446.4:c.176+19T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>107</td>\n",
       "      <td>11</td>\n",
       "      <td>7111667</td>\n",
       "      <td>7111667</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>13</td>\n",
       "      <td>0.289</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RBMXL2</td>\n",
       "      <td>ENST00000306904.5:c.*137C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>108</td>\n",
       "      <td>11</td>\n",
       "      <td>50368338</td>\n",
       "      <td>50368338</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>4</td>\n",
       "      <td>0.077</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-574M7.2</td>\n",
       "      <td>ENST00000532521.2:n.126A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>109</td>\n",
       "      <td>11</td>\n",
       "      <td>55111650</td>\n",
       "      <td>55111650</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>16</td>\n",
       "      <td>8</td>\n",
       "      <td>0.333</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR4A16</td>\n",
       "      <td>ENST00000314721.2:c.974C&gt;T</td>\n",
       "      <td>ENSP00000325128.2:p.Thr325Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated_low_confidence(0.25)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>110</td>\n",
       "      <td>11</td>\n",
       "      <td>55999897</td>\n",
       "      <td>55999897</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>102</td>\n",
       "      <td>7</td>\n",
       "      <td>0.064</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>OR5T2</td>\n",
       "      <td>ENST00000313264.4:c.765C&gt;T</td>\n",
       "      <td>ENSP00000323688.4:p.Ile255%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>111</td>\n",
       "      <td>11</td>\n",
       "      <td>56185159</td>\n",
       "      <td>56185159</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>60</td>\n",
       "      <td>3</td>\n",
       "      <td>0.048</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR5R1</td>\n",
       "      <td>ENST00000312253.1:c.550T&gt;C</td>\n",
       "      <td>ENSP00000308595.1:p.Phe184Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>112</td>\n",
       "      <td>11</td>\n",
       "      <td>69514109</td>\n",
       "      <td>69514109</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>6</td>\n",
       "      <td>0.100</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>FGF19</td>\n",
       "      <td>ENST00000294312.3:c.572C&gt;T</td>\n",
       "      <td>ENSP00000294312.3:p.Pro191Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.784)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>113</td>\n",
       "      <td>11</td>\n",
       "      <td>74408224</td>\n",
       "      <td>74408224</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.083</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CHRDL2</td>\n",
       "      <td>ENST00000263671.5:c.1268+26G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>114</td>\n",
       "      <td>11</td>\n",
       "      <td>102815030</td>\n",
       "      <td>102815030</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>16</td>\n",
       "      <td>0.432</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MMP13</td>\n",
       "      <td>ENST00000260302.3:c.1381C&gt;T</td>\n",
       "      <td>ENSP00000260302.3:p.Arg461Cys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.996)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>115</td>\n",
       "      <td>11</td>\n",
       "      <td>124743628</td>\n",
       "      <td>124743628</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>64</td>\n",
       "      <td>4</td>\n",
       "      <td>0.059</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ROBO3</td>\n",
       "      <td>ENST00000397801.1:c.1654C&gt;A</td>\n",
       "      <td>ENSP00000380903.1:p.Pro552Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.045)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>116</td>\n",
       "      <td>12</td>\n",
       "      <td>922751</td>\n",
       "      <td>922751</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>7</td>\n",
       "      <td>0.200</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>WNK1</td>\n",
       "      <td>ENST00000315939.6:c.760-57G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>117</td>\n",
       "      <td>12</td>\n",
       "      <td>4463042</td>\n",
       "      <td>4463042</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.077</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>C12orf5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>118</td>\n",
       "      <td>12</td>\n",
       "      <td>9211480</td>\n",
       "      <td>9211480</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>12</td>\n",
       "      <td>0.235</td>\n",
       "      <td>splice_region_variant&amp;intron_variant&amp;non_codin...</td>\n",
       "      <td>LOW</td>\n",
       "      <td>LINC00612</td>\n",
       "      <td>ENST00000538094.1:n.804-8T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>119</td>\n",
       "      <td>12</td>\n",
       "      <td>9310990</td>\n",
       "      <td>9310990</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>80</td>\n",
       "      <td>5</td>\n",
       "      <td>0.059</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PZP</td>\n",
       "      <td>ENST00000261336.2:c.3294+26C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>120</td>\n",
       "      <td>12</td>\n",
       "      <td>10041301</td>\n",
       "      <td>10041301</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>16</td>\n",
       "      <td>0.267</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KLRF2</td>\n",
       "      <td>ENST00000535540.1:c.170-31T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>121</td>\n",
       "      <td>12</td>\n",
       "      <td>14650757</td>\n",
       "      <td>14650757</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>14</td>\n",
       "      <td>0.259</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ATF7IP</td>\n",
       "      <td>ENST00000261168.4:c.3563G&gt;A</td>\n",
       "      <td>ENSP00000261168.4:p.Arg1188Gln</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.742)</td>\n",
       "      <td>deleterious_low_confidence(0.02)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>122</td>\n",
       "      <td>12</td>\n",
       "      <td>25398284</td>\n",
       "      <td>25398284</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>11</td>\n",
       "      <td>0.229</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>ENST00000256078.4:c.35G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly12Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.303)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>123</td>\n",
       "      <td>12</td>\n",
       "      <td>92537555</td>\n",
       "      <td>92537555</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>14</td>\n",
       "      <td>0.304</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>BTG1</td>\n",
       "      <td>ENST00000256015.3:c.*301A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>124</td>\n",
       "      <td>12</td>\n",
       "      <td>129822166</td>\n",
       "      <td>129822166</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>8</td>\n",
       "      <td>0.200</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TMEM132D</td>\n",
       "      <td>ENST00000422113.2:c.1299+13C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>125</td>\n",
       "      <td>13</td>\n",
       "      <td>21205259</td>\n",
       "      <td>21205259</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>63</td>\n",
       "      <td>5</td>\n",
       "      <td>0.074</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IFT88</td>\n",
       "      <td>ENST00000319980.6:c.1413+18T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>126</td>\n",
       "      <td>13</td>\n",
       "      <td>53617212</td>\n",
       "      <td>53617212</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>82</td>\n",
       "      <td>4</td>\n",
       "      <td>0.047</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>OLFM4</td>\n",
       "      <td>ENST00000219022.2:c.571-28A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>127</td>\n",
       "      <td>13</td>\n",
       "      <td>96624810</td>\n",
       "      <td>96624810</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>96</td>\n",
       "      <td>4</td>\n",
       "      <td>0.040</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>UGGT2</td>\n",
       "      <td>ENST00000376747.3:c.1184+24A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>128</td>\n",
       "      <td>13</td>\n",
       "      <td>102375156</td>\n",
       "      <td>102375156</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>284</td>\n",
       "      <td>13</td>\n",
       "      <td>0.044</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FGF14</td>\n",
       "      <td>ENST00000376131.4:c.*25T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>129</td>\n",
       "      <td>13</td>\n",
       "      <td>108955694</td>\n",
       "      <td>108955694</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>53</td>\n",
       "      <td>0.654</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TNFSF13B</td>\n",
       "      <td>ENST00000375887.4:c.575A&gt;C</td>\n",
       "      <td>ENSP00000365048.3:p.Tyr192Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>130</td>\n",
       "      <td>14</td>\n",
       "      <td>21144748</td>\n",
       "      <td>21144748</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>3</td>\n",
       "      <td>0.049</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>Y_RNA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>131</td>\n",
       "      <td>14</td>\n",
       "      <td>34411365</td>\n",
       "      <td>34411365</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>82</td>\n",
       "      <td>4</td>\n",
       "      <td>0.047</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>EGLN3</td>\n",
       "      <td>ENST00000250457.3:c.357+8237T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>132</td>\n",
       "      <td>14</td>\n",
       "      <td>38014889</td>\n",
       "      <td>38014889</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MIPOL1</td>\n",
       "      <td>ENST00000327441.7:c.1263-1221A&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>133</td>\n",
       "      <td>14</td>\n",
       "      <td>51443360</td>\n",
       "      <td>51443360</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>0.075</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TRIM9</td>\n",
       "      <td>ENST00000298355.3:c.*649T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>134</td>\n",
       "      <td>14</td>\n",
       "      <td>52520704</td>\n",
       "      <td>52520704</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>52</td>\n",
       "      <td>3</td>\n",
       "      <td>0.055</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NID2</td>\n",
       "      <td>ENST00000216286.5:c.1069+34C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>135</td>\n",
       "      <td>14</td>\n",
       "      <td>86090050</td>\n",
       "      <td>86090050</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>15</td>\n",
       "      <td>8</td>\n",
       "      <td>0.348</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FLRT2</td>\n",
       "      <td>ENST00000330753.4:c.*209T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>136</td>\n",
       "      <td>14</td>\n",
       "      <td>96792127</td>\n",
       "      <td>96792127</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>21</td>\n",
       "      <td>0.300</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>ATG2B</td>\n",
       "      <td>ENST00000359933.4:c.2296G&gt;T</td>\n",
       "      <td>ENSP00000353010.4:p.Glu766Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>137</td>\n",
       "      <td>14</td>\n",
       "      <td>106318585</td>\n",
       "      <td>106318585</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>20</td>\n",
       "      <td>14</td>\n",
       "      <td>0.412</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IGHM</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>138</td>\n",
       "      <td>15</td>\n",
       "      <td>23811592</td>\n",
       "      <td>23811592</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>55</td>\n",
       "      <td>4</td>\n",
       "      <td>0.068</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MKRN3</td>\n",
       "      <td>ENST00000314520.3:c.663C&gt;T</td>\n",
       "      <td>ENSP00000313881.3:p.Pro221%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>139</td>\n",
       "      <td>15</td>\n",
       "      <td>25936873</td>\n",
       "      <td>25936873</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>10</td>\n",
       "      <td>0.303</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>ATP10A</td>\n",
       "      <td>ENST00000356865.6:c.3154G&gt;T</td>\n",
       "      <td>ENSP00000349325.6:p.Glu1052Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>140</td>\n",
       "      <td>15</td>\n",
       "      <td>25936905</td>\n",
       "      <td>25936905</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>16</td>\n",
       "      <td>0.340</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ATP10A</td>\n",
       "      <td>ENST00000356865.6:c.3122T&gt;A</td>\n",
       "      <td>ENSP00000349325.6:p.Val1041Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.119)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>141</td>\n",
       "      <td>15</td>\n",
       "      <td>42041712</td>\n",
       "      <td>42041712</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>4</td>\n",
       "      <td>0.154</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MGA</td>\n",
       "      <td>ENST00000219905.7:c.5907G&gt;A</td>\n",
       "      <td>ENSP00000219905.7:p.Lys1969%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>142</td>\n",
       "      <td>15</td>\n",
       "      <td>48063999</td>\n",
       "      <td>48063999</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>73</td>\n",
       "      <td>4</td>\n",
       "      <td>0.052</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SEMA6D</td>\n",
       "      <td>ENST00000316364.5:c.*17T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>143</td>\n",
       "      <td>15</td>\n",
       "      <td>79339155</td>\n",
       "      <td>79339155</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>16</td>\n",
       "      <td>0.364</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RASGRF1</td>\n",
       "      <td>ENST00000419573.3:c.811C&gt;T</td>\n",
       "      <td>ENSP00000405963.3:p.Arg271Trp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.927)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>144</td>\n",
       "      <td>16</td>\n",
       "      <td>23407432</td>\n",
       "      <td>23407432</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>5</td>\n",
       "      <td>0.122</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>COG7</td>\n",
       "      <td>ENST00000307149.5:c.1887+1935G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>145</td>\n",
       "      <td>16</td>\n",
       "      <td>28855522</td>\n",
       "      <td>28855522</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>57</td>\n",
       "      <td>7</td>\n",
       "      <td>0.109</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TUFM</td>\n",
       "      <td>ENST00000313511.3:c.922+29C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>146</td>\n",
       "      <td>16</td>\n",
       "      <td>46651564</td>\n",
       "      <td>46651564</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>4</td>\n",
       "      <td>0.073</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>SHCBP1</td>\n",
       "      <td>ENST00000303383.3:c.369G&gt;A</td>\n",
       "      <td>ENSP00000306473.3:p.Val123%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>147</td>\n",
       "      <td>16</td>\n",
       "      <td>56441862</td>\n",
       "      <td>56441862</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>0.085</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AMFR</td>\n",
       "      <td>ENST00000290649.5:c.666+14T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>148</td>\n",
       "      <td>16</td>\n",
       "      <td>56533893</td>\n",
       "      <td>56533893</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>5</td>\n",
       "      <td>0.116</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>BBS2</td>\n",
       "      <td>ENST00000245157.5:c.1398-74A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>149</td>\n",
       "      <td>16</td>\n",
       "      <td>75252870</td>\n",
       "      <td>75252870</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>17</td>\n",
       "      <td>0.321</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CTRB1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>150</td>\n",
       "      <td>16</td>\n",
       "      <td>81929553</td>\n",
       "      <td>81929553</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>6</td>\n",
       "      <td>0.102</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PLCG2</td>\n",
       "      <td>ENST00000359376.3:c.1193+21G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>151</td>\n",
       "      <td>16</td>\n",
       "      <td>87742778</td>\n",
       "      <td>87742778</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>15</td>\n",
       "      <td>0.326</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KLHDC4</td>\n",
       "      <td>ENST00000270583.5:c.1447+93G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>152</td>\n",
       "      <td>17</td>\n",
       "      <td>6381939</td>\n",
       "      <td>6381939</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.077</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PITPNM3</td>\n",
       "      <td>ENST00000262483.8:c.705C&gt;T</td>\n",
       "      <td>ENSP00000262483.8:p.Thr235%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>153</td>\n",
       "      <td>17</td>\n",
       "      <td>6981397</td>\n",
       "      <td>6981397</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.077</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CLEC10A</td>\n",
       "      <td>ENST00000254868.4:c.103T&gt;C</td>\n",
       "      <td>ENSP00000254868.4:p.Cys35Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>154</td>\n",
       "      <td>17</td>\n",
       "      <td>7578260</td>\n",
       "      <td>7578260</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>15</td>\n",
       "      <td>0.288</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TP53</td>\n",
       "      <td>ENST00000269305.4:c.589G&gt;A</td>\n",
       "      <td>ENSP00000269305.4:p.Val197Met</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>probably_damaging(0.961)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>155</td>\n",
       "      <td>17</td>\n",
       "      <td>8215534</td>\n",
       "      <td>8215534</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>5</td>\n",
       "      <td>0.156</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ARHGEF15</td>\n",
       "      <td>ENST00000361926.3:c.177C&gt;T</td>\n",
       "      <td>ENSP00000355026.3:p.Thr59%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>156</td>\n",
       "      <td>17</td>\n",
       "      <td>38950272</td>\n",
       "      <td>38950272</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>4</td>\n",
       "      <td>0.114</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>KRT28</td>\n",
       "      <td>ENST00000306658.7:c.1005A&gt;G</td>\n",
       "      <td>ENSP00000305263.7:p.Thr335%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>157</td>\n",
       "      <td>17</td>\n",
       "      <td>38991518</td>\n",
       "      <td>38991518</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>3</td>\n",
       "      <td>0.136</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TMEM99</td>\n",
       "      <td>ENST00000301665.3:c.750T&gt;C</td>\n",
       "      <td>ENSP00000301665.3:p.Cys250%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>158</td>\n",
       "      <td>17</td>\n",
       "      <td>39593768</td>\n",
       "      <td>39593768</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRT38</td>\n",
       "      <td>ENST00000246646.3:c.1267T&gt;C</td>\n",
       "      <td>ENSP00000246646.2:p.Ser423Pro</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(0.58)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>159</td>\n",
       "      <td>17</td>\n",
       "      <td>39776854</td>\n",
       "      <td>39776854</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>12</td>\n",
       "      <td>3</td>\n",
       "      <td>0.200</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KRT17</td>\n",
       "      <td>ENST00000311208.8:c.1182-44C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>160</td>\n",
       "      <td>17</td>\n",
       "      <td>43196366</td>\n",
       "      <td>43196366</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>76</td>\n",
       "      <td>15</td>\n",
       "      <td>0.165</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLCD3</td>\n",
       "      <td>ENST00000322765.5:c.729C&gt;T</td>\n",
       "      <td>ENSP00000313731.5:p.Ile243%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>161</td>\n",
       "      <td>17</td>\n",
       "      <td>44128125</td>\n",
       "      <td>44128125</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>50</td>\n",
       "      <td>4</td>\n",
       "      <td>0.074</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KANSL1</td>\n",
       "      <td>ENST00000262419.6:c.1849-55T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>162</td>\n",
       "      <td>17</td>\n",
       "      <td>44248848</td>\n",
       "      <td>44248848</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>66</td>\n",
       "      <td>4</td>\n",
       "      <td>0.057</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KANSL1</td>\n",
       "      <td>ENST00000262419.6:c.662C&gt;T</td>\n",
       "      <td>ENSP00000262419.6:p.Thr221Ile</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.277)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>163</td>\n",
       "      <td>17</td>\n",
       "      <td>57743833</td>\n",
       "      <td>57743833</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>22</td>\n",
       "      <td>8</td>\n",
       "      <td>0.267</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>CLTC</td>\n",
       "      <td>ENST00000269122.3:c.1783-8T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>164</td>\n",
       "      <td>17</td>\n",
       "      <td>58888380</td>\n",
       "      <td>58888380</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>5</td>\n",
       "      <td>0.082</td>\n",
       "      <td>mature_miRNA_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>AC079005.1</td>\n",
       "      <td>ENST00000390181.1:n.63A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>165</td>\n",
       "      <td>17</td>\n",
       "      <td>61685187</td>\n",
       "      <td>61685187</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>73</td>\n",
       "      <td>4</td>\n",
       "      <td>0.052</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TACO1</td>\n",
       "      <td>ENST00000258975.6:c.719A&gt;G</td>\n",
       "      <td>ENSP00000258975.6:p.His240Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.023)</td>\n",
       "      <td>tolerated(0.75)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>166</td>\n",
       "      <td>17</td>\n",
       "      <td>72369850</td>\n",
       "      <td>72369850</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>8</td>\n",
       "      <td>0.229</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GPR142</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>167</td>\n",
       "      <td>17</td>\n",
       "      <td>74382542</td>\n",
       "      <td>74382542</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>15</td>\n",
       "      <td>0.238</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>UBE2O</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>168</td>\n",
       "      <td>17</td>\n",
       "      <td>76130947</td>\n",
       "      <td>76130947</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>3</td>\n",
       "      <td>0.064</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TMC8</td>\n",
       "      <td>ENST00000318430.5:c.988-4G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>169</td>\n",
       "      <td>17</td>\n",
       "      <td>78071171</td>\n",
       "      <td>78071171</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>8</td>\n",
       "      <td>0.167</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>GAA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>170</td>\n",
       "      <td>18</td>\n",
       "      <td>61305135</td>\n",
       "      <td>61305135</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>109</td>\n",
       "      <td>5</td>\n",
       "      <td>0.044</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SERPINB4</td>\n",
       "      <td>ENST00000341074.5:c.991A&gt;G</td>\n",
       "      <td>ENSP00000343445.5:p.Lys331Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.96)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>171</td>\n",
       "      <td>19</td>\n",
       "      <td>2041032</td>\n",
       "      <td>2041032</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>4</td>\n",
       "      <td>0.114</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>MKNK2</td>\n",
       "      <td>ENST00000250896.3:c.1110+7A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>172</td>\n",
       "      <td>19</td>\n",
       "      <td>2431488</td>\n",
       "      <td>2431488</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>14</td>\n",
       "      <td>0.298</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TIMM13</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>173</td>\n",
       "      <td>19</td>\n",
       "      <td>3586543</td>\n",
       "      <td>3586543</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>69</td>\n",
       "      <td>5</td>\n",
       "      <td>0.068</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GIPC3</td>\n",
       "      <td>ENST00000322315.5:c.276G&gt;A</td>\n",
       "      <td>ENSP00000319254.5:p.Leu92%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>174</td>\n",
       "      <td>19</td>\n",
       "      <td>6400058</td>\n",
       "      <td>6400058</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>9</td>\n",
       "      <td>0.158</td>\n",
       "      <td>intron_variant&amp;non_coding_transcript_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CTB-180A7.8</td>\n",
       "      <td>ENST00000398173.3:n.204-61A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>175</td>\n",
       "      <td>19</td>\n",
       "      <td>8654245</td>\n",
       "      <td>8654245</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>13</td>\n",
       "      <td>0.289</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ADAMTS10</td>\n",
       "      <td>ENST00000270328.4:c.2039T&gt;G</td>\n",
       "      <td>ENSP00000270328.4:p.Val680Gly</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>176</td>\n",
       "      <td>19</td>\n",
       "      <td>15378338</td>\n",
       "      <td>15378338</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.078</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>BRD4</td>\n",
       "      <td>ENST00000263377.2:c.448G&gt;A</td>\n",
       "      <td>ENSP00000263377.1:p.Ala150Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>177</td>\n",
       "      <td>19</td>\n",
       "      <td>19656563</td>\n",
       "      <td>19656563</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>75</td>\n",
       "      <td>9</td>\n",
       "      <td>0.107</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>CILP2</td>\n",
       "      <td>ENST00000291495.5:c.3209G&gt;T</td>\n",
       "      <td>ENSP00000291495.3:p.Arg1070Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.892)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>178</td>\n",
       "      <td>19</td>\n",
       "      <td>35616086</td>\n",
       "      <td>35616086</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.125</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LGI4</td>\n",
       "      <td>ENST00000310123.3:c.*11G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>179</td>\n",
       "      <td>19</td>\n",
       "      <td>41825546</td>\n",
       "      <td>41825546</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>68</td>\n",
       "      <td>14</td>\n",
       "      <td>0.171</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>CCDC97</td>\n",
       "      <td>ENST00000269967.3:c.570G&gt;T</td>\n",
       "      <td>ENSP00000269967.2:p.Gly190%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>180</td>\n",
       "      <td>19</td>\n",
       "      <td>46846056</td>\n",
       "      <td>46846056</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>5</td>\n",
       "      <td>0.098</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PPP5C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>181</td>\n",
       "      <td>19</td>\n",
       "      <td>49227711</td>\n",
       "      <td>49227711</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>39</td>\n",
       "      <td>15</td>\n",
       "      <td>0.278</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>RASIP1</td>\n",
       "      <td>ENST00000222145.4:c.2427G&gt;A</td>\n",
       "      <td>ENSP00000222145.3:p.Leu809%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>182</td>\n",
       "      <td>19</td>\n",
       "      <td>51323232</td>\n",
       "      <td>51323232</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>3</td>\n",
       "      <td>0.054</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KLK1</td>\n",
       "      <td>ENST00000301420.2:c.556A&gt;G</td>\n",
       "      <td>ENSP00000301420.1:p.Lys186Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.02)</td>\n",
       "      <td>tolerated(0.84)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>183</td>\n",
       "      <td>20</td>\n",
       "      <td>309751</td>\n",
       "      <td>309751</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>32</td>\n",
       "      <td>16</td>\n",
       "      <td>0.333</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SOX12</td>\n",
       "      <td>ENST00000342665.2:c.*2235T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>184</td>\n",
       "      <td>20</td>\n",
       "      <td>5560725</td>\n",
       "      <td>5560725</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GPCPD1</td>\n",
       "      <td>ENST00000379019.4:c.402A&gt;G</td>\n",
       "      <td>ENSP00000368305.4:p.Arg134%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>185</td>\n",
       "      <td>20</td>\n",
       "      <td>33586203</td>\n",
       "      <td>33586203</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>82</td>\n",
       "      <td>7</td>\n",
       "      <td>0.079</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TRPC4AP</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>186</td>\n",
       "      <td>21</td>\n",
       "      <td>28307024</td>\n",
       "      <td>28307024</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>6</td>\n",
       "      <td>0.140</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ADAMTS5</td>\n",
       "      <td>ENST00000284987.5:c.1450G&gt;A</td>\n",
       "      <td>ENSP00000284987.5:p.Glu484Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.71)</td>\n",
       "      <td>tolerated(0.16)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>187</td>\n",
       "      <td>21</td>\n",
       "      <td>32525074</td>\n",
       "      <td>32525074</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>5</td>\n",
       "      <td>0.089</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TIAM1</td>\n",
       "      <td>ENST00000286827.3:c.3246C&gt;A</td>\n",
       "      <td>ENSP00000286827.3:p.Asp1082Glu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>tolerated(0.96)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>188</td>\n",
       "      <td>21</td>\n",
       "      <td>33847179</td>\n",
       "      <td>33847179</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>45</td>\n",
       "      <td>37</td>\n",
       "      <td>0.451</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>EVA1C</td>\n",
       "      <td>ENST00000300255.2:c.634+7023A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>189</td>\n",
       "      <td>21</td>\n",
       "      <td>41452034</td>\n",
       "      <td>41452034</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>67</td>\n",
       "      <td>5</td>\n",
       "      <td>0.069</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DSCAM</td>\n",
       "      <td>ENST00000400454.1:c.4420+45G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>190</td>\n",
       "      <td>21</td>\n",
       "      <td>45675904</td>\n",
       "      <td>45675904</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>67</td>\n",
       "      <td>4</td>\n",
       "      <td>0.056</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DNMT3L</td>\n",
       "      <td>ENST00000270172.3:c.604+46A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>191</td>\n",
       "      <td>22</td>\n",
       "      <td>20819078</td>\n",
       "      <td>20819078</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>10</td>\n",
       "      <td>0.233</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KLHL22</td>\n",
       "      <td>ENST00000328879.4:c.1112+67A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>192</td>\n",
       "      <td>22</td>\n",
       "      <td>37499386</td>\n",
       "      <td>37499386</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>3</td>\n",
       "      <td>0.061</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>TMPRSS6</td>\n",
       "      <td>ENST00000346753.3:c.99G&gt;A</td>\n",
       "      <td>ENSP00000334962.5:p.Pro33%3D</td>\n",
       "      <td>likely_benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>193</td>\n",
       "      <td>22</td>\n",
       "      <td>47274638</td>\n",
       "      <td>47274638</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>4</td>\n",
       "      <td>0.080</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TBC1D22A</td>\n",
       "      <td>ENST00000337137.4:c.708+19C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>194</td>\n",
       "      <td>22</td>\n",
       "      <td>49947515</td>\n",
       "      <td>49947515</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>10</td>\n",
       "      <td>0.244</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP1-29C18.10</td>\n",
       "      <td>ENST00000391625.2:n.1288C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>195</td>\n",
       "      <td>X</td>\n",
       "      <td>7252077</td>\n",
       "      <td>7252077</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>26</td>\n",
       "      <td>14</td>\n",
       "      <td>0.350</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>STS</td>\n",
       "      <td>ENST00000217961.4:c.1307G&gt;A</td>\n",
       "      <td>ENSP00000217961.4:p.Arg436His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>196</td>\n",
       "      <td>X</td>\n",
       "      <td>27998117</td>\n",
       "      <td>27998117</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>54</td>\n",
       "      <td>3</td>\n",
       "      <td>0.053</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>DCAF8L1</td>\n",
       "      <td>ENST00000441525.1:c.1335C&gt;G</td>\n",
       "      <td>ENSP00000405222.1:p.Phe445Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.609)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>197</td>\n",
       "      <td>X</td>\n",
       "      <td>50122932</td>\n",
       "      <td>50122932</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.083</td>\n",
       "      <td>intron_variant&amp;non_coding_transcript_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DGKK</td>\n",
       "      <td>ENST00000376025.2:n.2814+47T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>198</td>\n",
       "      <td>X</td>\n",
       "      <td>50212273</td>\n",
       "      <td>50212273</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>3</td>\n",
       "      <td>0.088</td>\n",
       "      <td>intron_variant&amp;non_coding_transcript_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DGKK</td>\n",
       "      <td>ENST00000376025.2:n.705+760C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>199</td>\n",
       "      <td>X</td>\n",
       "      <td>77243944</td>\n",
       "      <td>77243944</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>56</td>\n",
       "      <td>3</td>\n",
       "      <td>0.051</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ATP7A</td>\n",
       "      <td>ENST00000341514.6:c.327G&gt;A</td>\n",
       "      <td>ENSP00000345728.6:p.Lys109%3D</td>\n",
       "      <td>benign</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>200</td>\n",
       "      <td>X</td>\n",
       "      <td>78216024</td>\n",
       "      <td>78216024</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>48</td>\n",
       "      <td>5</td>\n",
       "      <td>0.094</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>P2RY10</td>\n",
       "      <td>ENST00000171757.2:c.7A&gt;C</td>\n",
       "      <td>ENSP00000171757.2:p.Asn3His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated_low_confidence(0.26)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>201</td>\n",
       "      <td>X</td>\n",
       "      <td>117810737</td>\n",
       "      <td>117810737</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>72</td>\n",
       "      <td>4</td>\n",
       "      <td>0.053</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DOCK11</td>\n",
       "      <td>ENST00000276202.7:c.5460+37T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>202</td>\n",
       "      <td>X</td>\n",
       "      <td>151815441</td>\n",
       "      <td>151815441</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>0.085</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>GABRQ</td>\n",
       "      <td>ENST00000370306.2:c.339T&gt;A</td>\n",
       "      <td>ENSP00000359329.2:p.Thr113%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>203</td>\n",
       "      <td>Y</td>\n",
       "      <td>15409597</td>\n",
       "      <td>15409597</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>13</td>\n",
       "      <td>0.295</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>UTY</td>\n",
       "      <td>ENST00000331397.4:c.3709G&gt;A</td>\n",
       "      <td>ENSP00000328939.4:p.Glu1237Lys</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.824)</td>\n",
       "      <td>tolerated(0.17)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>204</td>\n",
       "      <td>Y</td>\n",
       "      <td>15409751</td>\n",
       "      <td>15409751</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>5</td>\n",
       "      <td>0.156</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>UTY</td>\n",
       "      <td>ENST00000331397.4:c.3578-23G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>205</td>\n",
       "      <td>4</td>\n",
       "      <td>173930335</td>\n",
       "      <td>173930335</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>10</td>\n",
       "      <td>0.270</td>\n",
       "      <td>frameshift_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>GALNTL6</td>\n",
       "      <td>ENST00000506823.1:c.1378_1379insT</td>\n",
       "      <td>ENSP00000423313.1:p.Asn460IlefsTer30</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>206</td>\n",
       "      <td>7</td>\n",
       "      <td>36124231</td>\n",
       "      <td>36124231</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>5</td>\n",
       "      <td>0.102</td>\n",
       "      <td>frameshift_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>PP13004</td>\n",
       "      <td>ENST00000381493.2:c.234dup</td>\n",
       "      <td>ENSP00000370904.2:p.Lys79Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>207</td>\n",
       "      <td>10</td>\n",
       "      <td>124749672</td>\n",
       "      <td>124749674</td>\n",
       "      <td>3</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>10</td>\n",
       "      <td>0.175</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IKZF5</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>208</td>\n",
       "      <td>X</td>\n",
       "      <td>54282134</td>\n",
       "      <td>54282135</td>\n",
       "      <td>2</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>19</td>\n",
       "      <td>0.452</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>WNK3</td>\n",
       "      <td>ENST00000354646.2:c.2250+38del</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>209</td>\n",
       "      <td>1</td>\n",
       "      <td>97700589</td>\n",
       "      <td>97700589</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>3</td>\n",
       "      <td>0.210</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DPYD</td>\n",
       "      <td>ENST00000370192.3:c.2300-39G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>not_provided</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>210</td>\n",
       "      <td>1</td>\n",
       "      <td>151679963</td>\n",
       "      <td>151679963</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.116</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CELF3</td>\n",
       "      <td>ENST00000290583.4:c.772+20G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>211</td>\n",
       "      <td>1</td>\n",
       "      <td>152280084</td>\n",
       "      <td>152280084</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>168</td>\n",
       "      <td>25</td>\n",
       "      <td>0.133</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>FLG</td>\n",
       "      <td>ENST00000368799.1:c.7278C&gt;T</td>\n",
       "      <td>ENSP00000357789.1:p.Ser2426%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>212</td>\n",
       "      <td>1</td>\n",
       "      <td>157660351</td>\n",
       "      <td>157660351</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>21</td>\n",
       "      <td>6</td>\n",
       "      <td>0.241</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FCRL3</td>\n",
       "      <td>ENST00000368184.3:c.1412-28T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>213</td>\n",
       "      <td>1</td>\n",
       "      <td>160771720</td>\n",
       "      <td>160771720</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>30</td>\n",
       "      <td>6</td>\n",
       "      <td>0.164</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LY9</td>\n",
       "      <td>ENST00000263285.6:c.454+1848A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>214</td>\n",
       "      <td>1</td>\n",
       "      <td>166245976</td>\n",
       "      <td>166245976</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>93</td>\n",
       "      <td>21</td>\n",
       "      <td>0.189</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-9L18.2</td>\n",
       "      <td>ENST00000400979.2:n.859G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>215</td>\n",
       "      <td>2</td>\n",
       "      <td>97882963</td>\n",
       "      <td>97882963</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>70</td>\n",
       "      <td>17</td>\n",
       "      <td>0.202</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ANKRD36</td>\n",
       "      <td>ENST00000420699.2:c.3780+21A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>216</td>\n",
       "      <td>3</td>\n",
       "      <td>7620139</td>\n",
       "      <td>7620139</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>5</td>\n",
       "      <td>0.136</td>\n",
       "      <td>stop_gained</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>GRM7</td>\n",
       "      <td>ENST00000357716.4:c.1546C&gt;T</td>\n",
       "      <td>ENSP00000350348.4:p.Arg516Ter</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>217</td>\n",
       "      <td>3</td>\n",
       "      <td>10311939</td>\n",
       "      <td>10311939</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>67</td>\n",
       "      <td>4</td>\n",
       "      <td>0.067</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TATDN2</td>\n",
       "      <td>ENST00000287652.4:c.1073C&gt;T</td>\n",
       "      <td>ENSP00000287652.4:p.Pro358Leu</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.006)</td>\n",
       "      <td>tolerated(0.29)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>218</td>\n",
       "      <td>3</td>\n",
       "      <td>48731921</td>\n",
       "      <td>48731921</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>3</td>\n",
       "      <td>0.167</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IP6K2</td>\n",
       "      <td>ENST00000328631.5:c.202+602T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>219</td>\n",
       "      <td>4</td>\n",
       "      <td>85564148</td>\n",
       "      <td>85564148</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>36</td>\n",
       "      <td>3</td>\n",
       "      <td>0.097</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CDS1</td>\n",
       "      <td>ENST00000295887.5:c.1033-29C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>220</td>\n",
       "      <td>5</td>\n",
       "      <td>126776623</td>\n",
       "      <td>126776623</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>58</td>\n",
       "      <td>6</td>\n",
       "      <td>0.106</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MEGF10</td>\n",
       "      <td>ENST00000274473.6:c.2362+64T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>221</td>\n",
       "      <td>5</td>\n",
       "      <td>137525311</td>\n",
       "      <td>137525311</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>9</td>\n",
       "      <td>0.237</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CDC23</td>\n",
       "      <td>ENST00000394886.2:c.1623+11G&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>222</td>\n",
       "      <td>5</td>\n",
       "      <td>180037317</td>\n",
       "      <td>180037317</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>3</td>\n",
       "      <td>0.138</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>FLT4</td>\n",
       "      <td>ENST00000261937.6:c.3687-292G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>223</td>\n",
       "      <td>7</td>\n",
       "      <td>23454979</td>\n",
       "      <td>23454979</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>126</td>\n",
       "      <td>8</td>\n",
       "      <td>0.066</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IGF2BP3</td>\n",
       "      <td>ENST00000258729.3:c.285+3416C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>224</td>\n",
       "      <td>7</td>\n",
       "      <td>36396358</td>\n",
       "      <td>36396358</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>27</td>\n",
       "      <td>4</td>\n",
       "      <td>0.153</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>KIAA0895</td>\n",
       "      <td>ENST00000297063.6:c.867+153C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>225</td>\n",
       "      <td>7</td>\n",
       "      <td>100244750</td>\n",
       "      <td>100244750</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>4</td>\n",
       "      <td>0.166</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ACTL6B</td>\n",
       "      <td>ENST00000160382.5:c.822-42T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>226</td>\n",
       "      <td>7</td>\n",
       "      <td>137025901</td>\n",
       "      <td>137025901</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>4</td>\n",
       "      <td>0.158</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PTN</td>\n",
       "      <td>ENST00000348225.2:c.-2+2284T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>227</td>\n",
       "      <td>7</td>\n",
       "      <td>158550099</td>\n",
       "      <td>158550099</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>31</td>\n",
       "      <td>7</td>\n",
       "      <td>0.199</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ESYT2</td>\n",
       "      <td>ENST00000251527.5:c.1563+2078G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>228</td>\n",
       "      <td>8</td>\n",
       "      <td>125551397</td>\n",
       "      <td>125551397</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>45</td>\n",
       "      <td>10</td>\n",
       "      <td>0.192</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NDUFB9</td>\n",
       "      <td>ENST00000276689.3:c.-31C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>229</td>\n",
       "      <td>9</td>\n",
       "      <td>88464268</td>\n",
       "      <td>88464268</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>7</td>\n",
       "      <td>0.153</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-213G2.3</td>\n",
       "      <td>ENST00000431724.1:n.1498A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>230</td>\n",
       "      <td>9</td>\n",
       "      <td>107332395</td>\n",
       "      <td>107332395</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>15</td>\n",
       "      <td>0.262</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>OR13C8</td>\n",
       "      <td>ENST00000335040.1:c.947T&gt;C</td>\n",
       "      <td>ENSP00000334068.1:p.Phe316Ser</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.003)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>231</td>\n",
       "      <td>10</td>\n",
       "      <td>7780723</td>\n",
       "      <td>7780723</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>24</td>\n",
       "      <td>10</td>\n",
       "      <td>0.305</td>\n",
       "      <td>splice_donor_variant</td>\n",
       "      <td>HIGH</td>\n",
       "      <td>ITIH2</td>\n",
       "      <td>ENST00000358415.4:c.2095+2T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>232</td>\n",
       "      <td>10</td>\n",
       "      <td>48390042</td>\n",
       "      <td>48390042</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>16</td>\n",
       "      <td>5</td>\n",
       "      <td>0.260</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>RBP3</td>\n",
       "      <td>ENST00000224600.4:c.836C&gt;T</td>\n",
       "      <td>ENSP00000224600.4:p.Thr279Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.998)</td>\n",
       "      <td>tolerated(0.07)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>233</td>\n",
       "      <td>10</td>\n",
       "      <td>134004239</td>\n",
       "      <td>134004239</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>67</td>\n",
       "      <td>6</td>\n",
       "      <td>0.093</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DPYSL4</td>\n",
       "      <td>ENST00000338492.4:c.40-12T&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>234</td>\n",
       "      <td>11</td>\n",
       "      <td>26587352</td>\n",
       "      <td>26587352</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>3</td>\n",
       "      <td>0.103</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ANO3</td>\n",
       "      <td>ENST00000256737.3:c.1447+6026T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>235</td>\n",
       "      <td>11</td>\n",
       "      <td>60885630</td>\n",
       "      <td>60885630</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>6</td>\n",
       "      <td>0.130</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>CD5</td>\n",
       "      <td>ENST00000347785.3:c.95-17A&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>236</td>\n",
       "      <td>11</td>\n",
       "      <td>67203501</td>\n",
       "      <td>67203501</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>34</td>\n",
       "      <td>5</td>\n",
       "      <td>0.146</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RPS6KB2</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>237</td>\n",
       "      <td>12</td>\n",
       "      <td>40085906</td>\n",
       "      <td>40085906</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>28</td>\n",
       "      <td>4</td>\n",
       "      <td>0.146</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>C12orf40</td>\n",
       "      <td>ENST00000324616.5:c.1403A&gt;T</td>\n",
       "      <td>ENSP00000317671.5:p.Lys468Met</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.031)</td>\n",
       "      <td>tolerated(0.1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>238</td>\n",
       "      <td>12</td>\n",
       "      <td>131649643</td>\n",
       "      <td>131649643</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>40</td>\n",
       "      <td>4</td>\n",
       "      <td>0.109</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>RP11-638F5.1</td>\n",
       "      <td>ENST00000376678.2:n.88G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>239</td>\n",
       "      <td>14</td>\n",
       "      <td>74976089</td>\n",
       "      <td>74976089</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>3</td>\n",
       "      <td>0.081</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>LTBP2</td>\n",
       "      <td>ENST00000261978.4:c.3278-23G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>240</td>\n",
       "      <td>14</td>\n",
       "      <td>94912977</td>\n",
       "      <td>94912977</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>3</td>\n",
       "      <td>0.105</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SERPINA11</td>\n",
       "      <td>ENST00000334708.3:c.644-36A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>241</td>\n",
       "      <td>14</td>\n",
       "      <td>106610235</td>\n",
       "      <td>106610235</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>4</td>\n",
       "      <td>0.101</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IGHV3-15</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>242</td>\n",
       "      <td>15</td>\n",
       "      <td>41849496</td>\n",
       "      <td>41849496</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>19</td>\n",
       "      <td>0.400</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TYRO3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>243</td>\n",
       "      <td>15</td>\n",
       "      <td>45415231</td>\n",
       "      <td>45415231</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>49</td>\n",
       "      <td>4</td>\n",
       "      <td>0.090</td>\n",
       "      <td>5_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>DUOXA1</td>\n",
       "      <td>ENST00000267803.4:c.-19C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>244</td>\n",
       "      <td>15</td>\n",
       "      <td>56961125</td>\n",
       "      <td>56961125</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>14</td>\n",
       "      <td>8</td>\n",
       "      <td>0.373</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ZNF280D</td>\n",
       "      <td>ENST00000267807.7:c.1441A&gt;C</td>\n",
       "      <td>ENSP00000267807.7:p.Arg481%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>245</td>\n",
       "      <td>16</td>\n",
       "      <td>70553596</td>\n",
       "      <td>70553596</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>42</td>\n",
       "      <td>5</td>\n",
       "      <td>0.121</td>\n",
       "      <td>upstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SF3B3</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>246</td>\n",
       "      <td>16</td>\n",
       "      <td>88809368</td>\n",
       "      <td>88809368</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>51</td>\n",
       "      <td>4</td>\n",
       "      <td>0.088</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PIEZO1</td>\n",
       "      <td>ENST00000301015.9:c.161-538C&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>247</td>\n",
       "      <td>17</td>\n",
       "      <td>9057860</td>\n",
       "      <td>9057860</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>46</td>\n",
       "      <td>5</td>\n",
       "      <td>0.112</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NTN1</td>\n",
       "      <td>ENST00000173229.2:c.1019-8270T&gt;C</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>248</td>\n",
       "      <td>17</td>\n",
       "      <td>43487574</td>\n",
       "      <td>43487574</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>38</td>\n",
       "      <td>3</td>\n",
       "      <td>0.094</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ARHGAP27</td>\n",
       "      <td>ENST00000376922.2:c.-366-4125G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>249</td>\n",
       "      <td>17</td>\n",
       "      <td>60865795</td>\n",
       "      <td>60865795</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>8</td>\n",
       "      <td>0.311</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>MARCH10</td>\n",
       "      <td>ENST00000311269.5:c.210+46C&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>250</td>\n",
       "      <td>19</td>\n",
       "      <td>1531885</td>\n",
       "      <td>1531885</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>53</td>\n",
       "      <td>6</td>\n",
       "      <td>0.114</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PLK5</td>\n",
       "      <td>ENST00000334770.4:c.714+3A&gt;G</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>251</td>\n",
       "      <td>19</td>\n",
       "      <td>3823237</td>\n",
       "      <td>3823237</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>66</td>\n",
       "      <td>4</td>\n",
       "      <td>0.069</td>\n",
       "      <td>splice_region_variant&amp;intron_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>ZFR2</td>\n",
       "      <td>ENST00000262961.4:c.1371+7C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>252</td>\n",
       "      <td>19</td>\n",
       "      <td>12541532</td>\n",
       "      <td>12541532</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>78</td>\n",
       "      <td>6</td>\n",
       "      <td>0.081</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF443</td>\n",
       "      <td>ENST00000301547.5:c.1454G&gt;T</td>\n",
       "      <td>ENSP00000301547.5:p.Cys485Phe</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0)</td>\n",
       "      <td>tolerated(1)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>253</td>\n",
       "      <td>19</td>\n",
       "      <td>12541547</td>\n",
       "      <td>12541547</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>80</td>\n",
       "      <td>6</td>\n",
       "      <td>0.080</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ZNF443</td>\n",
       "      <td>ENST00000301547.5:c.1439A&gt;C</td>\n",
       "      <td>ENSP00000301547.5:p.Lys480Thr</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.93)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>254</td>\n",
       "      <td>19</td>\n",
       "      <td>49950356</td>\n",
       "      <td>49950356</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>44</td>\n",
       "      <td>4</td>\n",
       "      <td>0.098</td>\n",
       "      <td>splice_region_variant&amp;synonymous_variant</td>\n",
       "      <td>LOW</td>\n",
       "      <td>PIH1D1</td>\n",
       "      <td>ENST00000262265.5:c.612G&gt;C</td>\n",
       "      <td>ENSP00000262265.4:p.Gly204%3D</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>255</td>\n",
       "      <td>19</td>\n",
       "      <td>52705184</td>\n",
       "      <td>52705184</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>47</td>\n",
       "      <td>4</td>\n",
       "      <td>0.093</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>PPP2R1A</td>\n",
       "      <td>ENST00000322088.6:c.79-13G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>256</td>\n",
       "      <td>22</td>\n",
       "      <td>17600293</td>\n",
       "      <td>17600293</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>3</td>\n",
       "      <td>0.117</td>\n",
       "      <td>downstream_gene_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>IL17RA</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>257</td>\n",
       "      <td>22</td>\n",
       "      <td>46684528</td>\n",
       "      <td>46684528</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>3</td>\n",
       "      <td>0.167</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>TTC38</td>\n",
       "      <td>ENST00000381031.3:c.1082+43C&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>258</td>\n",
       "      <td>X</td>\n",
       "      <td>48123139</td>\n",
       "      <td>48123139</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>113</td>\n",
       "      <td>8</td>\n",
       "      <td>0.072</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>SSX1</td>\n",
       "      <td>ENST00000376919.3:c.331-78G&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>259</td>\n",
       "      <td>X</td>\n",
       "      <td>77268502</td>\n",
       "      <td>77268502</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>59</td>\n",
       "      <td>4</td>\n",
       "      <td>0.076</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ATP7A</td>\n",
       "      <td>ENST00000341514.6:c.2299G&gt;C</td>\n",
       "      <td>ENSP00000345728.6:p.Val767Leu</td>\n",
       "      <td>benign</td>\n",
       "      <td>probably_damaging(0.961)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>260</td>\n",
       "      <td>X</td>\n",
       "      <td>107924910</td>\n",
       "      <td>107924910</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>62</td>\n",
       "      <td>5</td>\n",
       "      <td>0.085</td>\n",
       "      <td>intron_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>COL4A5</td>\n",
       "      <td>ENST00000328300.6:c.4088-80G&gt;T</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>261</td>\n",
       "      <td>2</td>\n",
       "      <td>238303417</td>\n",
       "      <td>238303418</td>\n",
       "      <td>2</td>\n",
       "      <td>*</td>\n",
       "      <td>77</td>\n",
       "      <td>22</td>\n",
       "      <td>0.228</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>COL6A3</td>\n",
       "      <td>ENST00000295550.4:c.521_522delinsAC</td>\n",
       "      <td>ENSP00000295550.4:p.Ala174Asp</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.993)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    seqnames      start        end  width strand   AD  AD.1     AF  \\\n",
       "0          1    1267294    1267294      1      *   36     8  0.182   \n",
       "1          1   16161836   16161836      1      *   44     3  0.064   \n",
       "2          1   37324811   37324811      1      *   34     4  0.105   \n",
       "3          1   89654458   89654458      1      *   41     4  0.089   \n",
       "4          1   94488912   94488912      1      *   23     7  0.233   \n",
       "5          1  149859333  149859333      1      *   30     3  0.091   \n",
       "6          1  154841903  154841903      1      *   54     3  0.053   \n",
       "7          1  158623085  158623085      1      *   44    10  0.185   \n",
       "8          1  158632710  158632710      1      *   46     9  0.164   \n",
       "9          1  169781280  169781280      1      *   51    12  0.190   \n",
       "10         1  169890543  169890543      1      *   41     3  0.068   \n",
       "11         1  177902482  177902482      1      *   93     5  0.051   \n",
       "12         1  208200122  208200122      1      *   16     6  0.273   \n",
       "13         1  248004756  248004756      1      *   53     5  0.086   \n",
       "14         2   27457500   27457500      1      *   44    19  0.302   \n",
       "15         2   32692725   32692725      1      *   63     4  0.060   \n",
       "16         2   62451310   62451310      1      *   28    10  0.263   \n",
       "17         2   64862518   64862518      1      *   23     8  0.258   \n",
       "18         2   71062648   71062648      1      *   67     3  0.043   \n",
       "19         2   74185876   74185876      1      *   64     3  0.045   \n",
       "20         2   87085411   87085411      1      *   37    15  0.288   \n",
       "21         2  137814138  137814138      1      *   91     4  0.042   \n",
       "22         2  137852449  137852449      1      *   56     9  0.138   \n",
       "23         2  165996101  165996101      1      *   60    13  0.178   \n",
       "24         2  167266247  167266247      1      *   31     7  0.184   \n",
       "25         2  168921838  168921838      1      *   60     3  0.048   \n",
       "26         2  170402703  170402703      1      *   29     5  0.147   \n",
       "27         2  183623602  183623602      1      *   29     3  0.094   \n",
       "28         2  198570729  198570729      1      *   97     8  0.076   \n",
       "29         2  201434487  201434487      1      *   29    11  0.275   \n",
       "30         2  218762536  218762536      1      *   54    11  0.169   \n",
       "31         2  226378136  226378136      1      *   48    14  0.226   \n",
       "32         3    4810190    4810190      1      *   58     3  0.049   \n",
       "33         3   39136589   39136589      1      *   92     4  0.042   \n",
       "34         3   44438259   44438259      1      *   54     8  0.129   \n",
       "35         3   57875754   57875754      1      *   35     4  0.103   \n",
       "36         3   60898389   60898389      1      *   44     4  0.083   \n",
       "37         3   84339405   84339405      1      *   65     6  0.085   \n",
       "38         3  134967276  134967276      1      *   74    25  0.253   \n",
       "39         3  140401955  140401955      1      *   40    18  0.310   \n",
       "40         3  151012419  151012419      1      *   30     7  0.189   \n",
       "41         3  159703343  159703343      1      *   56     7  0.111   \n",
       "42         3  178916930  178916930      1      *   19     4  0.174   \n",
       "43         4    1737062    1737062      1      *   71     4  0.053   \n",
       "44         4    1737109    1737109      1      *   46     5  0.098   \n",
       "45         4   73933754   73933754      1      *   21     3  0.125   \n",
       "46         4   96123981   96123981      1      *   59     3  0.048   \n",
       "47         4  123745305  123745305      1      *  107    12  0.101   \n",
       "48         4  151772031  151772031      1      *   22     6  0.214   \n",
       "49         4  156764877  156764877      1      *   72     4  0.053   \n",
       "50         4  158074147  158074147      1      *   38     5  0.116   \n",
       "51         5   38481688   38481688      1      *   47    13  0.217   \n",
       "52         5   50084445   50084445      1      *   36    11  0.234   \n",
       "53         5   99921931   99921931      1      *   22     5  0.185   \n",
       "54         5  126769146  126769146      1      *   54     3  0.053   \n",
       "55         5  129520330  129520330      1      *   56     9  0.138   \n",
       "56         5  132005592  132005592      1      *   24     8  0.250   \n",
       "57         5  140536977  140536977      1      *   12     8  0.400   \n",
       "58         5  141027663  141027663      1      *   17     3  0.150   \n",
       "59         5  154296729  154296729      1      *   41    20  0.328   \n",
       "60         5  177612983  177612983      1      *   79    10  0.112   \n",
       "61         6   31431757   31431757      1      *   56     4  0.067   \n",
       "62         6   44311563   44311563      1      *  108    21  0.163   \n",
       "63         6   51491959   51491959      1      *   40    14  0.259   \n",
       "64         6   74001806   74001806      1      *   30     6  0.167   \n",
       "65         6  118801734  118801734      1      *   18    11  0.379   \n",
       "66         6  130497280  130497280      1      *   47     4  0.078   \n",
       "67         6  136403705  136403705      1      *   39    12  0.235   \n",
       "68         6  139598024  139598024      1      *   22     4  0.154   \n",
       "69         6  160831788  160831788      1      *   61    15  0.197   \n",
       "70         7   12399204   12399204      1      *   61     3  0.047   \n",
       "71         7   31746856   31746856      1      *   54    18  0.250   \n",
       "72         7   36763672   36763672      1      *   43     3  0.065   \n",
       "73         7   37272689   37272689      1      *   42    12  0.222   \n",
       "74         7   43519218   43519218      1      *   74    19  0.204   \n",
       "75         7   64439575   64439575      1      *   43    11  0.204   \n",
       "76         7   72756743   72756743      1      *   52     3  0.055   \n",
       "77         7   87083892   87083892      1      *   40     5  0.111   \n",
       "78         7  136978168  136978168      1      *   93     6  0.061   \n",
       "79         7  137015603  137015603      1      *   53     3  0.054   \n",
       "80         7  137092658  137092658      1      *   80    18  0.184   \n",
       "81         7  150164063  150164063      1      *   76    20  0.208   \n",
       "82         8   23225607   23225607      1      *   66     4  0.057   \n",
       "83         8   69026619   69026619      1      *   18     7  0.280   \n",
       "84         8   87671840   87671840      1      *   77    19  0.198   \n",
       "85         8   97514016   97514016      1      *   23     3  0.115   \n",
       "86         8  109455863  109455863      1      *   29     3  0.094   \n",
       "87         9   26923301   26923301      1      *   74     4  0.051   \n",
       "88         9   27103306   27103306      1      *   53    11  0.172   \n",
       "89         9   73000586   73000586      1      *   34    13  0.277   \n",
       "90         9  110087166  110087166      1      *   21     5  0.192   \n",
       "91         9  116731456  116731456      1      *   58     6  0.094   \n",
       "92         9  130506868  130506868      1      *   30     6  0.167   \n",
       "93         9  136907079  136907079      1      *   79     4  0.048   \n",
       "94         9  137705778  137705778      1      *  113    12  0.096   \n",
       "95         9  137711921  137711921      1      *   44    12  0.214   \n",
       "96        10   33506179   33506179      1      *   72     4  0.053   \n",
       "97        10   33561788   33561788      1      *   54     6  0.100   \n",
       "98        10   50339754   50339754      1      *   60     3  0.048   \n",
       "99        10   71390392   71390392      1      *   26     3  0.103   \n",
       "100       10   74671439   74671439      1      *   80    16  0.167   \n",
       "101       10   75530013   75530013      1      *   52     4  0.071   \n",
       "102       10  105912432  105912432      1      *   63    15  0.192   \n",
       "103       10  115958855  115958855      1      *   45     3  0.062   \n",
       "104       10  115971806  115971806      1      *   22     7  0.241   \n",
       "105       10  118220506  118220506      1      *   62    18  0.225   \n",
       "106       10  124459112  124459112      1      *   42    11  0.208   \n",
       "107       11    7111667    7111667      1      *   32    13  0.289   \n",
       "108       11   50368338   50368338      1      *   48     4  0.077   \n",
       "109       11   55111650   55111650      1      *   16     8  0.333   \n",
       "110       11   55999897   55999897      1      *  102     7  0.064   \n",
       "111       11   56185159   56185159      1      *   60     3  0.048   \n",
       "112       11   69514109   69514109      1      *   54     6  0.100   \n",
       "113       11   74408224   74408224      1      *   33     3  0.083   \n",
       "114       11  102815030  102815030      1      *   21    16  0.432   \n",
       "115       11  124743628  124743628      1      *   64     4  0.059   \n",
       "116       12     922751     922751      1      *   28     7  0.200   \n",
       "117       12    4463042    4463042      1      *   36     3  0.077   \n",
       "118       12    9211480    9211480      1      *   39    12  0.235   \n",
       "119       12    9310990    9310990      1      *   80     5  0.059   \n",
       "120       12   10041301   10041301      1      *   44    16  0.267   \n",
       "121       12   14650757   14650757      1      *   40    14  0.259   \n",
       "122       12   25398284   25398284      1      *   37    11  0.229   \n",
       "123       12   92537555   92537555      1      *   32    14  0.304   \n",
       "124       12  129822166  129822166      1      *   32     8  0.200   \n",
       "125       13   21205259   21205259      1      *   63     5  0.074   \n",
       "126       13   53617212   53617212      1      *   82     4  0.047   \n",
       "127       13   96624810   96624810      1      *   96     4  0.040   \n",
       "128       13  102375156  102375156      1      *  284    13  0.044   \n",
       "129       13  108955694  108955694      1      *   28    53  0.654   \n",
       "130       14   21144748   21144748      1      *   58     3  0.049   \n",
       "131       14   34411365   34411365      1      *   82     4  0.047   \n",
       "132       14   38014889   38014889      1      *   38     5  0.116   \n",
       "133       14   51443360   51443360      1      *   49     4  0.075   \n",
       "134       14   52520704   52520704      1      *   52     3  0.055   \n",
       "135       14   86090050   86090050      1      *   15     8  0.348   \n",
       "136       14   96792127   96792127      1      *   49    21  0.300   \n",
       "137       14  106318585  106318585      1      *   20    14  0.412   \n",
       "138       15   23811592   23811592      1      *   55     4  0.068   \n",
       "139       15   25936873   25936873      1      *   23    10  0.303   \n",
       "140       15   25936905   25936905      1      *   31    16  0.340   \n",
       "141       15   42041712   42041712      1      *   22     4  0.154   \n",
       "142       15   48063999   48063999      1      *   73     4  0.052   \n",
       "143       15   79339155   79339155      1      *   28    16  0.364   \n",
       "144       16   23407432   23407432      1      *   36     5  0.122   \n",
       "145       16   28855522   28855522      1      *   57     7  0.109   \n",
       "146       16   46651564   46651564      1      *   51     4  0.073   \n",
       "147       16   56441862   56441862      1      *   43     4  0.085   \n",
       "148       16   56533893   56533893      1      *   38     5  0.116   \n",
       "149       16   75252870   75252870      1      *   36    17  0.321   \n",
       "150       16   81929553   81929553      1      *   53     6  0.102   \n",
       "151       16   87742778   87742778      1      *   31    15  0.326   \n",
       "152       17    6381939    6381939      1      *   36     3  0.077   \n",
       "153       17    6981397    6981397      1      *   36     3  0.077   \n",
       "154       17    7578260    7578260      1      *   37    15  0.288   \n",
       "155       17    8215534    8215534      1      *   27     5  0.156   \n",
       "156       17   38950272   38950272      1      *   31     4  0.114   \n",
       "157       17   38991518   38991518      1      *   19     3  0.136   \n",
       "158       17   39593768   39593768      1      *   29     3  0.094   \n",
       "159       17   39776854   39776854      1      *   12     3  0.200   \n",
       "160       17   43196366   43196366      1      *   76    15  0.165   \n",
       "161       17   44128125   44128125      1      *   50     4  0.074   \n",
       "162       17   44248848   44248848      1      *   66     4  0.057   \n",
       "163       17   57743833   57743833      1      *   22     8  0.267   \n",
       "164       17   58888380   58888380      1      *   56     5  0.082   \n",
       "165       17   61685187   61685187      1      *   73     4  0.052   \n",
       "166       17   72369850   72369850      1      *   27     8  0.229   \n",
       "167       17   74382542   74382542      1      *   48    15  0.238   \n",
       "168       17   76130947   76130947      1      *   44     3  0.064   \n",
       "169       17   78071171   78071171      1      *   40     8  0.167   \n",
       "170       18   61305135   61305135      1      *  109     5  0.044   \n",
       "171       19    2041032    2041032      1      *   31     4  0.114   \n",
       "172       19    2431488    2431488      1      *   33    14  0.298   \n",
       "173       19    3586543    3586543      1      *   69     5  0.068   \n",
       "174       19    6400058    6400058      1      *   48     9  0.158   \n",
       "175       19    8654245    8654245      1      *   32    13  0.289   \n",
       "176       19   15378338   15378338      1      *   47     4  0.078   \n",
       "177       19   19656563   19656563      1      *   75     9  0.107   \n",
       "178       19   35616086   35616086      1      *   28     4  0.125   \n",
       "179       19   41825546   41825546      1      *   68    14  0.171   \n",
       "180       19   46846056   46846056      1      *   46     5  0.098   \n",
       "181       19   49227711   49227711      1      *   39    15  0.278   \n",
       "182       19   51323232   51323232      1      *   53     3  0.054   \n",
       "183       20     309751     309751      1      *   32    16  0.333   \n",
       "184       20    5560725    5560725      1      *   37     4  0.098   \n",
       "185       20   33586203   33586203      1      *   82     7  0.079   \n",
       "186       21   28307024   28307024      1      *   37     6  0.140   \n",
       "187       21   32525074   32525074      1      *   51     5  0.089   \n",
       "188       21   33847179   33847179      1      *   45    37  0.451   \n",
       "189       21   41452034   41452034      1      *   67     5  0.069   \n",
       "190       21   45675904   45675904      1      *   67     4  0.056   \n",
       "191       22   20819078   20819078      1      *   33    10  0.233   \n",
       "192       22   37499386   37499386      1      *   46     3  0.061   \n",
       "193       22   47274638   47274638      1      *   46     4  0.080   \n",
       "194       22   49947515   49947515      1      *   31    10  0.244   \n",
       "195        X    7252077    7252077      1      *   26    14  0.350   \n",
       "196        X   27998117   27998117      1      *   54     3  0.053   \n",
       "197        X   50122932   50122932      1      *   33     3  0.083   \n",
       "198        X   50212273   50212273      1      *   31     3  0.088   \n",
       "199        X   77243944   77243944      1      *   56     3  0.051   \n",
       "200        X   78216024   78216024      1      *   48     5  0.094   \n",
       "201        X  117810737  117810737      1      *   72     4  0.053   \n",
       "202        X  151815441  151815441      1      *   43     4  0.085   \n",
       "203        Y   15409597   15409597      1      *   31    13  0.295   \n",
       "204        Y   15409751   15409751      1      *   27     5  0.156   \n",
       "205        4  173930335  173930335      1      *   27    10  0.270   \n",
       "206        7   36124231   36124231      1      *   44     5  0.102   \n",
       "207       10  124749672  124749674      3      *   47    10  0.175   \n",
       "208        X   54282134   54282135      2      *   23    19  0.452   \n",
       "209        1   97700589   97700589      1      *   14     3  0.210   \n",
       "210        1  151679963  151679963      1      *   29     3  0.116   \n",
       "211        1  152280084  152280084      1      *  168    25  0.133   \n",
       "212        1  157660351  157660351      1      *   21     6  0.241   \n",
       "213        1  160771720  160771720      1      *   30     6  0.164   \n",
       "214        1  166245976  166245976      1      *   93    21  0.189   \n",
       "215        2   97882963   97882963      1      *   70    17  0.202   \n",
       "216        3    7620139    7620139      1      *   37     5  0.136   \n",
       "217        3   10311939   10311939      1      *   67     4  0.067   \n",
       "218        3   48731921   48731921      1      *   19     3  0.167   \n",
       "219        4   85564148   85564148      1      *   36     3  0.097   \n",
       "220        5  126776623  126776623      1      *   58     6  0.106   \n",
       "221        5  137525311  137525311      1      *   31     9  0.237   \n",
       "222        5  180037317  180037317      1      *   24     3  0.138   \n",
       "223        7   23454979   23454979      1      *  126     8  0.066   \n",
       "224        7   36396358   36396358      1      *   27     4  0.153   \n",
       "225        7  100244750  100244750      1      *   24     4  0.166   \n",
       "226        7  137025901  137025901      1      *   23     4  0.158   \n",
       "227        7  158550099  158550099      1      *   31     7  0.199   \n",
       "228        8  125551397  125551397      1      *   45    10  0.192   \n",
       "229        9   88464268   88464268      1      *   43     7  0.153   \n",
       "230        9  107332395  107332395      1      *   44    15  0.262   \n",
       "231       10    7780723    7780723      1      *   24    10  0.305   \n",
       "232       10   48390042   48390042      1      *   16     5  0.260   \n",
       "233       10  134004239  134004239      1      *   67     6  0.093   \n",
       "234       11   26587352   26587352      1      *   34     3  0.103   \n",
       "235       11   60885630   60885630      1      *   44     6  0.130   \n",
       "236       11   67203501   67203501      1      *   34     5  0.146   \n",
       "237       12   40085906   40085906      1      *   28     4  0.146   \n",
       "238       12  131649643  131649643      1      *   40     4  0.109   \n",
       "239       14   74976089   74976089      1      *   44     3  0.081   \n",
       "240       14   94912977   94912977      1      *   33     3  0.105   \n",
       "241       14  106610235  106610235      1      *   43     4  0.101   \n",
       "242       15   41849496   41849496      1      *   29    19  0.400   \n",
       "243       15   45415231   45415231      1      *   49     4  0.090   \n",
       "244       15   56961125   56961125      1      *   14     8  0.373   \n",
       "245       16   70553596   70553596      1      *   42     5  0.121   \n",
       "246       16   88809368   88809368      1      *   51     4  0.088   \n",
       "247       17    9057860    9057860      1      *   46     5  0.112   \n",
       "248       17   43487574   43487574      1      *   38     3  0.094   \n",
       "249       17   60865795   60865795      1      *   19     8  0.311   \n",
       "250       19    1531885    1531885      1      *   53     6  0.114   \n",
       "251       19    3823237    3823237      1      *   66     4  0.069   \n",
       "252       19   12541532   12541532      1      *   78     6  0.081   \n",
       "253       19   12541547   12541547      1      *   80     6  0.080   \n",
       "254       19   49950356   49950356      1      *   44     4  0.098   \n",
       "255       19   52705184   52705184      1      *   47     4  0.093   \n",
       "256       22   17600293   17600293      1      *   29     3  0.117   \n",
       "257       22   46684528   46684528      1      *   19     3  0.167   \n",
       "258        X   48123139   48123139      1      *  113     8  0.072   \n",
       "259        X   77268502   77268502      1      *   59     4  0.076   \n",
       "260        X  107924910  107924910      1      *   62     5  0.085   \n",
       "261        2  238303417  238303418      2      *   77    22  0.228   \n",
       "\n",
       "                                           Consequence    IMPACT  \\\n",
       "0                                     missense_variant  MODERATE   \n",
       "1                   non_coding_transcript_exon_variant  MODIFIER   \n",
       "2                                   synonymous_variant       LOW   \n",
       "3                                     missense_variant  MODERATE   \n",
       "4                                       intron_variant  MODIFIER   \n",
       "5                                     missense_variant  MODERATE   \n",
       "6                                     missense_variant  MODERATE   \n",
       "7                                     missense_variant  MODERATE   \n",
       "8                                     missense_variant  MODERATE   \n",
       "9                                       intron_variant  MODIFIER   \n",
       "10                                 3_prime_UTR_variant  MODIFIER   \n",
       "11                                      intron_variant  MODIFIER   \n",
       "12                                 3_prime_UTR_variant  MODIFIER   \n",
       "13                                    missense_variant  MODERATE   \n",
       "14                                    missense_variant  MODERATE   \n",
       "15                                    missense_variant  MODERATE   \n",
       "16                                 3_prime_UTR_variant  MODIFIER   \n",
       "17                                 3_prime_UTR_variant  MODIFIER   \n",
       "18                                    missense_variant  MODERATE   \n",
       "19                                 3_prime_UTR_variant  MODIFIER   \n",
       "20                                    missense_variant  MODERATE   \n",
       "21                                  synonymous_variant       LOW   \n",
       "22                                  synonymous_variant       LOW   \n",
       "23                                  synonymous_variant       LOW   \n",
       "24                                    missense_variant  MODERATE   \n",
       "25                                  synonymous_variant       LOW   \n",
       "26                                      intron_variant  MODIFIER   \n",
       "27                                    missense_variant  MODERATE   \n",
       "28                                  synonymous_variant       LOW   \n",
       "29                                    missense_variant  MODERATE   \n",
       "30                                  synonymous_variant       LOW   \n",
       "31                                    missense_variant  MODERATE   \n",
       "32                splice_region_variant&intron_variant       LOW   \n",
       "33                                 3_prime_UTR_variant  MODIFIER   \n",
       "34                                    missense_variant  MODERATE   \n",
       "35                                      intron_variant  MODIFIER   \n",
       "36                                      intron_variant  MODIFIER   \n",
       "37                               upstream_gene_variant  MODIFIER   \n",
       "38                                    missense_variant  MODERATE   \n",
       "39                                  synonymous_variant       LOW   \n",
       "40                                  synonymous_variant       LOW   \n",
       "41                               upstream_gene_variant  MODIFIER   \n",
       "42                                    missense_variant  MODERATE   \n",
       "43                                      intron_variant  MODIFIER   \n",
       "44                                      intron_variant  MODIFIER   \n",
       "45                                  synonymous_variant       LOW   \n",
       "46                                  synonymous_variant       LOW   \n",
       "47                               upstream_gene_variant  MODIFIER   \n",
       "48                                  synonymous_variant       LOW   \n",
       "49                                    missense_variant  MODERATE   \n",
       "50                                  synonymous_variant       LOW   \n",
       "51                                 3_prime_UTR_variant  MODIFIER   \n",
       "52                                      intron_variant  MODIFIER   \n",
       "53                                 3_prime_UTR_variant  MODIFIER   \n",
       "54                                  synonymous_variant       LOW   \n",
       "55                                    missense_variant  MODERATE   \n",
       "56                               upstream_gene_variant  MODIFIER   \n",
       "57                             downstream_gene_variant  MODIFIER   \n",
       "58            splice_region_variant&synonymous_variant       LOW   \n",
       "59                                    missense_variant  MODERATE   \n",
       "60                  non_coding_transcript_exon_variant  MODIFIER   \n",
       "61                  non_coding_transcript_exon_variant  MODIFIER   \n",
       "62                                      intron_variant  MODIFIER   \n",
       "63                                      intron_variant  MODIFIER   \n",
       "64                                      intron_variant  MODIFIER   \n",
       "65                                      intron_variant  MODIFIER   \n",
       "66                                      intron_variant  MODIFIER   \n",
       "67                                      intron_variant  MODIFIER   \n",
       "68                                  synonymous_variant       LOW   \n",
       "69                                  synonymous_variant       LOW   \n",
       "70              missense_variant&splice_region_variant  MODERATE   \n",
       "71                                    missense_variant  MODERATE   \n",
       "72                                    missense_variant  MODERATE   \n",
       "73                                      intron_variant  MODIFIER   \n",
       "74                                    missense_variant  MODERATE   \n",
       "75                                    missense_variant  MODERATE   \n",
       "76                                      intron_variant  MODIFIER   \n",
       "77                                  synonymous_variant       LOW   \n",
       "78                                      intron_variant  MODIFIER   \n",
       "79                                      intron_variant  MODIFIER   \n",
       "80                                  synonymous_variant       LOW   \n",
       "81                                    missense_variant  MODERATE   \n",
       "82                                  synonymous_variant       LOW   \n",
       "83                                      intron_variant  MODIFIER   \n",
       "84                                      intron_variant  MODIFIER   \n",
       "85                                      intron_variant  MODIFIER   \n",
       "86                                 5_prime_UTR_variant  MODIFIER   \n",
       "87                                    missense_variant  MODERATE   \n",
       "88        intron_variant&non_coding_transcript_variant  MODIFIER   \n",
       "89                                 3_prime_UTR_variant  MODIFIER   \n",
       "90                                    missense_variant  MODERATE   \n",
       "91                                      intron_variant  MODIFIER   \n",
       "92                                    missense_variant  MODERATE   \n",
       "93                splice_region_variant&intron_variant       LOW   \n",
       "94                                      intron_variant  MODIFIER   \n",
       "95                                      intron_variant  MODIFIER   \n",
       "96                                      intron_variant  MODIFIER   \n",
       "97                                      intron_variant  MODIFIER   \n",
       "98                                    missense_variant  MODERATE   \n",
       "99                                      intron_variant  MODIFIER   \n",
       "100                                     intron_variant  MODIFIER   \n",
       "101                                     intron_variant  MODIFIER   \n",
       "102                                   missense_variant  MODERATE   \n",
       "103                                     intron_variant  MODIFIER   \n",
       "104                                     intron_variant  MODIFIER   \n",
       "105                                 synonymous_variant       LOW   \n",
       "106                                     intron_variant  MODIFIER   \n",
       "107                                3_prime_UTR_variant  MODIFIER   \n",
       "108                 non_coding_transcript_exon_variant  MODIFIER   \n",
       "109                                   missense_variant  MODERATE   \n",
       "110                                 synonymous_variant       LOW   \n",
       "111                                   missense_variant  MODERATE   \n",
       "112                                   missense_variant  MODERATE   \n",
       "113                                     intron_variant  MODIFIER   \n",
       "114                                   missense_variant  MODERATE   \n",
       "115                                   missense_variant  MODERATE   \n",
       "116                                     intron_variant  MODIFIER   \n",
       "117                            downstream_gene_variant  MODIFIER   \n",
       "118  splice_region_variant&intron_variant&non_codin...       LOW   \n",
       "119                                     intron_variant  MODIFIER   \n",
       "120                                     intron_variant  MODIFIER   \n",
       "121                                   missense_variant  MODERATE   \n",
       "122                                   missense_variant  MODERATE   \n",
       "123                                3_prime_UTR_variant  MODIFIER   \n",
       "124                                     intron_variant  MODIFIER   \n",
       "125                                     intron_variant  MODIFIER   \n",
       "126                                     intron_variant  MODIFIER   \n",
       "127                                     intron_variant  MODIFIER   \n",
       "128                                3_prime_UTR_variant  MODIFIER   \n",
       "129                                   missense_variant  MODERATE   \n",
       "130                              upstream_gene_variant  MODIFIER   \n",
       "131                                     intron_variant  MODIFIER   \n",
       "132                                     intron_variant  MODIFIER   \n",
       "133                                3_prime_UTR_variant  MODIFIER   \n",
       "134                                     intron_variant  MODIFIER   \n",
       "135                                3_prime_UTR_variant  MODIFIER   \n",
       "136                                        stop_gained      HIGH   \n",
       "137                            downstream_gene_variant  MODIFIER   \n",
       "138                                 synonymous_variant       LOW   \n",
       "139                                        stop_gained      HIGH   \n",
       "140                                   missense_variant  MODERATE   \n",
       "141                                 synonymous_variant       LOW   \n",
       "142                                3_prime_UTR_variant  MODIFIER   \n",
       "143                                   missense_variant  MODERATE   \n",
       "144                                     intron_variant  MODIFIER   \n",
       "145                                     intron_variant  MODIFIER   \n",
       "146                                 synonymous_variant       LOW   \n",
       "147                                     intron_variant  MODIFIER   \n",
       "148                                     intron_variant  MODIFIER   \n",
       "149                              upstream_gene_variant  MODIFIER   \n",
       "150                                     intron_variant  MODIFIER   \n",
       "151                                     intron_variant  MODIFIER   \n",
       "152                                 synonymous_variant       LOW   \n",
       "153                                   missense_variant  MODERATE   \n",
       "154                                   missense_variant  MODERATE   \n",
       "155                                 synonymous_variant       LOW   \n",
       "156                                 synonymous_variant       LOW   \n",
       "157                                 synonymous_variant       LOW   \n",
       "158                                   missense_variant  MODERATE   \n",
       "159                                     intron_variant  MODIFIER   \n",
       "160                                 synonymous_variant       LOW   \n",
       "161                                     intron_variant  MODIFIER   \n",
       "162                                   missense_variant  MODERATE   \n",
       "163               splice_region_variant&intron_variant       LOW   \n",
       "164                               mature_miRNA_variant  MODIFIER   \n",
       "165                                   missense_variant  MODERATE   \n",
       "166                            downstream_gene_variant  MODIFIER   \n",
       "167                            downstream_gene_variant  MODIFIER   \n",
       "168               splice_region_variant&intron_variant       LOW   \n",
       "169                              upstream_gene_variant  MODIFIER   \n",
       "170                                   missense_variant  MODERATE   \n",
       "171               splice_region_variant&intron_variant       LOW   \n",
       "172                              upstream_gene_variant  MODIFIER   \n",
       "173                                 synonymous_variant       LOW   \n",
       "174       intron_variant&non_coding_transcript_variant  MODIFIER   \n",
       "175                                   missense_variant  MODERATE   \n",
       "176                                   missense_variant  MODERATE   \n",
       "177                                   missense_variant  MODERATE   \n",
       "178                                3_prime_UTR_variant  MODIFIER   \n",
       "179                                 synonymous_variant       LOW   \n",
       "180                              upstream_gene_variant  MODIFIER   \n",
       "181                                 synonymous_variant       LOW   \n",
       "182                                   missense_variant  MODERATE   \n",
       "183                                3_prime_UTR_variant  MODIFIER   \n",
       "184                                 synonymous_variant       LOW   \n",
       "185                            downstream_gene_variant  MODIFIER   \n",
       "186                                   missense_variant  MODERATE   \n",
       "187                                   missense_variant  MODERATE   \n",
       "188                                     intron_variant  MODIFIER   \n",
       "189                                     intron_variant  MODIFIER   \n",
       "190                                     intron_variant  MODIFIER   \n",
       "191                                     intron_variant  MODIFIER   \n",
       "192                                 synonymous_variant       LOW   \n",
       "193                                     intron_variant  MODIFIER   \n",
       "194                 non_coding_transcript_exon_variant  MODIFIER   \n",
       "195                                   missense_variant  MODERATE   \n",
       "196                                   missense_variant  MODERATE   \n",
       "197       intron_variant&non_coding_transcript_variant  MODIFIER   \n",
       "198       intron_variant&non_coding_transcript_variant  MODIFIER   \n",
       "199                                 synonymous_variant       LOW   \n",
       "200                                   missense_variant  MODERATE   \n",
       "201                                     intron_variant  MODIFIER   \n",
       "202                                 synonymous_variant       LOW   \n",
       "203                                   missense_variant  MODERATE   \n",
       "204                                     intron_variant  MODIFIER   \n",
       "205                                 frameshift_variant      HIGH   \n",
       "206                                 frameshift_variant      HIGH   \n",
       "207                            downstream_gene_variant  MODIFIER   \n",
       "208                                     intron_variant  MODIFIER   \n",
       "209                                     intron_variant  MODIFIER   \n",
       "210                                     intron_variant  MODIFIER   \n",
       "211                                 synonymous_variant       LOW   \n",
       "212                                     intron_variant  MODIFIER   \n",
       "213                                     intron_variant  MODIFIER   \n",
       "214                 non_coding_transcript_exon_variant  MODIFIER   \n",
       "215                                     intron_variant  MODIFIER   \n",
       "216                                        stop_gained      HIGH   \n",
       "217                                   missense_variant  MODERATE   \n",
       "218                                     intron_variant  MODIFIER   \n",
       "219                                     intron_variant  MODIFIER   \n",
       "220                                     intron_variant  MODIFIER   \n",
       "221                                     intron_variant  MODIFIER   \n",
       "222                                     intron_variant  MODIFIER   \n",
       "223                                     intron_variant  MODIFIER   \n",
       "224                                     intron_variant  MODIFIER   \n",
       "225                                     intron_variant  MODIFIER   \n",
       "226                                     intron_variant  MODIFIER   \n",
       "227                                     intron_variant  MODIFIER   \n",
       "228                                5_prime_UTR_variant  MODIFIER   \n",
       "229                 non_coding_transcript_exon_variant  MODIFIER   \n",
       "230                                   missense_variant  MODERATE   \n",
       "231                               splice_donor_variant      HIGH   \n",
       "232                                   missense_variant  MODERATE   \n",
       "233                                     intron_variant  MODIFIER   \n",
       "234                                     intron_variant  MODIFIER   \n",
       "235                                     intron_variant  MODIFIER   \n",
       "236                            downstream_gene_variant  MODIFIER   \n",
       "237                                   missense_variant  MODERATE   \n",
       "238                 non_coding_transcript_exon_variant  MODIFIER   \n",
       "239                                     intron_variant  MODIFIER   \n",
       "240                                     intron_variant  MODIFIER   \n",
       "241                            downstream_gene_variant  MODIFIER   \n",
       "242                              upstream_gene_variant  MODIFIER   \n",
       "243                                5_prime_UTR_variant  MODIFIER   \n",
       "244                                 synonymous_variant       LOW   \n",
       "245                              upstream_gene_variant  MODIFIER   \n",
       "246                                     intron_variant  MODIFIER   \n",
       "247                                     intron_variant  MODIFIER   \n",
       "248                                     intron_variant  MODIFIER   \n",
       "249                                     intron_variant  MODIFIER   \n",
       "250               splice_region_variant&intron_variant       LOW   \n",
       "251               splice_region_variant&intron_variant       LOW   \n",
       "252                                   missense_variant  MODERATE   \n",
       "253                                   missense_variant  MODERATE   \n",
       "254           splice_region_variant&synonymous_variant       LOW   \n",
       "255                                     intron_variant  MODIFIER   \n",
       "256                            downstream_gene_variant  MODIFIER   \n",
       "257                                     intron_variant  MODIFIER   \n",
       "258                                     intron_variant  MODIFIER   \n",
       "259                                   missense_variant  MODERATE   \n",
       "260                                     intron_variant  MODIFIER   \n",
       "261                                   missense_variant  MODERATE   \n",
       "\n",
       "            SYMBOL                                HGVSc  \\\n",
       "0           TAS1R3           ENST00000339381.5:c.468C>A   \n",
       "1    RP11-169K16.9          ENST00000317122.1:n.1606A>G   \n",
       "2            GRIK3          ENST00000373091.3:c.1002C>T   \n",
       "3             GBP4          ENST00000355754.6:c.1217A>G   \n",
       "4            ABCA4       ENST00000370225.3:c.4667+30T>C   \n",
       "5        HIST2H2AB           ENST00000331128.3:c.134G>T   \n",
       "6            KCNN3           ENST00000271915.4:c.538T>G   \n",
       "7            SPTA1          ENST00000368147.4:c.3167G>A   \n",
       "8            SPTA1          ENST00000368147.4:c.2246T>C   \n",
       "9         C1orf112      ENST00000286031.6:c.702+4210A>T   \n",
       "10          KIFAP3          ENST00000361580.2:c.*274G>T   \n",
       "11          SEC16B       ENST00000308284.6:c.2725-35C>T   \n",
       "12          PLXNA2          ENST00000367033.3:c.*466T>G   \n",
       "13          OR11L1           ENST00000355784.2:c.443C>T   \n",
       "14             CAD          ENST00000264705.4:c.3733G>C   \n",
       "15           BIRC6          ENST00000421745.2:c.5489G>T   \n",
       "16          B3GNT2          ENST00000301998.4:c.*761G>T   \n",
       "17         SERTAD2          ENST00000313349.3:c.*543T>A   \n",
       "18           CD207           ENST00000410009.3:c.164C>T   \n",
       "19           DGUOK           ENST00000264093.4:c.*13A>T   \n",
       "20            CD8B           ENST00000331469.2:c.172C>T   \n",
       "21          THSD7B           ENST00000272643.3:c.288T>C   \n",
       "22          THSD7B           ENST00000272643.3:c.957C>T   \n",
       "23           SCN3A          ENST00000283254.7:c.2037G>A   \n",
       "24           SCN7A          ENST00000409855.1:c.3910G>A   \n",
       "25           STK39          ENST00000355999.4:c.1296C>T   \n",
       "26         FASTKD1       ENST00000453153.2:c.1701+25A>G   \n",
       "27         DNAJC10          ENST00000264065.7:c.1982G>C   \n",
       "28           MARS2           ENST00000282276.6:c.600G>A   \n",
       "29           SGOL2           ENST00000357799.4:c.575G>T   \n",
       "30            TNS1           ENST00000171887.4:c.153T>C   \n",
       "31           NYAP2           ENST00000272907.6:c.271C>T   \n",
       "32           ITPR1        ENST00000302640.8:c.5681-5T>C   \n",
       "33           WDR48          ENST00000302313.5:c.*355G>T   \n",
       "34           TCAIM           ENST00000342649.4:c.818G>T   \n",
       "35           SLMAP       ENST00000295951.3:c.1199-14A>G   \n",
       "36            FHIT    ENST00000468189.1:c.-110-90684T>C   \n",
       "37      AC108696.1                                  NaN   \n",
       "38           EPHB1          ENST00000398015.3:c.2615C>T   \n",
       "39          TRIM42           ENST00000286349.3:c.993C>T   \n",
       "40           GPR87           ENST00000260843.4:c.615G>A   \n",
       "41           IL12A                                  NaN   \n",
       "42          PIK3CA           ENST00000263967.3:c.317G>T   \n",
       "43           TACC3       ENST00000313288.4:c.1644+10A>G   \n",
       "44           TACC3       ENST00000313288.4:c.1644+57G>A   \n",
       "45           COX18           ENST00000295890.4:c.426A>G   \n",
       "46           UNC5C          ENST00000453304.1:c.2037G>A   \n",
       "47            FGF2                                  NaN   \n",
       "48            LRBA          ENST00000357115.3:c.3849A>C   \n",
       "49           ASIC5           ENST00000537611.2:c.817T>C   \n",
       "50            GLRB          ENST00000264428.4:c.1182T>C   \n",
       "51            LIFR            ENST00000263409.4:c.*9C>T   \n",
       "52           PARP8        ENST00000281631.5:c.737+22A>G   \n",
       "53         FAM174A           ENST00000312637.4:c.*43A>T   \n",
       "54          MEGF10          ENST00000274473.6:c.1785T>C   \n",
       "55           CHSY3          ENST00000305031.4:c.1495G>A   \n",
       "56             IL4                                  NaN   \n",
       "57          PCDHB6                                  NaN   \n",
       "58          FCHSD1           ENST00000435817.2:c.577C>T   \n",
       "59          GEMIN5          ENST00000285873.7:c.1684A>G   \n",
       "60         GMCL1P1          ENST00000463439.1:n.1318C>T   \n",
       "61            HCP5           ENST00000414046.2:n.829A>G   \n",
       "62          SPATS1       ENST00000288390.2:c.139+592G>A   \n",
       "63           PKHD1      ENST00000371117.3:c.11666-45C>G   \n",
       "64           KHDC1        ENST00000370384.3:c.164-43T>C   \n",
       "65          CEP85L       ENST00000368488.5:c.1755-58C>A   \n",
       "66           SAMD3       ENST00000368134.2:c.655-127T>C   \n",
       "67           PDE7B      ENST00000308191.6:c.83-26164C>A   \n",
       "68           TXLNB           ENST00000358430.3:c.459C>T   \n",
       "69         SLC22A3           ENST00000275300.2:c.885G>C   \n",
       "70            VWDE          ENST00000275358.3:c.3274A>T   \n",
       "71         PPP1R17           ENST00000342032.3:c.427G>A   \n",
       "72            AOAH            ENST00000258749.5:c.82G>A   \n",
       "73           ELMO1        ENST00000310758.4:c.549+11C>A   \n",
       "74           HECW1          ENST00000395891.2:c.3109G>A   \n",
       "75          ZNF117           ENST00000282869.6:c.374C>T   \n",
       "76           FKBP6        ENST00000252037.4:c.894-64G>T   \n",
       "77           ABCB4           ENST00000265723.4:c.303G>A   \n",
       "78             PTN      ENST00000348225.2:c.-1-38447A>G   \n",
       "79             PTN      ENST00000348225.2:c.-2+12582T>A   \n",
       "80            DGKI          ENST00000288490.5:c.2907C>T   \n",
       "81          GIMAP8           ENST00000307271.3:c.277A>T   \n",
       "82           LOXL2           ENST00000389131.3:c.258C>T   \n",
       "83           PREX2     ENST00000288368.4:c.3147-1369G>A   \n",
       "84           CNGB3      ENST00000320005.5:c.853-5550G>C   \n",
       "85            SDC2       ENST00000302190.4:c.60+7457T>C   \n",
       "86            EMC2           ENST00000220853.3:c.-25C>T   \n",
       "87            PLAA           ENST00000397292.3:c.914C>T   \n",
       "88    RP11-56F10.3       ENST00000522960.1:n.447+232A>C   \n",
       "89            KLF9         ENST00000377126.2:c.*2106G>C   \n",
       "90          RAD23B           ENST00000358015.3:c.997C>G   \n",
       "91          ZNF618         ENST00000288466.7:c.77+16C>T   \n",
       "92          SH2D3C          ENST00000314830.8:c.1775G>A   \n",
       "93            BRD3        ENST00000303407.7:c.1216-6C>T   \n",
       "94          COL5A1       ENST00000371817.3:c.3853-51C>T   \n",
       "95          COL5A1       ENST00000371817.3:c.4447-41G>T   \n",
       "96            NRP1     ENST00000265371.4:c.1283-3534G>A   \n",
       "97            NRP1      ENST00000265371.4:c.249-2004G>A   \n",
       "98         FAM170B           ENST00000311787.5:c.756C>A   \n",
       "99        C10orf35      ENST00000373279.4:c.-104+323C>T   \n",
       "100           OIT3        ENST00000334011.5:c.668-36C>T   \n",
       "101         SEC24C       ENST00000339365.2:c.2863-25C>T   \n",
       "102          WDR96          ENST00000357060.3:c.3593A>T   \n",
       "103          TDRD1        ENST00000251864.2:c.385-77G>T   \n",
       "104          TDRD1       ENST00000251864.2:c.1831+11T>C   \n",
       "105       PNLIPRP3           ENST00000369230.3:c.594C>T   \n",
       "106      C10orf120        ENST00000329446.4:c.176+19T>G   \n",
       "107         RBMXL2          ENST00000306904.5:c.*137C>T   \n",
       "108   RP11-574M7.2           ENST00000532521.2:n.126A>G   \n",
       "109         OR4A16           ENST00000314721.2:c.974C>T   \n",
       "110          OR5T2           ENST00000313264.4:c.765C>T   \n",
       "111          OR5R1           ENST00000312253.1:c.550T>C   \n",
       "112          FGF19           ENST00000294312.3:c.572C>T   \n",
       "113         CHRDL2       ENST00000263671.5:c.1268+26G>T   \n",
       "114          MMP13          ENST00000260302.3:c.1381C>T   \n",
       "115          ROBO3          ENST00000397801.1:c.1654C>A   \n",
       "116           WNK1        ENST00000315939.6:c.760-57G>A   \n",
       "117        C12orf5                                  NaN   \n",
       "118      LINC00612         ENST00000538094.1:n.804-8T>C   \n",
       "119            PZP       ENST00000261336.2:c.3294+26C>A   \n",
       "120          KLRF2        ENST00000535540.1:c.170-31T>C   \n",
       "121         ATF7IP          ENST00000261168.4:c.3563G>A   \n",
       "122           KRAS            ENST00000256078.4:c.35G>A   \n",
       "123           BTG1          ENST00000256015.3:c.*301A>G   \n",
       "124       TMEM132D       ENST00000422113.2:c.1299+13C>T   \n",
       "125          IFT88       ENST00000319980.6:c.1413+18T>G   \n",
       "126          OLFM4        ENST00000219022.2:c.571-28A>T   \n",
       "127          UGGT2       ENST00000376747.3:c.1184+24A>C   \n",
       "128          FGF14           ENST00000376131.4:c.*25T>G   \n",
       "129       TNFSF13B           ENST00000375887.4:c.575A>C   \n",
       "130          Y_RNA                                  NaN   \n",
       "131          EGLN3      ENST00000250457.3:c.357+8237T>C   \n",
       "132         MIPOL1     ENST00000327441.7:c.1263-1221A>T   \n",
       "133          TRIM9          ENST00000298355.3:c.*649T>C   \n",
       "134           NID2       ENST00000216286.5:c.1069+34C>T   \n",
       "135          FLRT2          ENST00000330753.4:c.*209T>C   \n",
       "136          ATG2B          ENST00000359933.4:c.2296G>T   \n",
       "137           IGHM                                  NaN   \n",
       "138          MKRN3           ENST00000314520.3:c.663C>T   \n",
       "139         ATP10A          ENST00000356865.6:c.3154G>T   \n",
       "140         ATP10A          ENST00000356865.6:c.3122T>A   \n",
       "141            MGA          ENST00000219905.7:c.5907G>A   \n",
       "142         SEMA6D           ENST00000316364.5:c.*17T>A   \n",
       "143        RASGRF1           ENST00000419573.3:c.811C>T   \n",
       "144           COG7     ENST00000307149.5:c.1887+1935G>A   \n",
       "145           TUFM        ENST00000313511.3:c.922+29C>G   \n",
       "146         SHCBP1           ENST00000303383.3:c.369G>A   \n",
       "147           AMFR        ENST00000290649.5:c.666+14T>C   \n",
       "148           BBS2       ENST00000245157.5:c.1398-74A>G   \n",
       "149          CTRB1                                  NaN   \n",
       "150          PLCG2       ENST00000359376.3:c.1193+21G>T   \n",
       "151         KLHDC4       ENST00000270583.5:c.1447+93G>T   \n",
       "152        PITPNM3           ENST00000262483.8:c.705C>T   \n",
       "153        CLEC10A           ENST00000254868.4:c.103T>C   \n",
       "154           TP53           ENST00000269305.4:c.589G>A   \n",
       "155       ARHGEF15           ENST00000361926.3:c.177C>T   \n",
       "156          KRT28          ENST00000306658.7:c.1005A>G   \n",
       "157         TMEM99           ENST00000301665.3:c.750T>C   \n",
       "158          KRT38          ENST00000246646.3:c.1267T>C   \n",
       "159          KRT17       ENST00000311208.8:c.1182-44C>G   \n",
       "160          PLCD3           ENST00000322765.5:c.729C>T   \n",
       "161         KANSL1       ENST00000262419.6:c.1849-55T>G   \n",
       "162         KANSL1           ENST00000262419.6:c.662C>T   \n",
       "163           CLTC        ENST00000269122.3:c.1783-8T>C   \n",
       "164     AC079005.1            ENST00000390181.1:n.63A>G   \n",
       "165          TACO1           ENST00000258975.6:c.719A>G   \n",
       "166         GPR142                                  NaN   \n",
       "167          UBE2O                                  NaN   \n",
       "168           TMC8         ENST00000318430.5:c.988-4G>T   \n",
       "169            GAA                                  NaN   \n",
       "170       SERPINB4           ENST00000341074.5:c.991A>G   \n",
       "171          MKNK2        ENST00000250896.3:c.1110+7A>G   \n",
       "172         TIMM13                                  NaN   \n",
       "173          GIPC3           ENST00000322315.5:c.276G>A   \n",
       "174    CTB-180A7.8        ENST00000398173.3:n.204-61A>C   \n",
       "175       ADAMTS10          ENST00000270328.4:c.2039T>G   \n",
       "176           BRD4           ENST00000263377.2:c.448G>A   \n",
       "177          CILP2          ENST00000291495.5:c.3209G>T   \n",
       "178           LGI4           ENST00000310123.3:c.*11G>A   \n",
       "179         CCDC97           ENST00000269967.3:c.570G>T   \n",
       "180          PPP5C                                  NaN   \n",
       "181         RASIP1          ENST00000222145.4:c.2427G>A   \n",
       "182           KLK1           ENST00000301420.2:c.556A>G   \n",
       "183          SOX12         ENST00000342665.2:c.*2235T>C   \n",
       "184         GPCPD1           ENST00000379019.4:c.402A>G   \n",
       "185        TRPC4AP                                  NaN   \n",
       "186        ADAMTS5          ENST00000284987.5:c.1450G>A   \n",
       "187          TIAM1          ENST00000286827.3:c.3246C>A   \n",
       "188          EVA1C      ENST00000300255.2:c.634+7023A>G   \n",
       "189          DSCAM       ENST00000400454.1:c.4420+45G>A   \n",
       "190         DNMT3L        ENST00000270172.3:c.604+46A>C   \n",
       "191         KLHL22       ENST00000328879.4:c.1112+67A>C   \n",
       "192        TMPRSS6            ENST00000346753.3:c.99G>A   \n",
       "193       TBC1D22A        ENST00000337137.4:c.708+19C>T   \n",
       "194   RP1-29C18.10          ENST00000391625.2:n.1288C>T   \n",
       "195            STS          ENST00000217961.4:c.1307G>A   \n",
       "196        DCAF8L1          ENST00000441525.1:c.1335C>G   \n",
       "197           DGKK       ENST00000376025.2:n.2814+47T>A   \n",
       "198           DGKK       ENST00000376025.2:n.705+760C>A   \n",
       "199          ATP7A           ENST00000341514.6:c.327G>A   \n",
       "200         P2RY10             ENST00000171757.2:c.7A>C   \n",
       "201         DOCK11       ENST00000276202.7:c.5460+37T>A   \n",
       "202          GABRQ           ENST00000370306.2:c.339T>A   \n",
       "203            UTY          ENST00000331397.4:c.3709G>A   \n",
       "204            UTY       ENST00000331397.4:c.3578-23G>A   \n",
       "205        GALNTL6    ENST00000506823.1:c.1378_1379insT   \n",
       "206        PP13004           ENST00000381493.2:c.234dup   \n",
       "207          IKZF5                                  NaN   \n",
       "208           WNK3       ENST00000354646.2:c.2250+38del   \n",
       "209           DPYD       ENST00000370192.3:c.2300-39G>A   \n",
       "210          CELF3        ENST00000290583.4:c.772+20G>A   \n",
       "211            FLG          ENST00000368799.1:c.7278C>T   \n",
       "212          FCRL3       ENST00000368184.3:c.1412-28T>A   \n",
       "213            LY9      ENST00000263285.6:c.454+1848A>C   \n",
       "214    RP11-9L18.2           ENST00000400979.2:n.859G>A   \n",
       "215        ANKRD36       ENST00000420699.2:c.3780+21A>G   \n",
       "216           GRM7          ENST00000357716.4:c.1546C>T   \n",
       "217         TATDN2          ENST00000287652.4:c.1073C>T   \n",
       "218          IP6K2       ENST00000328631.5:c.202+602T>A   \n",
       "219           CDS1       ENST00000295887.5:c.1033-29C>A   \n",
       "220         MEGF10       ENST00000274473.6:c.2362+64T>G   \n",
       "221          CDC23       ENST00000394886.2:c.1623+11G>C   \n",
       "222           FLT4      ENST00000261937.6:c.3687-292G>A   \n",
       "223        IGF2BP3      ENST00000258729.3:c.285+3416C>T   \n",
       "224       KIAA0895       ENST00000297063.6:c.867+153C>T   \n",
       "225         ACTL6B        ENST00000160382.5:c.822-42T>G   \n",
       "226            PTN       ENST00000348225.2:c.-2+2284T>A   \n",
       "227          ESYT2     ENST00000251527.5:c.1563+2078G>T   \n",
       "228         NDUFB9           ENST00000276689.3:c.-31C>A   \n",
       "229   RP11-213G2.3          ENST00000431724.1:n.1498A>C   \n",
       "230         OR13C8           ENST00000335040.1:c.947T>C   \n",
       "231          ITIH2        ENST00000358415.4:c.2095+2T>A   \n",
       "232           RBP3           ENST00000224600.4:c.836C>T   \n",
       "233         DPYSL4         ENST00000338492.4:c.40-12T>G   \n",
       "234           ANO3     ENST00000256737.3:c.1447+6026T>A   \n",
       "235            CD5         ENST00000347785.3:c.95-17A>C   \n",
       "236        RPS6KB2                                  NaN   \n",
       "237       C12orf40          ENST00000324616.5:c.1403A>T   \n",
       "238   RP11-638F5.1            ENST00000376678.2:n.88G>A   \n",
       "239          LTBP2       ENST00000261978.4:c.3278-23G>A   \n",
       "240      SERPINA11        ENST00000334708.3:c.644-36A>G   \n",
       "241       IGHV3-15                                  NaN   \n",
       "242          TYRO3                                  NaN   \n",
       "243         DUOXA1           ENST00000267803.4:c.-19C>A   \n",
       "244        ZNF280D          ENST00000267807.7:c.1441A>C   \n",
       "245          SF3B3                                  NaN   \n",
       "246         PIEZO1       ENST00000301015.9:c.161-538C>A   \n",
       "247           NTN1     ENST00000173229.2:c.1019-8270T>C   \n",
       "248       ARHGAP27     ENST00000376922.2:c.-366-4125G>T   \n",
       "249        MARCH10        ENST00000311269.5:c.210+46C>G   \n",
       "250           PLK5         ENST00000334770.4:c.714+3A>G   \n",
       "251           ZFR2        ENST00000262961.4:c.1371+7C>T   \n",
       "252         ZNF443          ENST00000301547.5:c.1454G>T   \n",
       "253         ZNF443          ENST00000301547.5:c.1439A>C   \n",
       "254         PIH1D1           ENST00000262265.5:c.612G>C   \n",
       "255        PPP2R1A         ENST00000322088.6:c.79-13G>A   \n",
       "256         IL17RA                                  NaN   \n",
       "257          TTC38       ENST00000381031.3:c.1082+43C>T   \n",
       "258           SSX1        ENST00000376919.3:c.331-78G>A   \n",
       "259          ATP7A          ENST00000341514.6:c.2299G>C   \n",
       "260         COL4A5       ENST00000328300.6:c.4088-80G>T   \n",
       "261         COL6A3  ENST00000295550.4:c.521_522delinsAC   \n",
       "\n",
       "                                    HGVSp  \\\n",
       "0           ENSP00000344411.5:p.Phe156Leu   \n",
       "1                                     NaN   \n",
       "2           ENSP00000362183.3:p.Ser334%3D   \n",
       "3           ENSP00000359490.5:p.Lys406Arg   \n",
       "4                                     NaN   \n",
       "5            ENSP00000332790.3:p.Gly45Val   \n",
       "6           ENSP00000271915.3:p.Tyr180Asp   \n",
       "7          ENSP00000357129.4:p.Arg1056His   \n",
       "8           ENSP00000357129.4:p.Leu749Pro   \n",
       "9                                     NaN   \n",
       "10                                    NaN   \n",
       "11                                    NaN   \n",
       "12                                    NaN   \n",
       "13          ENSP00000348033.2:p.Ser148Phe   \n",
       "14         ENSP00000264705.3:p.Glu1245Gln   \n",
       "15         ENSP00000393596.2:p.Arg1830Leu   \n",
       "16                                    NaN   \n",
       "17                                    NaN   \n",
       "18           ENSP00000386378.3:p.Ala55Val   \n",
       "19                                    NaN   \n",
       "20           ENSP00000331172.2:p.Arg58Cys   \n",
       "21           ENSP00000272643.3:p.Val96%3D   \n",
       "22          ENSP00000272643.3:p.Cys319%3D   \n",
       "23          ENSP00000283254.7:p.Thr679%3D   \n",
       "24         ENSP00000386796.1:p.Ala1304Thr   \n",
       "25          ENSP00000348278.4:p.His432%3D   \n",
       "26                                    NaN   \n",
       "27          ENSP00000264065.6:p.Arg661Thr   \n",
       "28          ENSP00000282276.6:p.Gly200%3D   \n",
       "29          ENSP00000350447.4:p.Gly192Val   \n",
       "30           ENSP00000171887.4:p.His51%3D   \n",
       "31           ENSP00000272907.6:p.His91Tyr   \n",
       "32                                    NaN   \n",
       "33                                    NaN   \n",
       "34          ENSP00000341539.4:p.Arg273Leu   \n",
       "35                                    NaN   \n",
       "36                                    NaN   \n",
       "37                                    NaN   \n",
       "38          ENSP00000381097.3:p.Ala872Val   \n",
       "39          ENSP00000286349.3:p.Ser331%3D   \n",
       "40          ENSP00000260843.4:p.Thr205%3D   \n",
       "41                                    NaN   \n",
       "42          ENSP00000263967.3:p.Gly106Val   \n",
       "43                                    NaN   \n",
       "44                                    NaN   \n",
       "45          ENSP00000295890.4:p.Arg142%3D   \n",
       "46          ENSP00000406022.1:p.Ala679%3D   \n",
       "47                                    NaN   \n",
       "48         ENSP00000349629.3:p.Pro1283%3D   \n",
       "49          ENSP00000442477.2:p.Ser273Pro   \n",
       "50          ENSP00000264428.4:p.His394%3D   \n",
       "51                                    NaN   \n",
       "52                                    NaN   \n",
       "53                                    NaN   \n",
       "54          ENSP00000274473.6:p.Pro595%3D   \n",
       "55          ENSP00000302629.4:p.Val499Ile   \n",
       "56                                    NaN   \n",
       "57                                    NaN   \n",
       "58          ENSP00000399259.2:p.Leu193%3D   \n",
       "59          ENSP00000285873.6:p.Ile562Val   \n",
       "60                                    NaN   \n",
       "61                                    NaN   \n",
       "62                                    NaN   \n",
       "63                                    NaN   \n",
       "64                                    NaN   \n",
       "65                                    NaN   \n",
       "66                                    NaN   \n",
       "67                                    NaN   \n",
       "68          ENSP00000351206.3:p.Asn153%3D   \n",
       "69          ENSP00000275300.2:p.Leu295%3D   \n",
       "70         ENSP00000275358.3:p.Asn1092Tyr   \n",
       "71          ENSP00000340125.3:p.Val143Met   \n",
       "72           ENSP00000258749.5:p.Asp28Asn   \n",
       "73                                    NaN   \n",
       "74         ENSP00000379228.1:p.Val1037Met   \n",
       "75          ENSP00000282869.5:p.Thr125Ile   \n",
       "76                                    NaN   \n",
       "77          ENSP00000265723.4:p.Ser101%3D   \n",
       "78                                    NaN   \n",
       "79                                    NaN   \n",
       "80          ENSP00000288490.4:p.Thr969%3D   \n",
       "81           ENSP00000305107.3:p.Ser93Cys   \n",
       "82           ENSP00000373783.3:p.Asp86%3D   \n",
       "83                                    NaN   \n",
       "84                                    NaN   \n",
       "85                                    NaN   \n",
       "86                                    NaN   \n",
       "87          ENSP00000380460.3:p.Ala305Val   \n",
       "88                                    NaN   \n",
       "89                                    NaN   \n",
       "90          ENSP00000350708.3:p.Gln333Glu   \n",
       "91                                    NaN   \n",
       "92          ENSP00000317817.8:p.Arg592Gln   \n",
       "93                                    NaN   \n",
       "94                                    NaN   \n",
       "95                                    NaN   \n",
       "96                                    NaN   \n",
       "97                                    NaN   \n",
       "98          ENSP00000308292.5:p.Asp252Glu   \n",
       "99                                    NaN   \n",
       "100                                   NaN   \n",
       "101                                   NaN   \n",
       "102        ENSP00000349568.3:p.Gln1198Leu   \n",
       "103                                   NaN   \n",
       "104                                   NaN   \n",
       "105         ENSP00000358232.3:p.His198%3D   \n",
       "106                                   NaN   \n",
       "107                                   NaN   \n",
       "108                                   NaN   \n",
       "109         ENSP00000325128.2:p.Thr325Ile   \n",
       "110         ENSP00000323688.4:p.Ile255%3D   \n",
       "111         ENSP00000308595.1:p.Phe184Leu   \n",
       "112         ENSP00000294312.3:p.Pro191Leu   \n",
       "113                                   NaN   \n",
       "114         ENSP00000260302.3:p.Arg461Cys   \n",
       "115         ENSP00000380903.1:p.Pro552Thr   \n",
       "116                                   NaN   \n",
       "117                                   NaN   \n",
       "118                                   NaN   \n",
       "119                                   NaN   \n",
       "120                                   NaN   \n",
       "121        ENSP00000261168.4:p.Arg1188Gln   \n",
       "122          ENSP00000256078.4:p.Gly12Asp   \n",
       "123                                   NaN   \n",
       "124                                   NaN   \n",
       "125                                   NaN   \n",
       "126                                   NaN   \n",
       "127                                   NaN   \n",
       "128                                   NaN   \n",
       "129         ENSP00000365048.3:p.Tyr192Ser   \n",
       "130                                   NaN   \n",
       "131                                   NaN   \n",
       "132                                   NaN   \n",
       "133                                   NaN   \n",
       "134                                   NaN   \n",
       "135                                   NaN   \n",
       "136         ENSP00000353010.4:p.Glu766Ter   \n",
       "137                                   NaN   \n",
       "138         ENSP00000313881.3:p.Pro221%3D   \n",
       "139        ENSP00000349325.6:p.Glu1052Ter   \n",
       "140        ENSP00000349325.6:p.Val1041Glu   \n",
       "141        ENSP00000219905.7:p.Lys1969%3D   \n",
       "142                                   NaN   \n",
       "143         ENSP00000405963.3:p.Arg271Trp   \n",
       "144                                   NaN   \n",
       "145                                   NaN   \n",
       "146         ENSP00000306473.3:p.Val123%3D   \n",
       "147                                   NaN   \n",
       "148                                   NaN   \n",
       "149                                   NaN   \n",
       "150                                   NaN   \n",
       "151                                   NaN   \n",
       "152         ENSP00000262483.8:p.Thr235%3D   \n",
       "153          ENSP00000254868.4:p.Cys35Arg   \n",
       "154         ENSP00000269305.4:p.Val197Met   \n",
       "155          ENSP00000355026.3:p.Thr59%3D   \n",
       "156         ENSP00000305263.7:p.Thr335%3D   \n",
       "157         ENSP00000301665.3:p.Cys250%3D   \n",
       "158         ENSP00000246646.2:p.Ser423Pro   \n",
       "159                                   NaN   \n",
       "160         ENSP00000313731.5:p.Ile243%3D   \n",
       "161                                   NaN   \n",
       "162         ENSP00000262419.6:p.Thr221Ile   \n",
       "163                                   NaN   \n",
       "164                                   NaN   \n",
       "165         ENSP00000258975.6:p.His240Arg   \n",
       "166                                   NaN   \n",
       "167                                   NaN   \n",
       "168                                   NaN   \n",
       "169                                   NaN   \n",
       "170         ENSP00000343445.5:p.Lys331Glu   \n",
       "171                                   NaN   \n",
       "172                                   NaN   \n",
       "173          ENSP00000319254.5:p.Leu92%3D   \n",
       "174                                   NaN   \n",
       "175         ENSP00000270328.4:p.Val680Gly   \n",
       "176         ENSP00000263377.1:p.Ala150Thr   \n",
       "177        ENSP00000291495.3:p.Arg1070Leu   \n",
       "178                                   NaN   \n",
       "179         ENSP00000269967.2:p.Gly190%3D   \n",
       "180                                   NaN   \n",
       "181         ENSP00000222145.3:p.Leu809%3D   \n",
       "182         ENSP00000301420.1:p.Lys186Glu   \n",
       "183                                   NaN   \n",
       "184         ENSP00000368305.4:p.Arg134%3D   \n",
       "185                                   NaN   \n",
       "186         ENSP00000284987.5:p.Glu484Lys   \n",
       "187        ENSP00000286827.3:p.Asp1082Glu   \n",
       "188                                   NaN   \n",
       "189                                   NaN   \n",
       "190                                   NaN   \n",
       "191                                   NaN   \n",
       "192          ENSP00000334962.5:p.Pro33%3D   \n",
       "193                                   NaN   \n",
       "194                                   NaN   \n",
       "195         ENSP00000217961.4:p.Arg436His   \n",
       "196         ENSP00000405222.1:p.Phe445Leu   \n",
       "197                                   NaN   \n",
       "198                                   NaN   \n",
       "199         ENSP00000345728.6:p.Lys109%3D   \n",
       "200           ENSP00000171757.2:p.Asn3His   \n",
       "201                                   NaN   \n",
       "202         ENSP00000359329.2:p.Thr113%3D   \n",
       "203        ENSP00000328939.4:p.Glu1237Lys   \n",
       "204                                   NaN   \n",
       "205  ENSP00000423313.1:p.Asn460IlefsTer30   \n",
       "206          ENSP00000370904.2:p.Lys79Ter   \n",
       "207                                   NaN   \n",
       "208                                   NaN   \n",
       "209                                   NaN   \n",
       "210                                   NaN   \n",
       "211        ENSP00000357789.1:p.Ser2426%3D   \n",
       "212                                   NaN   \n",
       "213                                   NaN   \n",
       "214                                   NaN   \n",
       "215                                   NaN   \n",
       "216         ENSP00000350348.4:p.Arg516Ter   \n",
       "217         ENSP00000287652.4:p.Pro358Leu   \n",
       "218                                   NaN   \n",
       "219                                   NaN   \n",
       "220                                   NaN   \n",
       "221                                   NaN   \n",
       "222                                   NaN   \n",
       "223                                   NaN   \n",
       "224                                   NaN   \n",
       "225                                   NaN   \n",
       "226                                   NaN   \n",
       "227                                   NaN   \n",
       "228                                   NaN   \n",
       "229                                   NaN   \n",
       "230         ENSP00000334068.1:p.Phe316Ser   \n",
       "231                                   NaN   \n",
       "232         ENSP00000224600.4:p.Thr279Met   \n",
       "233                                   NaN   \n",
       "234                                   NaN   \n",
       "235                                   NaN   \n",
       "236                                   NaN   \n",
       "237         ENSP00000317671.5:p.Lys468Met   \n",
       "238                                   NaN   \n",
       "239                                   NaN   \n",
       "240                                   NaN   \n",
       "241                                   NaN   \n",
       "242                                   NaN   \n",
       "243                                   NaN   \n",
       "244         ENSP00000267807.7:p.Arg481%3D   \n",
       "245                                   NaN   \n",
       "246                                   NaN   \n",
       "247                                   NaN   \n",
       "248                                   NaN   \n",
       "249                                   NaN   \n",
       "250                                   NaN   \n",
       "251                                   NaN   \n",
       "252         ENSP00000301547.5:p.Cys485Phe   \n",
       "253         ENSP00000301547.5:p.Lys480Thr   \n",
       "254         ENSP00000262265.4:p.Gly204%3D   \n",
       "255                                   NaN   \n",
       "256                                   NaN   \n",
       "257                                   NaN   \n",
       "258                                   NaN   \n",
       "259         ENSP00000345728.6:p.Val767Leu   \n",
       "260                                   NaN   \n",
       "261         ENSP00000295550.4:p.Ala174Asp   \n",
       "\n",
       "                                     CLIN_SIG                  PolyPhen  \\\n",
       "0                                         NaN             benign(0.007)   \n",
       "1                                         NaN                       NaN   \n",
       "2                                         NaN                       NaN   \n",
       "3                                         NaN             benign(0.208)   \n",
       "4                                         NaN                       NaN   \n",
       "5                                         NaN  probably_damaging(0.992)   \n",
       "6                                         NaN  probably_damaging(0.997)   \n",
       "7                                         NaN  possibly_damaging(0.776)   \n",
       "8                                         NaN  possibly_damaging(0.811)   \n",
       "9                                         NaN                       NaN   \n",
       "10                                        NaN                       NaN   \n",
       "11                                        NaN                       NaN   \n",
       "12                                        NaN                       NaN   \n",
       "13                                        NaN  probably_damaging(0.992)   \n",
       "14                                        NaN             benign(0.146)   \n",
       "15                                        NaN  probably_damaging(0.982)   \n",
       "16                                        NaN                       NaN   \n",
       "17                                        NaN                       NaN   \n",
       "18                                        NaN             benign(0.115)   \n",
       "19                                     benign                       NaN   \n",
       "20                                        NaN             benign(0.035)   \n",
       "21                                        NaN                       NaN   \n",
       "22                                        NaN                       NaN   \n",
       "23                                        NaN                       NaN   \n",
       "24                                        NaN  possibly_damaging(0.461)   \n",
       "25                                        NaN                       NaN   \n",
       "26                                        NaN                       NaN   \n",
       "27                                        NaN  possibly_damaging(0.587)   \n",
       "28                                        NaN                       NaN   \n",
       "29                                        NaN                 benign(0)   \n",
       "30                                        NaN                       NaN   \n",
       "31                                        NaN  probably_damaging(0.946)   \n",
       "32                                        NaN                       NaN   \n",
       "33                                        NaN                       NaN   \n",
       "34                                        NaN             benign(0.034)   \n",
       "35                                        NaN                       NaN   \n",
       "36                                        NaN                       NaN   \n",
       "37                                        NaN                       NaN   \n",
       "38                                        NaN             benign(0.137)   \n",
       "39                                        NaN                       NaN   \n",
       "40                                        NaN                       NaN   \n",
       "41                                        NaN                       NaN   \n",
       "42   uncertain_significance&likely_pathogenic      probably_damaging(1)   \n",
       "43                                        NaN                       NaN   \n",
       "44                                        NaN                       NaN   \n",
       "45                                        NaN                       NaN   \n",
       "46                                        NaN                       NaN   \n",
       "47                                        NaN                       NaN   \n",
       "48                                        NaN                       NaN   \n",
       "49                                        NaN             benign(0.062)   \n",
       "50                                        NaN                       NaN   \n",
       "51                                        NaN                       NaN   \n",
       "52                                        NaN                       NaN   \n",
       "53                                        NaN                       NaN   \n",
       "54                              likely_benign                       NaN   \n",
       "55                                        NaN             benign(0.042)   \n",
       "56                                        NaN                       NaN   \n",
       "57                                        NaN                       NaN   \n",
       "58                                        NaN                       NaN   \n",
       "59                                        NaN             benign(0.015)   \n",
       "60                                        NaN                       NaN   \n",
       "61                                        NaN                       NaN   \n",
       "62                                        NaN                       NaN   \n",
       "63                                        NaN                       NaN   \n",
       "64                                        NaN                       NaN   \n",
       "65                                        NaN                       NaN   \n",
       "66                                        NaN                       NaN   \n",
       "67                                        NaN                       NaN   \n",
       "68                                        NaN                       NaN   \n",
       "69                                        NaN                       NaN   \n",
       "70                                        NaN             benign(0.005)   \n",
       "71                                        NaN                 benign(0)   \n",
       "72                                        NaN                 benign(0)   \n",
       "73                                        NaN                       NaN   \n",
       "74                                        NaN  possibly_damaging(0.744)   \n",
       "75                                        NaN             benign(0.202)   \n",
       "76                                        NaN                       NaN   \n",
       "77                     uncertain_significance                       NaN   \n",
       "78                                        NaN                       NaN   \n",
       "79                                        NaN                       NaN   \n",
       "80                                        NaN                       NaN   \n",
       "81                                        NaN  probably_damaging(0.979)   \n",
       "82                                        NaN                       NaN   \n",
       "83                                        NaN                       NaN   \n",
       "84                                        NaN                       NaN   \n",
       "85                                        NaN                       NaN   \n",
       "86                                        NaN                       NaN   \n",
       "87                                        NaN  probably_damaging(0.999)   \n",
       "88                                        NaN                       NaN   \n",
       "89                                        NaN                       NaN   \n",
       "90                                        NaN             benign(0.055)   \n",
       "91                                        NaN                       NaN   \n",
       "92                                        NaN             benign(0.003)   \n",
       "93                                        NaN                       NaN   \n",
       "94                                        NaN                       NaN   \n",
       "95                                        NaN                       NaN   \n",
       "96                                        NaN                       NaN   \n",
       "97                                        NaN                       NaN   \n",
       "98                                        NaN             benign(0.009)   \n",
       "99                                        NaN                       NaN   \n",
       "100                                       NaN                       NaN   \n",
       "101                                       NaN                       NaN   \n",
       "102                                       NaN  possibly_damaging(0.813)   \n",
       "103                                       NaN                       NaN   \n",
       "104                                       NaN                       NaN   \n",
       "105                                       NaN                       NaN   \n",
       "106                                       NaN                       NaN   \n",
       "107                                       NaN                       NaN   \n",
       "108                                       NaN                       NaN   \n",
       "109                                       NaN                 benign(0)   \n",
       "110                                       NaN                       NaN   \n",
       "111                                       NaN                 benign(0)   \n",
       "112                                       NaN  possibly_damaging(0.784)   \n",
       "113                                       NaN                       NaN   \n",
       "114                                       NaN  probably_damaging(0.996)   \n",
       "115                                       NaN             benign(0.045)   \n",
       "116                                       NaN                       NaN   \n",
       "117                                       NaN                       NaN   \n",
       "118                                       NaN                       NaN   \n",
       "119                                       NaN                       NaN   \n",
       "120                                       NaN                       NaN   \n",
       "121                                       NaN  possibly_damaging(0.742)   \n",
       "122              pathogenic&likely_pathogenic             benign(0.303)   \n",
       "123                                       NaN                       NaN   \n",
       "124                                       NaN                       NaN   \n",
       "125                                       NaN                       NaN   \n",
       "126                                       NaN                       NaN   \n",
       "127                                       NaN                       NaN   \n",
       "128                                       NaN                       NaN   \n",
       "129                                       NaN      probably_damaging(1)   \n",
       "130                                       NaN                       NaN   \n",
       "131                                       NaN                       NaN   \n",
       "132                                       NaN                       NaN   \n",
       "133                                       NaN                       NaN   \n",
       "134                                       NaN                       NaN   \n",
       "135                                       NaN                       NaN   \n",
       "136                                       NaN                       NaN   \n",
       "137                                       NaN                       NaN   \n",
       "138                                       NaN                       NaN   \n",
       "139                                       NaN                       NaN   \n",
       "140                                       NaN             benign(0.119)   \n",
       "141                                       NaN                       NaN   \n",
       "142                                       NaN                       NaN   \n",
       "143                                       NaN  probably_damaging(0.927)   \n",
       "144                                       NaN                       NaN   \n",
       "145                                       NaN                       NaN   \n",
       "146                                       NaN                       NaN   \n",
       "147                                       NaN                       NaN   \n",
       "148                                       NaN                       NaN   \n",
       "149                                       NaN                       NaN   \n",
       "150                                       NaN                       NaN   \n",
       "151                                       NaN                       NaN   \n",
       "152                                    benign                       NaN   \n",
       "153                                       NaN                 benign(0)   \n",
       "154                    uncertain_significance  probably_damaging(0.961)   \n",
       "155                                       NaN                       NaN   \n",
       "156                                       NaN                       NaN   \n",
       "157                                       NaN                       NaN   \n",
       "158                                       NaN                 benign(0)   \n",
       "159                                       NaN                       NaN   \n",
       "160                                       NaN                       NaN   \n",
       "161                                       NaN                       NaN   \n",
       "162                                       NaN             benign(0.277)   \n",
       "163                                       NaN                       NaN   \n",
       "164                                       NaN                       NaN   \n",
       "165                                       NaN             benign(0.023)   \n",
       "166                                       NaN                       NaN   \n",
       "167                                       NaN                       NaN   \n",
       "168                                       NaN                       NaN   \n",
       "169                                       NaN                       NaN   \n",
       "170                                       NaN   probably_damaging(0.96)   \n",
       "171                                       NaN                       NaN   \n",
       "172                                       NaN                       NaN   \n",
       "173                                    benign                       NaN   \n",
       "174                                       NaN                       NaN   \n",
       "175                                       NaN  probably_damaging(0.999)   \n",
       "176                                       NaN  probably_damaging(0.998)   \n",
       "177                                       NaN  possibly_damaging(0.892)   \n",
       "178                                       NaN                       NaN   \n",
       "179                                       NaN                       NaN   \n",
       "180                                       NaN                       NaN   \n",
       "181                                       NaN                       NaN   \n",
       "182                                       NaN              benign(0.02)   \n",
       "183                                       NaN                       NaN   \n",
       "184                                       NaN                       NaN   \n",
       "185                                       NaN                       NaN   \n",
       "186                                       NaN   possibly_damaging(0.71)   \n",
       "187                                       NaN  probably_damaging(0.991)   \n",
       "188                                       NaN                       NaN   \n",
       "189                                       NaN                       NaN   \n",
       "190                                       NaN                       NaN   \n",
       "191                                       NaN                       NaN   \n",
       "192                             likely_benign                       NaN   \n",
       "193                                       NaN                       NaN   \n",
       "194                                       NaN                       NaN   \n",
       "195                                       NaN             benign(0.003)   \n",
       "196                                       NaN  possibly_damaging(0.609)   \n",
       "197                                       NaN                       NaN   \n",
       "198                                       NaN                       NaN   \n",
       "199                                    benign                       NaN   \n",
       "200                                       NaN                 benign(0)   \n",
       "201                                       NaN                       NaN   \n",
       "202                                       NaN                       NaN   \n",
       "203                                       NaN  possibly_damaging(0.824)   \n",
       "204                                       NaN                       NaN   \n",
       "205                                       NaN                       NaN   \n",
       "206                                       NaN                       NaN   \n",
       "207                                       NaN                       NaN   \n",
       "208                                       NaN                       NaN   \n",
       "209                              not_provided                       NaN   \n",
       "210                                       NaN                       NaN   \n",
       "211                                       NaN                       NaN   \n",
       "212                                       NaN                       NaN   \n",
       "213                                       NaN                       NaN   \n",
       "214                                       NaN                       NaN   \n",
       "215                                       NaN                       NaN   \n",
       "216                                       NaN                       NaN   \n",
       "217                                       NaN             benign(0.006)   \n",
       "218                                       NaN                       NaN   \n",
       "219                                       NaN                       NaN   \n",
       "220                                       NaN                       NaN   \n",
       "221                                       NaN                       NaN   \n",
       "222                                       NaN                       NaN   \n",
       "223                                       NaN                       NaN   \n",
       "224                                       NaN                       NaN   \n",
       "225                                       NaN                       NaN   \n",
       "226                                       NaN                       NaN   \n",
       "227                                       NaN                       NaN   \n",
       "228                                       NaN                       NaN   \n",
       "229                                       NaN                       NaN   \n",
       "230                                       NaN             benign(0.003)   \n",
       "231                                       NaN                       NaN   \n",
       "232                                       NaN  probably_damaging(0.998)   \n",
       "233                                       NaN                       NaN   \n",
       "234                                       NaN                       NaN   \n",
       "235                                       NaN                       NaN   \n",
       "236                                       NaN                       NaN   \n",
       "237                                       NaN             benign(0.031)   \n",
       "238                                       NaN                       NaN   \n",
       "239                                       NaN                       NaN   \n",
       "240                                       NaN                       NaN   \n",
       "241                                       NaN                       NaN   \n",
       "242                                       NaN                       NaN   \n",
       "243                                       NaN                       NaN   \n",
       "244                                       NaN                       NaN   \n",
       "245                                       NaN                       NaN   \n",
       "246                                       NaN                       NaN   \n",
       "247                                       NaN                       NaN   \n",
       "248                                       NaN                       NaN   \n",
       "249                                       NaN                       NaN   \n",
       "250                                       NaN                       NaN   \n",
       "251                                       NaN                       NaN   \n",
       "252                                       NaN                 benign(0)   \n",
       "253                                       NaN   probably_damaging(0.93)   \n",
       "254                                       NaN                       NaN   \n",
       "255                                       NaN                       NaN   \n",
       "256                                       NaN                       NaN   \n",
       "257                                       NaN                       NaN   \n",
       "258                                       NaN                       NaN   \n",
       "259                                    benign  probably_damaging(0.961)   \n",
       "260                                       NaN                       NaN   \n",
       "261                                       NaN  probably_damaging(0.993)   \n",
       "\n",
       "                                 SIFT  \n",
       "0                        tolerated(1)  \n",
       "1                                 NaN  \n",
       "2                                 NaN  \n",
       "3                     tolerated(0.37)  \n",
       "4                                 NaN  \n",
       "5       deleterious_low_confidence(0)  \n",
       "6                      deleterious(0)  \n",
       "7                   deleterious(0.03)  \n",
       "8                     tolerated(0.28)  \n",
       "9                                 NaN  \n",
       "10                                NaN  \n",
       "11                                NaN  \n",
       "12                                NaN  \n",
       "13                     deleterious(0)  \n",
       "14                    tolerated(0.56)  \n",
       "15                     deleterious(0)  \n",
       "16                                NaN  \n",
       "17                                NaN  \n",
       "18                    tolerated(0.17)  \n",
       "19                                NaN  \n",
       "20                    tolerated(0.31)  \n",
       "21                                NaN  \n",
       "22                                NaN  \n",
       "23                                NaN  \n",
       "24                  deleterious(0.01)  \n",
       "25                                NaN  \n",
       "26                                NaN  \n",
       "27                  deleterious(0.01)  \n",
       "28                                NaN  \n",
       "29                       tolerated(1)  \n",
       "30                                NaN  \n",
       "31                  deleterious(0.03)  \n",
       "32                                NaN  \n",
       "33                                NaN  \n",
       "34                    tolerated(0.48)  \n",
       "35                                NaN  \n",
       "36                                NaN  \n",
       "37                                NaN  \n",
       "38                  deleterious(0.05)  \n",
       "39                                NaN  \n",
       "40                                NaN  \n",
       "41                                NaN  \n",
       "42                  deleterious(0.01)  \n",
       "43                                NaN  \n",
       "44                                NaN  \n",
       "45                                NaN  \n",
       "46                                NaN  \n",
       "47                                NaN  \n",
       "48                                NaN  \n",
       "49                    tolerated(0.08)  \n",
       "50                                NaN  \n",
       "51                                NaN  \n",
       "52                                NaN  \n",
       "53                                NaN  \n",
       "54                                NaN  \n",
       "55                    tolerated(0.46)  \n",
       "56                                NaN  \n",
       "57                                NaN  \n",
       "58                                NaN  \n",
       "59                    tolerated(0.86)  \n",
       "60                                NaN  \n",
       "61                                NaN  \n",
       "62                                NaN  \n",
       "63                                NaN  \n",
       "64                                NaN  \n",
       "65                                NaN  \n",
       "66                                NaN  \n",
       "67                                NaN  \n",
       "68                                NaN  \n",
       "69                                NaN  \n",
       "70                    tolerated(0.11)  \n",
       "71        tolerated_low_confidence(1)  \n",
       "72                    tolerated(0.24)  \n",
       "73                                NaN  \n",
       "74                  deleterious(0.01)  \n",
       "75                    tolerated(0.13)  \n",
       "76                                NaN  \n",
       "77                                NaN  \n",
       "78                                NaN  \n",
       "79                                NaN  \n",
       "80                                NaN  \n",
       "81                    tolerated(0.06)  \n",
       "82                                NaN  \n",
       "83                                NaN  \n",
       "84                                NaN  \n",
       "85                                NaN  \n",
       "86                                NaN  \n",
       "87                     deleterious(0)  \n",
       "88                                NaN  \n",
       "89                                NaN  \n",
       "90                       tolerated(1)  \n",
       "91                                NaN  \n",
       "92                    tolerated(0.59)  \n",
       "93                                NaN  \n",
       "94                                NaN  \n",
       "95                                NaN  \n",
       "96                                NaN  \n",
       "97                                NaN  \n",
       "98                    tolerated(0.34)  \n",
       "99                                NaN  \n",
       "100                               NaN  \n",
       "101                               NaN  \n",
       "102                    deleterious(0)  \n",
       "103                               NaN  \n",
       "104                               NaN  \n",
       "105                               NaN  \n",
       "106                               NaN  \n",
       "107                               NaN  \n",
       "108                               NaN  \n",
       "109    tolerated_low_confidence(0.25)  \n",
       "110                               NaN  \n",
       "111                      tolerated(1)  \n",
       "112                 deleterious(0.01)  \n",
       "113                               NaN  \n",
       "114                    deleterious(0)  \n",
       "115                    tolerated(0.1)  \n",
       "116                               NaN  \n",
       "117                               NaN  \n",
       "118                               NaN  \n",
       "119                               NaN  \n",
       "120                               NaN  \n",
       "121  deleterious_low_confidence(0.02)  \n",
       "122                    deleterious(0)  \n",
       "123                               NaN  \n",
       "124                               NaN  \n",
       "125                               NaN  \n",
       "126                               NaN  \n",
       "127                               NaN  \n",
       "128                               NaN  \n",
       "129                    deleterious(0)  \n",
       "130                               NaN  \n",
       "131                               NaN  \n",
       "132                               NaN  \n",
       "133                               NaN  \n",
       "134                               NaN  \n",
       "135                               NaN  \n",
       "136                               NaN  \n",
       "137                               NaN  \n",
       "138                               NaN  \n",
       "139                               NaN  \n",
       "140                    deleterious(0)  \n",
       "141                               NaN  \n",
       "142                               NaN  \n",
       "143                    deleterious(0)  \n",
       "144                               NaN  \n",
       "145                               NaN  \n",
       "146                               NaN  \n",
       "147                               NaN  \n",
       "148                               NaN  \n",
       "149                               NaN  \n",
       "150                               NaN  \n",
       "151                               NaN  \n",
       "152                               NaN  \n",
       "153                   tolerated(0.07)  \n",
       "154                 deleterious(0.04)  \n",
       "155                               NaN  \n",
       "156                               NaN  \n",
       "157                               NaN  \n",
       "158                   tolerated(0.58)  \n",
       "159                               NaN  \n",
       "160                               NaN  \n",
       "161                               NaN  \n",
       "162                 deleterious(0.04)  \n",
       "163                               NaN  \n",
       "164                               NaN  \n",
       "165                   tolerated(0.75)  \n",
       "166                               NaN  \n",
       "167                               NaN  \n",
       "168                               NaN  \n",
       "169                               NaN  \n",
       "170                 deleterious(0.01)  \n",
       "171                               NaN  \n",
       "172                               NaN  \n",
       "173                               NaN  \n",
       "174                               NaN  \n",
       "175                    deleterious(0)  \n",
       "176                    deleterious(0)  \n",
       "177                    deleterious(0)  \n",
       "178                               NaN  \n",
       "179                               NaN  \n",
       "180                               NaN  \n",
       "181                               NaN  \n",
       "182                   tolerated(0.84)  \n",
       "183                               NaN  \n",
       "184                               NaN  \n",
       "185                               NaN  \n",
       "186                   tolerated(0.16)  \n",
       "187                   tolerated(0.96)  \n",
       "188                               NaN  \n",
       "189                               NaN  \n",
       "190                               NaN  \n",
       "191                               NaN  \n",
       "192                               NaN  \n",
       "193                               NaN  \n",
       "194                               NaN  \n",
       "195                      tolerated(1)  \n",
       "196                    deleterious(0)  \n",
       "197                               NaN  \n",
       "198                               NaN  \n",
       "199                               NaN  \n",
       "200    tolerated_low_confidence(0.26)  \n",
       "201                               NaN  \n",
       "202                               NaN  \n",
       "203                   tolerated(0.17)  \n",
       "204                               NaN  \n",
       "205                               NaN  \n",
       "206                               NaN  \n",
       "207                               NaN  \n",
       "208                               NaN  \n",
       "209                               NaN  \n",
       "210                               NaN  \n",
       "211                               NaN  \n",
       "212                               NaN  \n",
       "213                               NaN  \n",
       "214                               NaN  \n",
       "215                               NaN  \n",
       "216                               NaN  \n",
       "217                   tolerated(0.29)  \n",
       "218                               NaN  \n",
       "219                               NaN  \n",
       "220                               NaN  \n",
       "221                               NaN  \n",
       "222                               NaN  \n",
       "223                               NaN  \n",
       "224                               NaN  \n",
       "225                               NaN  \n",
       "226                               NaN  \n",
       "227                               NaN  \n",
       "228                               NaN  \n",
       "229                               NaN  \n",
       "230                 deleterious(0.04)  \n",
       "231                               NaN  \n",
       "232                   tolerated(0.07)  \n",
       "233                               NaN  \n",
       "234                               NaN  \n",
       "235                               NaN  \n",
       "236                               NaN  \n",
       "237                    tolerated(0.1)  \n",
       "238                               NaN  \n",
       "239                               NaN  \n",
       "240                               NaN  \n",
       "241                               NaN  \n",
       "242                               NaN  \n",
       "243                               NaN  \n",
       "244                               NaN  \n",
       "245                               NaN  \n",
       "246                               NaN  \n",
       "247                               NaN  \n",
       "248                               NaN  \n",
       "249                               NaN  \n",
       "250                               NaN  \n",
       "251                               NaN  \n",
       "252                      tolerated(1)  \n",
       "253                    deleterious(0)  \n",
       "254                               NaN  \n",
       "255                               NaN  \n",
       "256                               NaN  \n",
       "257                               NaN  \n",
       "258                               NaN  \n",
       "259                 deleterious(0.04)  \n",
       "260                               NaN  \n",
       "261                    deleterious(0)  "
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "vcf_2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "mapped=vcf_1.merge(vcf_2, on=['seqnames','start'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames</th>\n",
       "      <th>start</th>\n",
       "      <th>end_x</th>\n",
       "      <th>width_x</th>\n",
       "      <th>strand_x</th>\n",
       "      <th>AD_x</th>\n",
       "      <th>AD.1_x</th>\n",
       "      <th>AF_x</th>\n",
       "      <th>Consequence_x</th>\n",
       "      <th>IMPACT_x</th>\n",
       "      <th>SYMBOL_x</th>\n",
       "      <th>HGVSc_x</th>\n",
       "      <th>HGVSp_x</th>\n",
       "      <th>CLIN_SIG_x</th>\n",
       "      <th>PolyPhen_x</th>\n",
       "      <th>SIFT_x</th>\n",
       "      <th>end_y</th>\n",
       "      <th>width_y</th>\n",
       "      <th>strand_y</th>\n",
       "      <th>AD_y</th>\n",
       "      <th>AD.1_y</th>\n",
       "      <th>AF_y</th>\n",
       "      <th>Consequence_y</th>\n",
       "      <th>IMPACT_y</th>\n",
       "      <th>SYMBOL_y</th>\n",
       "      <th>HGVSc_y</th>\n",
       "      <th>HGVSp_y</th>\n",
       "      <th>CLIN_SIG_y</th>\n",
       "      <th>PolyPhen_y</th>\n",
       "      <th>SIFT_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [seqnames, start, end_x, width_x, strand_x, AD_x, AD.1_x, AF_x, Consequence_x, IMPACT_x, SYMBOL_x, HGVSc_x, HGVSp_x, CLIN_SIG_x, PolyPhen_x, SIFT_x, end_y, width_y, strand_y, AD_y, AD.1_y, AF_y, Consequence_y, IMPACT_y, SYMBOL_y, HGVSc_y, HGVSp_y, CLIN_SIG_y, PolyPhen_y, SIFT_y]\n",
       "Index: []"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapped"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [],
   "source": [
    "#merged=pd.merge(vcf_1, vcf_2,  how='left', left_on=['seqnames','start'], right_on = ['seqnames','seqnames'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Checking mapped positions based on gene\n",
    "mapped=vcf_1.merge(vcf_2, on='SYMBOL')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>seqnames_x</th>\n",
       "      <th>start_x</th>\n",
       "      <th>end_x</th>\n",
       "      <th>width_x</th>\n",
       "      <th>strand_x</th>\n",
       "      <th>AD_x</th>\n",
       "      <th>AD.1_x</th>\n",
       "      <th>AF_x</th>\n",
       "      <th>Consequence_x</th>\n",
       "      <th>IMPACT_x</th>\n",
       "      <th>SYMBOL</th>\n",
       "      <th>HGVSc_x</th>\n",
       "      <th>HGVSp_x</th>\n",
       "      <th>CLIN_SIG_x</th>\n",
       "      <th>PolyPhen_x</th>\n",
       "      <th>SIFT_x</th>\n",
       "      <th>seqnames_y</th>\n",
       "      <th>start_y</th>\n",
       "      <th>end_y</th>\n",
       "      <th>width_y</th>\n",
       "      <th>strand_y</th>\n",
       "      <th>AD_y</th>\n",
       "      <th>AD.1_y</th>\n",
       "      <th>AF_y</th>\n",
       "      <th>Consequence_y</th>\n",
       "      <th>IMPACT_y</th>\n",
       "      <th>HGVSc_y</th>\n",
       "      <th>HGVSp_y</th>\n",
       "      <th>CLIN_SIG_y</th>\n",
       "      <th>PolyPhen_y</th>\n",
       "      <th>SIFT_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>2</td>\n",
       "      <td>64863545</td>\n",
       "      <td>64863545</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>33</td>\n",
       "      <td>4</td>\n",
       "      <td>0.108</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>SERTAD2</td>\n",
       "      <td>ENST00000313349.3:c.461C&gt;A</td>\n",
       "      <td>ENSP00000326933.3:p.Pro154His</td>\n",
       "      <td>NaN</td>\n",
       "      <td>possibly_damaging(0.818)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "      <td>2</td>\n",
       "      <td>64862518</td>\n",
       "      <td>64862518</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>8</td>\n",
       "      <td>0.258</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>ENST00000313349.3:c.*543T&gt;A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>3</td>\n",
       "      <td>134644699</td>\n",
       "      <td>134644699</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>61</td>\n",
       "      <td>5</td>\n",
       "      <td>0.076</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>EPHB1</td>\n",
       "      <td>ENST00000398015.3:c.100T&gt;A</td>\n",
       "      <td>ENSP00000381097.3:p.Trp34Arg</td>\n",
       "      <td>NaN</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "      <td>3</td>\n",
       "      <td>134967276</td>\n",
       "      <td>134967276</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>74</td>\n",
       "      <td>25</td>\n",
       "      <td>0.253</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ENST00000398015.3:c.2615C&gt;T</td>\n",
       "      <td>ENSP00000381097.3:p.Ala872Val</td>\n",
       "      <td>NaN</td>\n",
       "      <td>benign(0.137)</td>\n",
       "      <td>deleterious(0.05)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>12</td>\n",
       "      <td>25398281</td>\n",
       "      <td>25398281</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>29</td>\n",
       "      <td>14</td>\n",
       "      <td>0.326</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>ENST00000256078.4:c.38G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly13Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.44)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "      <td>12</td>\n",
       "      <td>25398284</td>\n",
       "      <td>25398284</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>11</td>\n",
       "      <td>0.229</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ENST00000256078.4:c.35G&gt;A</td>\n",
       "      <td>ENSP00000256078.4:p.Gly12Asp</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>benign(0.303)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>17</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578395</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>23</td>\n",
       "      <td>7</td>\n",
       "      <td>0.233</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>TP53</td>\n",
       "      <td>ENST00000269305.4:c.535C&gt;T</td>\n",
       "      <td>ENSP00000269305.4:p.His179Tyr</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>deleterious(0)</td>\n",
       "      <td>17</td>\n",
       "      <td>7578260</td>\n",
       "      <td>7578260</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>37</td>\n",
       "      <td>15</td>\n",
       "      <td>0.288</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ENST00000269305.4:c.589G&gt;A</td>\n",
       "      <td>ENSP00000269305.4:p.Val197Met</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>probably_damaging(0.961)</td>\n",
       "      <td>deleterious(0.04)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>3</td>\n",
       "      <td>178936091</td>\n",
       "      <td>178936091</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>43</td>\n",
       "      <td>9</td>\n",
       "      <td>0.185</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>PIK3CA</td>\n",
       "      <td>ENST00000263967.3:c.1633G&gt;A</td>\n",
       "      <td>ENSP00000263967.3:p.Glu545Lys</td>\n",
       "      <td>not_provided&amp;pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(0.909)</td>\n",
       "      <td>deleterious(0.02)</td>\n",
       "      <td>3</td>\n",
       "      <td>178916930</td>\n",
       "      <td>178916930</td>\n",
       "      <td>1</td>\n",
       "      <td>*</td>\n",
       "      <td>19</td>\n",
       "      <td>4</td>\n",
       "      <td>0.174</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>ENST00000263967.3:c.317G&gt;T</td>\n",
       "      <td>ENSP00000263967.3:p.Gly106Val</td>\n",
       "      <td>uncertain_significance&amp;likely_pathogenic</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "      <td>deleterious(0.01)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  seqnames_x    start_x      end_x  width_x strand_x  AD_x  AD.1_x   AF_x  \\\n",
       "0          2   64863545   64863545        1        *    33       4  0.108   \n",
       "1          3  134644699  134644699        1        *    61       5  0.076   \n",
       "2         12   25398281   25398281        1        *    29      14  0.326   \n",
       "3         17    7578395    7578395        1        *    23       7  0.233   \n",
       "4          3  178936091  178936091        1        *    43       9  0.185   \n",
       "\n",
       "      Consequence_x  IMPACT_x   SYMBOL                      HGVSc_x  \\\n",
       "0  missense_variant  MODERATE  SERTAD2   ENST00000313349.3:c.461C>A   \n",
       "1  missense_variant  MODERATE    EPHB1   ENST00000398015.3:c.100T>A   \n",
       "2  missense_variant  MODERATE     KRAS    ENST00000256078.4:c.38G>A   \n",
       "3  missense_variant  MODERATE     TP53   ENST00000269305.4:c.535C>T   \n",
       "4  missense_variant  MODERATE   PIK3CA  ENST00000263967.3:c.1633G>A   \n",
       "\n",
       "                         HGVSp_x                                 CLIN_SIG_x  \\\n",
       "0  ENSP00000326933.3:p.Pro154His                                        NaN   \n",
       "1   ENSP00000381097.3:p.Trp34Arg                                        NaN   \n",
       "2   ENSP00000256078.4:p.Gly13Asp               pathogenic&likely_pathogenic   \n",
       "3  ENSP00000269305.4:p.His179Tyr               pathogenic&likely_pathogenic   \n",
       "4  ENSP00000263967.3:p.Glu545Lys  not_provided&pathogenic&likely_pathogenic   \n",
       "\n",
       "                 PolyPhen_x             SIFT_x seqnames_y    start_y  \\\n",
       "0  possibly_damaging(0.818)  deleterious(0.02)          2   64862518   \n",
       "1  probably_damaging(0.999)     deleterious(0)          3  134967276   \n",
       "2              benign(0.44)  deleterious(0.04)         12   25398284   \n",
       "3  probably_damaging(0.991)     deleterious(0)         17    7578260   \n",
       "4  probably_damaging(0.909)  deleterious(0.02)          3  178916930   \n",
       "\n",
       "       end_y  width_y strand_y  AD_y  AD.1_y   AF_y        Consequence_y  \\\n",
       "0   64862518        1        *    23       8  0.258  3_prime_UTR_variant   \n",
       "1  134967276        1        *    74      25  0.253     missense_variant   \n",
       "2   25398284        1        *    37      11  0.229     missense_variant   \n",
       "3    7578260        1        *    37      15  0.288     missense_variant   \n",
       "4  178916930        1        *    19       4  0.174     missense_variant   \n",
       "\n",
       "   IMPACT_y                      HGVSc_y                        HGVSp_y  \\\n",
       "0  MODIFIER  ENST00000313349.3:c.*543T>A                            NaN   \n",
       "1  MODERATE  ENST00000398015.3:c.2615C>T  ENSP00000381097.3:p.Ala872Val   \n",
       "2  MODERATE    ENST00000256078.4:c.35G>A   ENSP00000256078.4:p.Gly12Asp   \n",
       "3  MODERATE   ENST00000269305.4:c.589G>A  ENSP00000269305.4:p.Val197Met   \n",
       "4  MODERATE   ENST00000263967.3:c.317G>T  ENSP00000263967.3:p.Gly106Val   \n",
       "\n",
       "                                 CLIN_SIG_y                PolyPhen_y  \\\n",
       "0                                       NaN                       NaN   \n",
       "1                                       NaN             benign(0.137)   \n",
       "2              pathogenic&likely_pathogenic             benign(0.303)   \n",
       "3                    uncertain_significance  probably_damaging(0.961)   \n",
       "4  uncertain_significance&likely_pathogenic      probably_damaging(1)   \n",
       "\n",
       "              SIFT_y  \n",
       "0                NaN  \n",
       "1  deleterious(0.05)  \n",
       "2     deleterious(0)  \n",
       "3  deleterious(0.04)  \n",
       "4  deleterious(0.01)  "
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapped"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "mapped_t=mapped[['Consequence_x','Consequence_y','IMPACT_x','IMPACT_y','CLIN_SIG_x','CLIN_SIG_y','start_x','start_y','PolyPhen_x','PolyPhen_y']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Consequence_x</th>\n",
       "      <th>Consequence_y</th>\n",
       "      <th>IMPACT_x</th>\n",
       "      <th>IMPACT_y</th>\n",
       "      <th>CLIN_SIG_x</th>\n",
       "      <th>CLIN_SIG_y</th>\n",
       "      <th>start_x</th>\n",
       "      <th>start_y</th>\n",
       "      <th>PolyPhen_x</th>\n",
       "      <th>PolyPhen_y</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>0</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MODIFIER</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>64863545</td>\n",
       "      <td>64862518</td>\n",
       "      <td>possibly_damaging(0.818)</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>134644699</td>\n",
       "      <td>134967276</td>\n",
       "      <td>probably_damaging(0.999)</td>\n",
       "      <td>benign(0.137)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>2</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>25398281</td>\n",
       "      <td>25398284</td>\n",
       "      <td>benign(0.44)</td>\n",
       "      <td>benign(0.303)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>3</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>uncertain_significance</td>\n",
       "      <td>7578395</td>\n",
       "      <td>7578260</td>\n",
       "      <td>probably_damaging(0.991)</td>\n",
       "      <td>probably_damaging(0.961)</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>4</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>MODERATE</td>\n",
       "      <td>not_provided&amp;pathogenic&amp;likely_pathogenic</td>\n",
       "      <td>uncertain_significance&amp;likely_pathogenic</td>\n",
       "      <td>178936091</td>\n",
       "      <td>178916930</td>\n",
       "      <td>probably_damaging(0.909)</td>\n",
       "      <td>probably_damaging(1)</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Consequence_x        Consequence_y  IMPACT_x  IMPACT_y  \\\n",
       "0  missense_variant  3_prime_UTR_variant  MODERATE  MODIFIER   \n",
       "1  missense_variant     missense_variant  MODERATE  MODERATE   \n",
       "2  missense_variant     missense_variant  MODERATE  MODERATE   \n",
       "3  missense_variant     missense_variant  MODERATE  MODERATE   \n",
       "4  missense_variant     missense_variant  MODERATE  MODERATE   \n",
       "\n",
       "                                  CLIN_SIG_x  \\\n",
       "0                                        NaN   \n",
       "1                                        NaN   \n",
       "2               pathogenic&likely_pathogenic   \n",
       "3               pathogenic&likely_pathogenic   \n",
       "4  not_provided&pathogenic&likely_pathogenic   \n",
       "\n",
       "                                 CLIN_SIG_y    start_x    start_y  \\\n",
       "0                                       NaN   64863545   64862518   \n",
       "1                                       NaN  134644699  134967276   \n",
       "2              pathogenic&likely_pathogenic   25398281   25398284   \n",
       "3                    uncertain_significance    7578395    7578260   \n",
       "4  uncertain_significance&likely_pathogenic  178936091  178916930   \n",
       "\n",
       "                 PolyPhen_x                PolyPhen_y  \n",
       "0  possibly_damaging(0.818)                       NaN  \n",
       "1  probably_damaging(0.999)             benign(0.137)  \n",
       "2              benign(0.44)             benign(0.303)  \n",
       "3  probably_damaging(0.991)  probably_damaging(0.961)  \n",
       "4  probably_damaging(0.909)      probably_damaging(1)  "
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "mapped_t"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
